<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width">
<meta name="theme-color" content="#222"><meta name="generator" content="Hexo 7.3.0">

  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/apple-touch-icon-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">



<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/7.0.0/css/all.min.css" integrity="sha256-VHqXKFhhMxcpubYf9xiWdCiojEbY9NexQ4jh8AxbvcM=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.1.1/animate.min.css" integrity="sha256-PR7ttpcvz8qrF57fur/yAx1qXMFJeJFiA6pSzWi0OIE=" crossorigin="anonymous">

<script class="next-config" data-name="main" type="application/json">{"hostname":"dongzhang0725.github.io","root":"/","images":"/images","scheme":"Gemini","darkmode":false,"version":"8.25.0","exturl":false,"sidebar":{"position":"left","width_expanded":320,"width_dual_column":240,"display":"post","padding":18,"offset":12},"hljswrap":true,"codeblock":{"theme":{"light":"default","dark":"stackoverflow-dark"},"prism":{"light":"prism","dark":"prism-dark"},"copy_button":{"enable":false,"style":null},"fold":{"enable":false,"height":500},"language":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"stickytabs":false,"motion":{"enable":true,"async":false,"duration":200,"transition":{"menu_item":"fadeInDown","post_block":"fadeIn","post_header":"fadeInDown","post_body":"fadeInDown","coll_header":"fadeInLeft","sidebar":"fadeInUp"}},"i18n":{"placeholder":"Searching...","empty":"We didn't find any results for the search: ${query}","hits_time":"${hits} results found in ${time} ms","hits":"${hits} results found"},"path":"/search.xml","localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false}}</script><script src="/js/config.js" defer></script>

    <meta name="description" content="English   中文      PhyloSuite: from data acquisition to phylogenetic tree annotation (muti-gene dataset)  In order to make it easier for users to get started, here we briefly introduce the use of P">
<meta property="og:type" content="article">
<meta property="og:title" content="Muti-gene phylogeny">
<meta property="og:url" content="https://dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/index.html">
<meta property="og:site_name" content="PhyloSuite">
<meta property="og:description" content="English   中文      PhyloSuite: from data acquisition to phylogenetic tree annotation (muti-gene dataset)  In order to make it easier for users to get started, here we briefly introduce the use of P">
<meta property="og:locale" content="en_US">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cover.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/download_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_outgroup_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/highlight_identical_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/standard_trigger_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/tRNA_warning_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_results_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/manual_tRNA_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error1_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error2_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mark_NCR_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/extract_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/replace_name_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_copy_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_input_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_pcgs_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/inter_stp_codon_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cat_seq_drag_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/concatenation_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/missing_genes_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/custom_models_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/partitionfinder_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_length_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/IQ-TREE_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/flowchart_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_detect_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_error_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_input_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/bootstrap_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_order_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_lcolor_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/final_tree_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cover.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/download_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_outgroup_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/highlight_identical_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/standard_trigger_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/tRNA_warning_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_results_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/manual_tRNA_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error1_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error2_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mark_NCR_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/extract_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/replace_name_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_copy_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_input_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_pcgs_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cat_seq_drag_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/concatenation_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/missing_genes_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/custom_models_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/partitionfinder_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_length_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/IQ-TREE_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/flowchart_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_detect_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_error_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_input_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/bootstrap_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_order_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_lcolor_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/final_tree_en.jpg?raw=true">
<meta property="article:published_time" content="2019-03-02T03:06:19.000Z">
<meta property="article:modified_time" content="2025-11-25T03:09:44.000Z">
<meta property="article:author" content="Dong Zhang">
<meta property="article:tag" content="PyQT5">
<meta property="article:tag" content="Python">
<meta property="article:tag" content="Bioinformatics">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cover.jpg?raw=true">


<link rel="canonical" href="https://dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">


<script class="next-config" data-name="page" type="application/json">{"sidebar":"","isHome":false,"isPost":true,"lang":"en","comments":true,"permalink":"https://dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/","path":"PhyloSuite-demo/multi-gene-phylogeny/","title":"Muti-gene phylogeny"}</script>

<script class="next-config" data-name="calendar" type="application/json">""</script>
<title>Muti-gene phylogeny | PhyloSuite</title>
  








  
  <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js" integrity="sha256-XL2inqUJaslATFnHdJOi9GfQ60on8Wx1C2H8DYiN1xY=" crossorigin="anonymous" defer></script>
<script src="/js/utils.js" defer></script><script src="/js/motion.js" defer></script><script src="/js/sidebar.js" defer></script><script src="/js/next-boot.js" defer></script>

  <script src="https://cdnjs.cloudflare.com/ajax/libs/hexo-generator-searchdb/1.5.0/search.js" integrity="sha256-xFC6PJ82SL9b3WkGjFavNiA9gm5z6UBxWPiu4CYjptg=" crossorigin="anonymous" defer></script>
<script src="/js/third-party/search/local-search.js" defer></script>







  





  <noscript>
    <link rel="stylesheet" href="/css/noscript.css">
  </noscript>
</head>

<body itemscope itemtype="http://schema.org/WebPage" class="use-motion">
  <div class="headband"></div>

  <main class="main">
    <div class="column">
      <header class="header" itemscope itemtype="http://schema.org/WPHeader"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar" role="button">
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <i class="logo-line"></i>
      <p class="site-title">PhyloSuite</p>
      <i class="logo-line"></i>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger" aria-label="Search" role="button">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>



<nav class="site-nav">
  <ul class="main-menu menu"><li class="menu-item menu-item-home"><a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a></li><li class="menu-item menu-item-documentation"><a href="/documentation/" rel="section"><i class="fa fa-navicon fa-fw"></i>documentation</a></li><li class="menu-item menu-item-installation"><a href="/installation/" rel="section"><i class="fa fa-download fa-fw"></i>installation</a></li><li class="menu-item menu-item-example"><a href="/example/" rel="section"><i class="fa fa-print fa-fw"></i>example</a></li><li class="menu-item menu-item-categories"><a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>Categories</a></li><li class="menu-item menu-item-archives"><a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>Archives</a></li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>Search
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
      <div class="search-header">
        <span class="search-icon">
          <i class="fa fa-search"></i>
        </span>
        <div class="search-input-container">
          <input autocomplete="off" autocapitalize="off" maxlength="80"
                placeholder="Searching..." spellcheck="false"
                type="search" class="search-input">
        </div>
        <span class="popup-btn-close" role="button">
          <i class="fa fa-times-circle"></i>
        </span>
      </div>
      <div class="search-result-container">
        <div class="search-result-icon">
          <i class="fa fa-spinner fa-pulse fa-5x"></i>
        </div>
      </div>
    </div>
  </div>

</header>
        
  
  <aside class="sidebar">

    <div class="sidebar-inner sidebar-nav-active sidebar-toc-active">
      <ul class="sidebar-nav">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <div class="sidebar-panel-container">
        <!--noindex-->
        <div class="post-toc-wrap sidebar-panel">
            <div class="post-toc animated"><ol class="nav"><li class="nav-item nav-level-3"><a class="nav-link" href="#1-Sequence-download-and-preparation"><span class="nav-text">1. Sequence download and preparation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#2-Sequence-standardization"><span class="nav-text">2. Sequence standardization</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#3-Sequence-extraction"><span class="nav-text">3. Sequence extraction</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-Multiple-sequence-alignment"><span class="nav-text">4. Multiple sequence alignment</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-Sequence-concatenation"><span class="nav-text">5. Sequence concatenation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#6-Optimal-partitioning-strategy-and-model-selection"><span class="nav-text">6. Optimal partitioning strategy and model selection</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#7-Phylogenetic-tree-reconstruction-based-on-the-Bayesian-method-BI"><span class="nav-text">7. Phylogenetic tree reconstruction based on the Bayesian method (BI)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#8-Maximum-likelihood-method-ML-phylogenetic-tree-reconstruction"><span class="nav-text">8. Maximum likelihood method (ML) phylogenetic tree reconstruction</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#9-Flowchart"><span class="nav-text">9. Flowchart</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#10-Phylogenetic-Tree-Annotation"><span class="nav-text">10. Phylogenetic Tree Annotation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#11-Software-download-address"><span class="nav-text">11. Software download address</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#12-Reference"><span class="nav-text">12. Reference</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#1-%E5%BA%8F%E5%88%97%E4%B8%8B%E8%BD%BD%E4%B8%8E%E5%87%86%E5%A4%87"><span class="nav-text">1. 序列下载与准备</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#2-%E5%BA%8F%E5%88%97%E6%A0%87%E5%87%86%E5%8C%96"><span class="nav-text">2. 序列标准化</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#3-%E5%BA%8F%E5%88%97%E6%8F%90%E5%8F%96"><span class="nav-text">3. 序列提取</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-%E5%A4%9A%E5%BA%8F%E5%88%97%E6%AF%94%E5%AF%B9"><span class="nav-text">4. 多序列比对</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-%E5%BA%8F%E5%88%97%E4%B8%B2%E8%81%94"><span class="nav-text">5. 序列串联</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#6-%E6%9C%80%E4%BC%98%E5%88%86%E5%8C%BA%E7%AD%96%E7%95%A5%E5%92%8C%E6%A8%A1%E5%9E%8B%E9%80%89%E6%8B%A9"><span class="nav-text">6. 最优分区策略和模型选择</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#7-%E5%9F%BA%E4%BA%8E%E8%B4%9D%E5%8F%B6%E6%96%AF%E6%96%B9%E6%B3%95-BI-%E7%9A%84%E7%B3%BB%E7%BB%9F%E5%8F%91%E8%82%B2%E6%A0%91%E9%87%8D%E5%BB%BA"><span class="nav-text">7. 基于贝叶斯方法 (BI) 的系统发育树重建</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#8-%E6%9C%80%E5%A4%A7%E4%BC%BC%E7%84%B6%E6%B3%95-ML-%E7%B3%BB%E7%BB%9F%E5%8F%91%E8%82%B2%E6%A0%91%E9%87%8D%E5%BB%BA"><span class="nav-text">8. 最大似然法 (ML) 系统发育树重建</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#9-%E5%B7%A5%E4%BD%9C%E6%B5%81"><span class="nav-text">9. 工作流</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#10-%E7%B3%BB%E7%BB%9F%E5%8F%91%E8%82%B2%E6%A0%91%E6%B3%A8%E9%87%8A"><span class="nav-text">10. 系统发育树注释</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#11-%E8%BD%AF%E4%BB%B6%E4%B8%8B%E8%BD%BD%E5%9C%B0%E5%9D%80"><span class="nav-text">11. 软件下载地址</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#12-%E5%8F%82%E8%80%83%E6%96%87%E7%8C%AE"><span class="nav-text">12. 参考文献</span></a></li></ol></div>
        </div>
        <!--/noindex-->

        <div class="site-overview-wrap sidebar-panel">
          <div class="site-author animated" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">Dong Zhang</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap animated">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
        <a href="/archives/">
          <span class="site-state-item-count">8</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
          <a href="/categories/">
        <span class="site-state-item-count">1</span>
        <span class="site-state-item-name">categories</span></a>
      </div>
      <div class="site-state-item site-state-tags">
          <a href="/tags/">
        <span class="site-state-item-count">3</span>
        <span class="site-state-item-name">tags</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author animated">
      <span class="links-of-author-item">
        <a href="https://github.com/dongzhang0725/PhyloSuite" title="GitHub → https:&#x2F;&#x2F;github.com&#x2F;dongzhang0725&#x2F;PhyloSuite" rel="noopener me" target="_blank"><i class="fab fa-github fa-fw"></i>GitHub</a>
      </span>
      <span class="links-of-author-item">
        <a href="mailto:dongzhang0725@gmail.com" title="E-Mail → mailto:dongzhang0725@gmail.com" rel="noopener me" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
  </div>

        </div>
      </div>
        <div class="back-to-top animated" role="button" aria-label="Back to top">
          <i class="fa fa-arrow-up"></i>
          <span>0%</span>
        </div>
    </div>

    
  </aside>


    </div>

    <div class="main-inner post posts-expand">


  


<div class="post-block">
  
  

  <article itemscope itemtype="http://schema.org/Article" class="post-content" lang="en">
    <link itemprop="mainEntityOfPage" href="https://dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="Dong Zhang">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="PhyloSuite">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="post" itemscope itemtype="http://schema.org/CreativeWork">
      <meta itemprop="name" content="Muti-gene phylogeny | PhyloSuite">
      <meta itemprop="description" content="">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          Muti-gene phylogeny
        </h1>

        <div class="post-meta-container">
          <div class="post-meta">
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-calendar"></i>
      </span>
      <span class="post-meta-item-text">Posted on</span>

      <time title="Created: 2019-03-02 11:06:19" itemprop="dateCreated datePublished" datetime="2019-03-02T11:06:19+08:00">2019-03-02</time>
    </span>
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-calendar-check"></i>
      </span>
      <span class="post-meta-item-text">Edited on</span>
      <time title="Modified: 2025-11-25 11:09:44" itemprop="dateModified" datetime="2025-11-25T11:09:44+08:00">2025-11-25</time>
    </span>
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-folder"></i>
      </span>
      <span class="post-meta-item-text">In</span>
        <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
          <a href="/categories/PhyloSuite-demo/" itemprop="url" rel="index"><span itemprop="name">PhyloSuite demo</span></a>
        </span>
    </span>

  
</div>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody"><!-- 切换按钮 -->
<div id="lang-switcher" style="margin-bottom: 10px;">
  <button class="lang-btn" onclick="switchLang('en')">English</button>
  <button class="lang-btn" onclick="switchLang('zh')">中文</button>
</div>

<!-- 英文内容 -->
<div class="lang lang-en">

<p>PhyloSuite: from data acquisition to phylogenetic tree annotation (muti-gene dataset)</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cover.jpg?raw=true"></p>
<p>In order to make it easier for users to get started, here we briefly introduce the use of PhyloSuite using an example of all available Monogenea (class) (Platyhelminthes) mitochondrial genomes (mitogenomes).</p>
<p><span style="color:red">Author’s Note: </span>To save space, the contents duplicated with our <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single-gene tutorial</a > will not be detailed in this article.</p>
<span id="more"></span>

<h3 id="1-Sequence-download-and-preparation"><a href="#1-Sequence-download-and-preparation" class="headerlink" title="1. Sequence download and preparation"></a>1. Sequence download and preparation</h3><p>Since PhyloSuite is designed both for experts and students, we will briefly introduce how to search for sequences in GenBank. We will search for all available mitogenome sequences of the class Monogenea, but this method can be applied to any other taxonomic group. After entering the NCBI’s main website (https://www.ncbi.nlm.nih.gov/), follow the steps below:</p>
<ol>
<li>Select the Nucleotide database;</li>
<li>Enter <code>Monogenea[ORGN] AND (mitochondrion[TITL] OR mitochondrial[TITL]) AND 10000:50000[SLEN]</code> into the search box. This search comprises 3 parts: first we define the target taxonomic group  (<code>Monogenea[ORGN]</code>), then define sequence type <code>(mitochondrion[TITL] OR mitochondrial[TITL])</code>, and finally predefine the length range of the sequence (<code>10000:50000[SLEN]</code>). These keywords can be adjusted according to your needs;</li>
<li>Click <code>Send to</code>;</li>
<li>Select <code>Complete Record --&gt; File --&gt; GenBank</code> to save the complete mitogenomes. Alternatively, you may select just the Accession List (either via file or clipboard options).</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/download_seq_en.jpg?raw=true"></p>
<p>After the sequences&#x2F;IDs are downloaded&#x2F;copied, you can drag-drop the file directly into PhyloSuite, or click open file(s)&#x2F;input ID(s) and paste the list of accession numbers there and let PhyloSuite download the selected files.<br>1.	First select any working folder (files or flowchart);<br>2.	Drag the file containing downloaded sequences&#x2F;IDs and drop it in the area specified in the figure (display area).</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_seq_en.jpg?raw=true"></p>
<p>You can easily add other sequences to this dataset using the same methodology. Usually, you will have to add your own and outgroup sequence(s) separately. For this purpose, just drag them into the area shown in the figure. You may also add sequences via GB accession numbers, for which you can use <code>add file</code> function accessed via right-click - just paste accession number(s) into the <code>Input IDs</code> window, and PhyloSuite will download them automatically:</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_outgroup_en.jpg?raw=true"></p>
<p>When a mitochondrial genome is uploaded to the GenBank, it will be given an accession number; after verification, it will usually be given a separate RefSeq database accession number (starting with NC). Therefore, there will be duplicate mitochondrial genomes in the downloaded data. PhyloSuite can screen them using the <code>highlight identical sequences</code> function (identical sequences will be marked with same colors):<br>1.	Click the star-shaped button in the right bottom corner of the main interface, and let PhyloSuite search for duplicates;<br>2. If PhyloSuite identifies duplicates, the star-shaped icon will change into a <span style="color:purple">purple</span> brush. Clicking this button again will clean up the duplicates (by default, PhyloSuite preferentially retains the RefSeq IDs, starting with NC). In the demo dataset, this results in 34 retained unique sequences.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/highlight_identical_seq_en.jpg?raw=true"></p>
<p>The next step is to update the lineage data (taxonomy). Use <code>ctrl+A</code> to select all sequences, then right-click, choose <code>Get taxonomy (NCBI)</code> to automatically grab the lineage information from NCBI, and then manually modify the unrecognized information. See the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a> for other details.</p>
<h3 id="2-Sequence-standardization"><a href="#2-Sequence-standardization" class="headerlink" title="2. Sequence standardization"></a>2. Sequence standardization</h3><p>The function mainly has two aspects: the first is to standardize the synonymous gene names, and the second is to identify the problematic annotation features. For example, in mitogenomes, the two copies of trnS and trnL are commonly confused:<br>1.	Select all sequences, then right-click and select <code>Standardization</code>;<br>2.	A prompt will pop up to select the data type. Here you can select the predefined <code>Mitogenome</code>. You may customize the available data types (i.e., add new types or modify existent types).</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/standard_trigger_en.jpg?raw=true"></p>
<p>In the pop-up window, we find that there are 22 problematic trnS and trnL tRNA annotations in our dataset. If you click on the drop-down box, you can review them individually. If you intend to use only protein-coding genes (PCGs) in your analyses, you may ignore this problem, but if you are interested in the tRNA dataset, you should re-annotate them manually. PhyloSuite provides a convenient function for this:<br>1.	Click on each item in the drop-down box to view its annotation;<br>2.	Click on the <code>Predict tRNA (LEU and SER)</code> in the right bottom corner to conduct a semi-automatic ARWEN annotation.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/tRNA_warning_en.jpg?raw=true"></p>
<p>After clicking the Predict tRNA button, a window will pop up (see the figure below):<br>1.	Save the problematic tRNA gene sequences locally via the <code>Save to file</code> button;<br>2.	Click <code>ARWEN</code> link on the interface to open the ARWEN website. (<span style="color:red">Author’s note:</span> this sometimes does not work if you are using the Edge browser on Win10; use this link directly: <a target="_blank" rel="noopener" href="http://130.235.46.10/ARWEN/">http://130.235.46.10/ARWEN/</a>);<br>3.	Import the file saved in the first step into ARWEN;<br>4.	The first parameter will be selected by default. If it is not a mammalian mitochondrial genome, please uncheck it.<br>5.	Click <code>Submit</code>.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_en.jpg?raw=true"></p>
<p>After getting the ARWEN tRNA prediction:<br>1.	<code>ctrl+A</code> to select all results, and then copy them;<br>2.	Paste the copied result into the area shown in the figure;<br>3.	Click <code>Start</code> to start automatic annotation.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_results_en.jpg?raw=true"></p>
<p>After the ARWEN annotation, there may still be some tRNAs that are not annotated successfully. If you wish, you may annotate them manually:<br>1.	After locating the problematic annotation, I found that the problematic mitogenome had two annotated trnS2 sequences, so I changed the problematic one to S1 manually. This is a lazy way. The safest way is to use the semi-automatic step again to fetch the problematic sequences, align them both with all other homologs (trnS1 and trnS2) in the extracted results and identify them on the basis of their anti-codons and secondary structures;<br>2.	After the modification, click <code>Validate</code> to save the annotation.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/manual_tRNA_en.jpg?raw=true"></p>
<p>In addition to the above tRNA annotation issues, here are a few more examples of common errors:<br>If there are no feature annotations and sequences in the record, the best option is usually to delete it. You can delete it manually in this window (see below), but this is error-prone – if you don’t delete it cleanly, that is, from LOCUS to &#x2F;&#x2F;, the program may malfunction. It is better to delete it in the main interface (display area) via the right-click menu.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error1_en.jpg?raw=true"></p>
<p>The second common problem is that the specified feature does not contain default preset identifiers in the GenBank file. This happens mostly in the <code>gene</code> feature, because the identifiers are <code>gene</code>, <code>product</code> and <code>note</code>. In the screenshot below you can see that gene feature contains only <code>locus_tag</code> and <code>db_xref</code> identifiers. In this case, we can ignore this error, because the correct annotation is below, under the <code>tRNA</code> feature.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error2_en.jpg?raw=true"></p>
<p>In addition to checking annotations, this feature also supports automatic labeling of non-coding regions (useful for subsequent extraction and gene sequence analyses):<br>1.	Set the threshold for the non-coding region you wish to label and extract. When the number of intergeneric nucleotides between genes (the starting position of the downstream gene minus the end position of the upstream gene) is greater than this threshold, the intergenicspacer will be marked as a non-coding region (NCR);<br>2.	Click <code>Validate</code>. In the screenshot I set the threshold to 20, and 2 intergenic regions larger than 20 bp are marked.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mark_NCR_en.jpg?raw=true"></p>
<h3 id="3-Sequence-extraction"><a href="#3-Sequence-extraction" class="headerlink" title="3. Sequence extraction"></a>3. Sequence extraction</h3><ol>
<li>   In the main interface, select all sequences, right-click and choose <code>Extract</code> to enter the extraction interface;</li>
<li>   Monogeneans use the code table 9 (<code>Echinoderm and Flatworm Mitochondrial Code</code>). Selecting the right code table is necessary to correctly identify the stop codons, translate protein-coding genes, and generate an RSCU table;</li>
<li>   Select the predefined <code>Mitogenome</code> as the sequence type;</li>
<li>   Set parameters for the gene order visualization in iTOL, such as the shape, color and length of each gene icon;</li>
<li>   <code>Gene Interval</code> is used to set the spacing between gene icons in the iTOL gene order visualization;</li>
<li>   The content of <code>Lineage color</code> is explained in the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a>, so I won’t go into details here. Finally, click <code>Start</code> to start the program.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/extract_en.jpg?raw=true"></p>
<p>After extraction, the sequences corresponding to each <code>feature</code> are stored in folders named according to corresponding GenBank file features: CDS_NUC (nucleotide sequences of genes identified within the CDS; in this case 12 genes, as atp8 is missing), CDS_AA (translated amino acid sequences of the 12 genes), rRNA (2 genes) and tRNA (22 genes). Sometimes you may encounter a situation where gene names are not standardized. For detailed procedure please refer to the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a>. Briefly: open the <code>extract_results\StatFiles\name_for_unification.csv</code> file, replace the name in the <code>New Name</code> column, then save it (csv), and finally drag this file into the extraction settings interface (see screenshot below):</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/replace_name_en.jpg?raw=true"></p>
<p>Note that there may be some sequence files with ‘copy’ in the file name in the results folder; this means that there are duplicated genes in some sequences (copy2 means 2 genes, copy3 means 3, etc.). As shown in the figure (our demo dataset doesn’t have duplicated genes, this is from another dataset just for demonstration), there are two copies of trnA, trnK and trnS1 genes in some sequences (user can open fasta files of problematic sequences to view details):</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_copy_en.jpg?raw=true"></p>
<p>In addition to the extracted sequences, you may also find the gene order file (which can be used for CREx and TreeREx analyses) and various statistical tables in the results folder. Some of them can be used directly in your manuscript, and some can be used for downstream analyses (not shown in this tutorial).</p>
<h3 id="4-Multiple-sequence-alignment"><a href="#4-Multiple-sequence-alignment" class="headerlink" title="4. Multiple sequence alignment"></a>4. Multiple sequence alignment</h3><p>You can automatically input the results from the previous step into MAFFT in the following way (see the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a> for multiple ways of importing files):<br>1.	Check the <code>extract_results</code> result folder;<br>2.	Open MAFFT through the <code>Alignment</code> menu;<br>3.	<code>Extract-Seq</code> is automatically checked, the sequence extracted in the <code>extract_results</code> folder is automatically imported into MAFFT;<br>4.	Switch between <code>RNAs</code>, <code>PCGs</code> and <code>AA</code> to automatically import their corresponding sequence files.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_input_en.jpg?raw=true"></p>
<p>Next, batch alignment is started. Here, I selected the <code>Normal</code> alignment method for aligning the RNAs (22 tRNA genes + 2 rRNA genes). Ideally, the secondary structure alignment mode available in MAFFT-with-extensions should be used to align RNAs, but Windows doesn’t support it.<br>1.	Select <code>Normal</code> alignment mode;<br>2.	Other parameters can be set according to your needs, and then click <code>Start</code>;<br>3.	After the alignment is complete, click the folder icon to the right of <code>files</code>;<br>4.	Create a new folder named <code>alignments</code> within this folder;<br>5.	Copy the 24 sequences that have just been aligned to the <code>alignments</code> folder, as these sequences will be overwritten when you start the next alignment.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_en.jpg?raw=true"></p>
<p>Nucleotide sequences of the PCGs should to be aligned using the codon mode (this function was added by PhyloSuite, the procedure is that the nucleotide sequences are translated into amino acid sequences, aligned by MAFFT, and then translated back into nucleotides):<br>1.	Select PCGs to automatically import the extracted nucleotide sequence of all PCGs into MAFFT (see <code>Input</code>);<br>2.	Selected the code table (9);<br>3.	After setting other parameters, start the program.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_pcgs_en.jpg?raw=true"></p>
<p>You should get a prompt that internal stop codon was found inside a gene. If you are aware of this problem, feel free to ignore it and continue the alignment, otherwise terminate the alignment and inspect the problem:<br>1.	In the pop-up window, select <code>Yes</code> to view the problematic sequence;<br>2.	Select the stop codon number in the drop-down box in the right bottom corner to view each identified internal stop codon. For example, the stop codon TAG is marked here in sequence No. 31.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/inter_stp_codon_en.jpg?raw=true"></p>
<p>Internal termination codons are commonly caused by sequencing and annotation artefacts, but if you find a very large number of internal stop codons, you probably selected a wrong code table. If you only find a few, you can ignore or delete them (if deleting, use ctrl+s to overwrite the old file, and then continue to align). Here, I ignored the internal stop codons. When the alignment is completed, copy the alignments of the 12 protein genes into the <code>alignments</code> folder for downstream analyses.</p>
<h3 id="5-Sequence-concatenation"><a href="#5-Sequence-concatenation" class="headerlink" title="5. Sequence concatenation"></a>5. Sequence concatenation</h3><p>Since there are 36 aligned sequences, they need to be concatenated into a single dataset:<br>1.	Open the <code>Concatenation</code> window through the <code>Alignment</code> menu bar (<code>Alignment--&gt;Concatenation</code>);<br>2.	Select and drag all 36 sequences in the <code>alignments</code> folder into the file input box.<br>3.	You may adjust the gene order by dragging the files, but you must ensure that PCGs (12 in this case) are first, because they need to be broken down by codon positions for subsequent selection of the partition model.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cat_seq_drag_en.jpg?raw=true"></p>
<ol start="4">
<li>Select the format(s) of the output file;</li>
<li>The name of the output file can be customized;</li>
<li>Start the program.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/concatenation_en.jpg?raw=true"></p>
<p>Since some species do not contain all 36 genes, there is a prompt that the missing genes will be replaced with “?”. Missing genes can be viewed in the <code>missing_genes.txt</code> file in the <code>concatenate_results</code> result folder.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/missing_genes_en.jpg?raw=true"></p>
<h3 id="6-Optimal-partitioning-strategy-and-model-selection"><a href="#6-Optimal-partitioning-strategy-and-model-selection" class="headerlink" title="6. Optimal partitioning strategy and model selection"></a>6. Optimal partitioning strategy and model selection</h3><p>In the next step we will use <code>PartitionFinder2</code> to select the best-fit partitioning strategy and models for the concatenated sequences. Note that there is some functional overlap between this plug-in, ModelFinder and IQ-Tree, see the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a> for the other two tools. For more details about <code>PartitionFinder2</code> parameters, check the manual at <a target="_blank" rel="noopener" href="http://www/">http://www</a>. Robertlanfear.com&#x2F;partitionfinder&#x2F;assets&#x2F;Manual_v2.1.x.pdf.<br>1.	Select the <code>concatenate_results</code> folder;<br>2.	Open <code>Phylogeny--&gt;PartitionFinder2</code> through the menu bar, and the concatenated dataset with the position index of each gene will be automatically imported;<br>3.	Select the <code>Nucleotide</code> mode;<br>4.	Select <code>linked</code> for <code>branchlengths</code>. If unlinked, MrBayes will generate separate trees for each partition, and the analysis may not converge.<br>5.	For <code>models</code>, select <code>mrbayes</code>; this will calculate the models supported by MrBayes. The models supported by RAxML can be set by <code>--raxml</code> in <code>Command line options</code>. Currently, there is no predefined IQ-TREE support, so you can select <code>all</code>, or <code>&lt;list&gt;</code> to customize models (see screenshot below):</p>
<p>&emsp;<img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/custom_models_en.jpg?raw=true"><br>6.	<code>AICc</code> is the model selection criterion recommended by the PartitionFinder author;<br>7.	<code>search</code> method, you can choose <code>greedy</code>;<br>8.	<code>DATA BLOCKS</code> sets the position index for each gene, as sometimes we need to split the PCGs by codon positions to calculate the optimal partitioning strategy, and models for each codon site. Select 12 protein genes, then use the conversion button (shown in the screenshot) to split them by codon positions. Press the button again to revert back (<span style="color:red">Note: all the protein genes must be listed together and at the beginning of the concatenated dataset</span>);<br>9.	Other parameters can be set according to your own needs, and then start the program.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/partitionfinder_en.jpg?raw=true"></p>
<h3 id="7-Phylogenetic-tree-reconstruction-based-on-the-Bayesian-method-BI"><a href="#7-Phylogenetic-tree-reconstruction-based-on-the-Bayesian-method-BI" class="headerlink" title="7. Phylogenetic tree reconstruction based on the Bayesian method (BI)"></a>7. Phylogenetic tree reconstruction based on the Bayesian method (BI)</h3><p>Consequently, we can use the results of <code>PartitionFinder2</code> to reconstruct the phylogenetic tree using Bayesian method:<br>1.	Select the <code>PartFind_results</code> folder;<br>2.	Open the <code>Phylogeny--&gt;MrBayes</code> via the menu bar, <code>PartitionFinder2</code> results will be automatically imported;<br>3.	The sequence file in the <code>PartitionFinder2</code> result will be automatically converted to <code>NEXUS</code> format (interleave mode) as the input file for MrBayes. If it is not interleaved, there is a length limit of 99990 characters;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_length_en.jpg?raw=true"><br>4.	The outgroup can be selected through the drop-down menu (multiple outgroups can be set);<br>5.	Since we use the results of <code>PartitionFinder2</code>, the <code>Partition Models</code> mode will be automatically selected, and settings for some other model parameters, such as <code>State freq</code>, will be ignored;<br>6.	Set the number of generations;<br>7.	Play with other parameters if you wish, and then start the program.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_en.jpg?raw=true"></p>
<h3 id="8-Maximum-likelihood-method-ML-phylogenetic-tree-reconstruction"><a href="#8-Maximum-likelihood-method-ML-phylogenetic-tree-reconstruction" class="headerlink" title="8. Maximum likelihood method (ML) phylogenetic tree reconstruction"></a>8. Maximum likelihood method (ML) phylogenetic tree reconstruction</h3><p>Similar to <code>MrBayes</code>, the results of <code>PartitionFinder2</code> can be directly used by IQ-TREE:<br>1.	Select the <code>PartFind_results</code> result folder;<br>2.	Open <code>Phylogeny--&gt;IQ-TREE</code> through the menu bar. The result of <code>PartitionFinder2</code> is automatically imported;<br>3.	When using <code>PartitionFinder2</code>, the <code>Partition Mode</code> will be automatically selected, and the parameter settings in the <code>Substitution Model Options</code> section will be ignored;<br>4.	For relatively large datasets you can use the <code>Ultrafast</code> bootstrap method introduced by <code>IQ-TREE</code>, but the number of replicates should be relatively large (minimum 5000). If you choose <code>Standard</code>, it will be slower;<br>5.	Play with other parameters if you wish, and then start the program.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/IQ-TREE_en.jpg?raw=true"></p>
<h3 id="9-Flowchart"><a href="#9-Flowchart" class="headerlink" title="9. Flowchart"></a>9. Flowchart</h3><p>When you use datasets comprised of PCGs and RNAs (e.g., all 36&#x2F;37 mitochondrial genes), it is best to use different alignment modes, and the codon sites of the PCGs need to be tested (partitioning+models), so it is best to use the procedure described so far. However, if you want to reconstruct a phylogenetic tree using just PCGs (regardless of whether you wish to use amino acids or nucleotides), you can use Workflow function to <strong>complete the entire phylogenetic analysis operation in one step</strong>. The execution order of the workflow is: alignment –&gt; removal of poorly aligned sections (optional) –&gt; concatenation –&gt; partitioning and model selection –&gt; phylogeny reconstruction (Bayesian&#x2F;maximum likelihood method). You only have to select input files for the first program (MAFFT), as all downstream programs will automatically use the output of the upstream program as input file (see also the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a>):<br>1.	Click <code>Flowchart</code> in the menu bar;<br>2.	Select the desired analyses; here we will use <code>MAFFT</code>, <code>Gblocks</code>, <code>Concatenation</code>, <code>PartitionFinder</code>, <code>IQ-TREE </code>(ML) and <code>MrBayes</code> (BI);<br>3.	As stated above, you only need to input the file in the first program, so click the red <code>Input file here</code> to open the MAFFT program window;<br>4.	Click the file input box of MAFFT to view the automatically prepared input files (you may opt to use a different file via <code>No, thanks</code>);<br>5.	Select the results that you extracted in section 3 (<code>extract_results</code>);<br>6.	<code>Extract-Seq</code> option select <code>AA</code>, and the amino acid sequences will be imported automatically;<br>7.	Just close the window to save the imported files.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/flowchart_en.jpg?raw=true"></p>
<p>You may click the edit button next to each function pane (shown in the screenshot above) to set the parameters of each program, or modify the parameters in the parameter summary window (recommended), as follows (see also the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a>):<br>1.	Click the <code>Check | Start</code> button, and the parameter summary window will pop up, allowing you to check and modify the parameters;<br>2.	PhyloSuite is able to automatically check for conflicting parameters. For example, because <code>PartitionFinder</code> is used, partition mode should be checked in <code>IQ-TREE</code>. Clicking the blue ‘autocorrect’ function will automatically complete this operation;<br>3.	If you wish to edit any of the settings (blue letters), just click on them…<br>4.	…and the corresponding settings window will pop up, with the widget you wish to edit highlighted;<br>5.	After setting the parameters for each software, they will be automatically saved when you close the window;<br>6.	Click <code>Ok, start</code> to start the Flowchart.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_detect_en.jpg?raw=true"></p>
<p>You will notice that <code>MrBayes</code> reports and error. This is caused by an unrecognized character <code>B</code> in one of the sequences in our dataset, it is an annotation error in that GenBank record. Open the sequence viewer or input file, find the rogue character, and delete it:</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_error_en.jpg?raw=true"></p>
<p>After all the programs have finished, the main interface will display the summary of materials &amp; methods and references (also see the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a>).</p>
<h3 id="10-Phylogenetic-Tree-Annotation"><a href="#10-Phylogenetic-Tree-Annotation" class="headerlink" title="10. Phylogenetic Tree Annotation"></a>10. Phylogenetic Tree Annotation</h3><p>Finally, we will introduce the annotation of the phylogenetic tree, as PhyloSuite generates many files useful for this:<br>1.	First, open iTOL’s home page <a target="_blank" rel="noopener" href="https://itol.embl.de/">https://itol.embl.de/</a>, log into your account, and select <code>My Trees</code>;<br>2.	Double-click <code>MrBayes_results</code> to open the result of <code>MrBayes</code>, and drag the *.con.tre file into the area shown in the figure to import it automatically;<br>3.	Click on the tree to enter the editing interface.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_input_en.jpg?raw=true"></p>
<ol start="4">
<li>Support value can be displayed using the tab <code>Advanced</code>;</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/bootstrap_en.jpg?raw=true"></p>
<ol start="5">
<li><p>In PhyloSuite, double-click <code>extract_results</code> to open the extraction step results (section 3). Open the <code>itolFiles</code> folder, which contains files useful for the iTOL phylogram annotation. Just drag the file into iTOL interface to see what happens. Batch replacement of species names and marking taxa with colors are described in the <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">single gene tutorial</a>, so here we mainly focus on mapping gene orders to phylogenetic trees;<br>&emsp;a. Drag the <code>itol_gene_order.txt</code> file into the iTOL interface;<br>&emsp;b. In the <code>Datasets</code> tab you can play with different settings, such as <code>Label size factor</code> to control the font size of each gene name.<br>&emsp;<img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_order_en.jpg?raw=true"></p>
</li>
<li><p>Since there are many files that can be used for tree annotation, you can combine them into a variety of annotation effects. Here we demonstrate one:<br>&emsp;a. Drag <code>itol_Family_ColourRange.txt</code> and <code>itol_Family_ColourStrip.txt</code> to annotate the families;<br>&emsp;b. Close <code>color_strip</code>, as we only need this function to colorize the branches (they remain colorized even after you uncheck the strip);<br>&emsp;c. Click on the gear icon of the <code>Family text</code> to change settings;<br>&emsp;d. Move the text of the section to the left by adjusting the <code>Left margin</code> value, whereas text size can be changed via the <code>Text size factor</code>;<br>&emsp;e. Place the legend in a blank area before exporting the tree, as this makes subsequent editing easier;<br>&emsp;f. Select the <code>Export</code> tab to export the annotated phylogenetic tree (use <code>SVG</code> format if you intend to use Adobe Illustrator).</p>
</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_lcolor_en.jpg?raw=true"></p>
<p>Finally, you can use Adobe Illustrator (or other suitable software) to make final adjustments to the image, such as adjusting the position of the family name, the location of the legend, and adding gradient color blocks:</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/final_tree_en.jpg?raw=true"></p>
<p>By the way,<br>The advantages of this annotation method are especially pronounced when you have hundreds of species, or when you want to compare multiple topologies based on the same dataset. Colourated annotation is very helpful in such cases, especially for identifying paraphyly and polyphyly.</p>
<hr>

<h3 id="11-Software-download-address"><a href="#11-Software-download-address" class="headerlink" title="11. Software download address"></a>11. Software download address</h3><p><a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a> or <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link">https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link</a> (China)</p>
<h3 id="12-Reference"><a href="#12-Reference" class="headerlink" title="12. Reference"></a>12. Reference</h3><p>Zhang, D., F. Gao, I. Jakovlić, H. Zou, J. Zhang, W.X. Li, and G.T. Wang, PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Molecular Ecology Resources, 2020. 20(1): p. 348–355. DOI: 10.1111&#x2F;1755-0998.13096.</p>
</div>

<!-- 中文内容 -->
<div class="lang lang-zh" style="display:none;">
**PhyloSuite：从数据获取到系统发育树注释（多基因数据集）**

<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cover.jpg?raw=true"></p>
<p>为了使用户更容易上手，这里我们以所有可用的单殖吸虫纲（Monogenea）（扁形动物门，Platyhelminthes）线粒体基因组数据为例，简要介绍 PhyloSuite 的使用。</p>
<p><span style="color:red">作者注：</span>为节省篇幅，与我们的<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>重复的内容在本文中将不赘述。</p>
<!--more-->

<h3 id="1-序列下载与准备"><a href="#1-序列下载与准备" class="headerlink" title="1. 序列下载与准备"></a>1. 序列下载与准备</h3><p>由于 PhyloSuite 同时面向专家和学生设计，我们将简要介绍如何在 GenBank 中搜索序列。我们将搜索单殖吸虫纲所有可用的线粒体基因组序列，但此方法可应用于任何其他分类群。进入 NCBI 主网站 (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</a>) 后，按照以下步骤操作：</p>
<ol>
<li>选择 Nucleotide 数据库；</li>
<li>在搜索框中输入 <code>Monogenea[ORGN] AND (mitochondrion[TITL] OR mitochondrial[TITL]) AND 10000:50000[SLEN]</code>。此搜索包含 3 部分：首先定义目标分类群 (<code>Monogenea[ORGN]</code>)，然后定义序列类型 <code>(mitochondrion[TITL] OR mitochondrial[TITL])</code>，最后预设序列的长度范围 (<code>10000:50000[SLEN]</code>)。这些关键词可根据您的需要进行调整；</li>
<li>点击 <code>Send to</code>；</li>
<li>选择 <code>Complete Record --&gt; File --&gt; GenBank</code> 以保存完整的线粒体基因组。或者，您也可以只选择 Accession 号列表（通过文件或剪贴板选项）。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/download_seq_en.jpg?raw=true"></p>
<p>序列&#x2F;ID 下载&#x2F;复制后，您可以直接将文件拖放到 PhyloSuite 中，或者点击打开文件&#x2F;输入 ID 并将 accession 号列表粘贴到那里，让 PhyloSuite 下载选定的文件。</p>
<ol>
<li>首先选择任意工作文件夹（files 或 flowchart）；</li>
<li>将包含下载序列&#x2F;ID 的文件拖拽并放入图中指定区域（显示区域）。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_seq_en.jpg?raw=true"></p>
<p>您可以使用相同的方法轻松地将其他序列添加到此数据集中。通常，您需要分别添加您自己的序列和外群序列。为此，只需将它们拖入图中所示区域。您也可以通过 GB accession 号添加序列，可以使用右键菜单中的 <code>add file</code> 功能——只需将 accession 号粘贴到 <code>Input IDs</code> 窗口中，PhyloSuite 将自动下载它们：</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/load_outgroup_en.jpg?raw=true"></p>
<p>当一个线粒体基因组上传到 GenBank 时，它会获得一个 accession 号；经过验证后，通常会被赋予一个单独的 RefSeq 数据库 accession 号（以 NC 开头）。因此，下载的数据中会存在重复的线粒体基因组。PhyloSuite 可以使用 <code>highlight identical sequences</code> 功能筛选它们（相同序列将被标记为相同颜色）：</p>
<ol>
<li>点击主界面右下角的星形按钮，让 PhyloSuite 搜索重复项；</li>
<li>如果 PhyloSuite 识别出重复项，星形图标将变为<span style="color:purple">紫色</span>刷子。再次点击此按钮将清理重复项（默认情况下，PhyloSuite 优先保留以 NC 开头的 RefSeq ID）。在演示数据集中，这导致保留了 34 条唯一的序列。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/highlight_identical_seq_en.jpg?raw=true"></p>
<p>下一步是更新谱系数据（分类学）。使用 <code>ctrl+A</code> 选择所有序列，然后右键单击，选择 <code>Get taxonomy (NCBI)</code> 以自动从 NCBI 抓取谱系信息，然后手动修改未识别的信息。其他详细信息请参阅<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>。</p>
<h3 id="2-序列标准化"><a href="#2-序列标准化" class="headerlink" title="2. 序列标准化"></a>2. 序列标准化</h3><p>此功能主要有两个方面：一是标准化同义词基因名称，二是识别有问题的注释特征。例如，在线粒体基因组中，trnS 和 trnL 的两个拷贝经常被混淆：</p>
<ol>
<li>选择所有序列，然后右键单击并选择 <code>Standardization</code>；</li>
<li>会弹出一个提示选择数据类型的窗口。这里您可以选择预定义的 <code>Mitogenome</code>。您可以自定义可用的数据类型（即添加新类型或修改现有类型）。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/standard_trigger_en.jpg?raw=true"></p>
<p>在弹出窗口中，我们发现数据集中有 22 个有问题的 trnS 和 trnL tRNA 注释。如果您点击下拉框，可以逐个查看它们。如果您打算在分析中仅使用蛋白质编码基因（PCGs），可以忽略此问题，但如果您对 tRNA 数据集感兴趣，则应手动重新注释它们。PhyloSuite 为此提供了一个便捷的功能：</p>
<ol>
<li>点击下拉框中的每个项目查看其注释；</li>
<li>点击右下角的 <code>Predict tRNA (LEU and SER)</code> 进行半自动的 ARWEN 注释。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/tRNA_warning_en.jpg?raw=true"></p>
<p>点击 Predict tRNA 按钮后，将弹出一个窗口（见下图）：</p>
<ol>
<li>通过 <code>Save to file</code> 按钮将有问题的 tRNA 基因序列本地保存；</li>
<li>点击界面上的 <code>ARWEN</code> 链接打开 ARWEN 网站。（<span style="color:red">作者注：</span>如果您使用的是 Win10 上的 Edge 浏览器，有时这可能不起作用；请直接使用此链接：<a target="_blank" rel="noopener" href="http://130.235.46.10/ARWEN/%EF%BC%89%EF%BC%9B">http://130.235.46.10/ARWEN/）；</a></li>
<li>将第一步保存的文件导入 ARWEN；</li>
<li>第一个参数将默认被选中。如果不是哺乳动物线粒体基因组，请取消选中它。</li>
<li>点击 <code>Submit</code>。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_en.jpg?raw=true"></p>
<p>获得 ARWEN 的 tRNA 预测结果后：</p>
<ol>
<li><code>ctrl+A</code> 选择所有结果，然后复制它们；</li>
<li>将复制的结果粘贴到图中所示区域；</li>
<li>点击 <code>Start</code> 开始自动注释。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/arwen_results_en.jpg?raw=true"></p>
<p>ARWEN 注释后，可能仍有一些 tRNA 未能成功注释。如果您愿意，可以手动注释它们：</p>
<ol>
<li>定位到有问题的注释后，我发现有问题的线粒体基因组有两个注释为 trnS2 的序列，因此我手动将有问题的一个改为 S1。这是一种取巧的方法。最安全的方法是再次使用半自动步骤获取有问题的序列，将它们与提取结果中所有其他同源物（trnS1 和 trnS2）进行比对，并根据其反密码子和二级结构进行识别；</li>
<li>修改后，点击 <code>Validate</code> 保存注释。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/manual_tRNA_en.jpg?raw=true"></p>
<p>除了上述 tRNA 注释问题，这里还有几个常见错误的例子：<br>如果记录中没有特征注释和序列，通常最好的选择是删除它。您可以在此窗口中手动删除它（见下文），但这容易出错——如果您没有彻底删除它，即从 LOCUS 到 &#x2F;&#x2F;，程序可能会出错。最好通过主界面（显示区域）的右键菜单删除它。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error1_en.jpg?raw=true"></p>
<p>第二个常见问题是指定的特征在 GenBank 文件中不包含默认预设的标识符。这主要发生在 <code>gene</code> 特征中，因为标识符是 <code>gene</code>、<code>product</code> 和 <code>note</code>。在下面的屏幕截图中，您可以看到 gene 特征仅包含 <code>locus_tag</code> 和 <code>db_xref</code> 标识符。在这种情况下，我们可以忽略此错误，因为正确的注释在下面的 <code>tRNA</code> 特征下。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gb_error2_en.jpg?raw=true"></p>
<p>除了检查注释，此功能还支持自动标记非编码区域（对后续提取和基因序列分析有用）：</p>
<ol>
<li>设置您希望标记和提取的非编码区域的阈值。当基因间的核苷酸数量（下游基因的起始位置减去上游基因的结束位置）大于此阈值时，基因间间隔区将被标记为非编码区（NCR）；</li>
<li>点击 <code>Validate</code>。在截图中我将阈值设置为 20，结果标记了 2 个大于 20 bp 的基因间区域。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mark_NCR_en.jpg?raw=true"></p>
<h3 id="3-序列提取"><a href="#3-序列提取" class="headerlink" title="3. 序列提取"></a>3. 序列提取</h3><ol>
<li>在主界面中，选择所有序列，右键单击并选择 <code>Extract</code> 进入提取界面；</li>
<li>单殖吸虫使用密码子表 9 (<code>Echinoderm and Flatworm Mitochondrial Code</code>)。选择正确的密码子表对于正确识别终止密码子、翻译蛋白质编码基因以及生成 RSCU 表是必要的；</li>
<li>选择预定义的 <code>Mitogenome</code> 作为序列类型；</li>
<li>设置 iTOL 中基因顺序可视化的参数，例如每个基因图标的形状、颜色和长度；</li>
<li><code>Gene Interval</code> 用于设置 iTOL 基因顺序可视化中基因图标之间的间距；</li>
<li><code>Lineage color</code> 的内容在<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>中已有说明，此处不再赘述。最后，点击 <code>Start</code> 开始程序。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/extract_en.jpg?raw=true"></p>
<p>提取后，每个 <code>feature</code> 对应的序列存储在根据相应 GenBank 文件特征命名的文件夹中：CDS_NUC（在 CDS 内识别的基因的核苷酸序列；本例中为 12 个基因，因为缺少 atp8）、CDS_AA（12 个基因的翻译氨基酸序列）、rRNA（2 个基因）和 tRNA（22 个基因）。有时您可能会遇到基因名称不标准化的情况。详细步骤请参考<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>。简而言之：打开 <code>extract_results\StatFiles\name_for_unification.csv</code> 文件，替换 <code>New Name</code> 列中的名称，然后保存它（csv 格式），最后将此文件拖放到提取设置界面中（见下图截图）：</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/replace_name_en.jpg?raw=true"></p>
<p>请注意，结果文件夹中可能有一些文件名中包含 ‘copy’ 的序列文件；这意味着某些序列中存在重复的基因（copy2 表示 2 个基因，copy3 表示 3 个，等等）。如图所示（我们的演示数据集没有重复基因，这来自另一个仅用于演示的数据集），某些序列中存在 trnA、trnK 和 trnS1 基因的两个拷贝（用户可以打开有问题的序列的 fasta 文件查看详情）：</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_copy_en.jpg?raw=true"></p>
<p>除了提取的序列，您还可以在结果文件夹中找到基因顺序文件（可用于 CREx 和 TreeREx 分析）和各种统计表。其中一些可以直接用于您的稿件，另一些可用于下游分析（本教程未展示）。</p>
<h3 id="4-多序列比对"><a href="#4-多序列比对" class="headerlink" title="4. 多序列比对"></a>4. 多序列比对</h3><p>您可以通过以下方式将上一步的结果自动输入到 MAFFT 中（关于导入文件的多种方式，请参阅<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>）：</p>
<ol>
<li>勾选 <code>extract_results</code> 结果文件夹；</li>
<li>通过 <code>Alignment</code> 菜单打开 MAFFT；</li>
<li><code>Extract-Seq</code> 被自动勾选，在 <code>extract_results</code> 文件夹中提取的序列被自动导入到 MAFFT；</li>
<li>在 <code>RNAs</code>、<code>PCGs</code> 和 <code>AA</code> 之间切换以自动导入它们对应的序列文件。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_input_en.jpg?raw=true"></p>
<p>接下来，开始批量比对。这里，我选择了 <code>Normal</code> 比对方法来比对 RNAs（22 个 tRNA 基因 + 2 个 rRNA 基因）。理想情况下，应使用 MAFFT-with-extensions 中可用的二级结构比对模式来比对 RNAs，但 Windows 不支持它。</p>
<ol>
<li>选择 <code>Normal</code> 比对模式；</li>
<li>其他参数可根据需要设置，然后点击 <code>Start</code>；</li>
<li>比对完成后，点击 <code>files</code> 右侧的文件夹图标；</li>
<li>在此文件夹内创建一个名为 <code>alignments</code> 的新文件夹；</li>
<li>将刚刚比对完的 24 个序列复制到 <code>alignments</code> 文件夹中，因为这些序列在您开始下一次比对时会被覆盖。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_en.jpg?raw=true"></p>
<p>PCGs 的核苷酸序列应使用密码子模式进行比对（此功能由 PhyloSuite 添加，流程是将核苷酸序列翻译成氨基酸序列，通过 MAFFT 进行比对，然后翻译回核苷酸）：</p>
<ol>
<li>选择 PCGs 以自动将所有 PCGs 提取的核苷酸序列导入 MAFFT（见 <code>Input</code>）；</li>
<li>选择密码子表（9）；</li>
<li>设置其他参数后，启动程序。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mafft_pcgs_en.jpg?raw=true"></p>
<p>您应该会收到一个提示，说在基因内部发现了内部终止密码子。如果您了解此问题，可以忽略它并继续比对（选择‘Ignore’），否则终止比对并检查问题（选择‘Yes’）。<code>-adjustdirection</code> 可以自动调整某些序列的方向（即反向互补）。其他参数也可用，例如 <code>align strategy</code>、<code>export format</code> 和 <code>thread</code> 等。</p>
<p>输入文件并设置参数后，您可以点击 <code>Start</code> 按钮启动程序。可以通过 <code>Show log</code> 按钮查看运行日志。程序完成后，<code>MAFFT</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-序列串联"><a href="#5-序列串联" class="headerlink" title="5. 序列串联"></a>5. 序列串联</h3><p>由于有 36 个比对好的序列，需要将它们串联成一个数据集：</p>
<ol>
<li>通过 <code>Alignment</code> 菜单栏打开 <code>Concatenation</code> 窗口 (<code>Alignment--&gt;Concatenation</code>)；</li>
<li>选择并将 <code>alignments</code> 文件夹中的所有 36 个序列拖放到文件输入框中。</li>
<li>您可以通过拖拽文件来调整基因顺序，但必须确保 PCGs（本例中为 12 个）在前，因为它们需要按密码子位置分解以供后续选择分区模型。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/cat_seq_drag_en.jpg?raw=true"></p>
<ol start="4">
<li>选择输出文件的格式；</li>
<li>输出文件的名称可以自定义；</li>
<li>启动程序。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/concatenation_en.jpg?raw=true"></p>
<p>由于某些物种不包含全部 36 个基因，会有一个提示，说明缺失的基因将被 “?” 替换。缺失的基因可以在 <code>concatenate_results</code> 结果文件夹中的 <code>missing_genes.txt</code> 文件中查看。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/missing_genes_en.jpg?raw=true"></p>
<h3 id="6-最优分区策略和模型选择"><a href="#6-最优分区策略和模型选择" class="headerlink" title="6. 最优分区策略和模型选择"></a>6. 最优分区策略和模型选择</h3><p>下一步我们将使用 <code>PartitionFinder2</code> 为串联序列选择最佳拟合的分区策略和模型。请注意，此插件与 ModelFinder 和 IQ-Tree 之间存在一些功能重叠，关于另外两个工具请参阅<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>。有关 <code>PartitionFinder2</code> 参数的更多详细信息，请查看 <a target="_blank" rel="noopener" href="http://www/">http://www</a>. Robertlanfear.com&#x2F;partitionfinder&#x2F;assets&#x2F;Manual_v2.1.x.pdf 的手册。</p>
<ol>
<li>选择 <code>concatenate_results</code> 文件夹；</li>
<li>通过菜单栏打开 <code>Phylogeny--&gt;PartitionFinder2</code>，带有每个基因位置索引的串联数据集将自动导入；</li>
<li>选择 <code>Nucleotide</code> 模式；</li>
<li>为 <code>branchlengths</code> 选择 <code>linked</code>。如果选择 unlinked，MrBayes 将为每个分区生成单独的树，并且分析可能无法收敛。</li>
<li>对于 <code>models</code>，选择 <code>mrbayes</code>；这将计算 MrBayes 支持的模型。RAxML 支持的模型可以通过在 <code>Command line options</code> 中设置 <code>--raxml</code> 来指定。目前，没有预定义的 IQ-TREE 支持，因此您可以选择 <code>all</code>，或选择 <code>&lt;list&gt;</code> 来自定义模型（见下图截图）：</li>
</ol>
<p>&emsp;<img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/custom_models_en.jpg?raw=true"><br>6.  <code>AICc</code> 是 PartitionFinder 作者推荐的模型选择标准；<br>7.  <code>search</code> 方法，您可以选择 <code>greedy</code>；<br>8.  <code>DATA BLOCKS</code> 设置每个基因的位置索引，因为有时我们需要按密码子位置拆分 PCGs 以计算最优分区策略和每个密码子位点的模型。选择 12 个蛋白质基因，然后使用转换按钮（见截图所示）按密码子位置拆分它们。再次按下按钮可恢复原状（<span style="color:red">注意：所有蛋白质基因必须列在一起并位于串联数据集的开头</span>）；<br>9.  其他参数可根据您自己的需要设置，然后启动程序。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/partitionfinder_en.jpg?raw=true"></p>
<h3 id="7-基于贝叶斯方法-BI-的系统发育树重建"><a href="#7-基于贝叶斯方法-BI-的系统发育树重建" class="headerlink" title="7. 基于贝叶斯方法 (BI) 的系统发育树重建"></a>7. 基于贝叶斯方法 (BI) 的系统发育树重建</h3><p>因此，我们可以使用 <code>PartitionFinder2</code> 的结果通过贝叶斯方法重建系统发育树：</p>
<ol>
<li>选择 <code>PartFind_results</code> 文件夹；</li>
<li>通过菜单栏打开 <code>Phylogeny--&gt;MrBayes</code>，<code>PartitionFinder2</code> 的结果将自动导入；</li>
<li><code>PartitionFinder2</code> 结果中的序列文件将自动转换为 <code>NEXUS</code> 格式（交错模式）作为 MrBayes 的输入文件。如果不是交错模式，则有 99990 个字符的长度限制；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_length_en.jpg?raw=true"></li>
<li>外群可以通过下拉菜单选择（可以设置多个外群）；</li>
<li>由于我们使用 <code>PartitionFinder2</code> 的结果，将自动选择 <code>Partition Models</code> 模式，并且将忽略一些其他模型参数的设置，例如 <code>State freq</code>；</li>
<li>设置代数；</li>
<li>如果您愿意，可以调整其他参数，然后启动程序。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_en.jpg?raw=true"></p>
<h3 id="8-最大似然法-ML-系统发育树重建"><a href="#8-最大似然法-ML-系统发育树重建" class="headerlink" title="8. 最大似然法 (ML) 系统发育树重建"></a>8. 最大似然法 (ML) 系统发育树重建</h3><p>与 <code>MrBayes</code> 类似，<code>PartitionFinder2</code> 的结果可以直接被 IQ-TREE 使用：</p>
<ol>
<li>选择 <code>PartFind_results</code> 结果文件夹；</li>
<li>通过菜单栏打开 <code>Phylogeny--&gt;IQ-TREE</code>。<code>PartitionFinder2</code> 的结果自动导入；</li>
<li>使用 <code>PartitionFinder2</code> 时，将自动选择 <code>Partition Mode</code>，并且将忽略 <code>Substitution Model Options</code> 部分中的参数设置；</li>
<li>对于相对较大的数据集，您可以使用 <code>IQ-TREE</code> 引入的 <code>Ultrafast</code> 自举法，但重复次数应相对较大（最少 5000）。如果选择 <code>Standard</code>，速度会较慢；</li>
<li>如果您愿意，可以调整其他参数，然后启动程序。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/IQ-TREE_en.jpg?raw=true"></p>
<h3 id="9-工作流"><a href="#9-工作流" class="headerlink" title="9. 工作流"></a>9. 工作流</h3><p>当您使用包含 PCGs 和 RNAs 的数据集（例如，所有 36&#x2F;37 个线粒体基因）时，最好使用不同的比对模式，并且 PCGs 的密码子位点需要测试（分区+模型），因此最好使用到目前为止描述的程序。但是，如果您只想使用 PCGs（无论您想使用氨基酸还是核苷酸）重建系统发育树，您可以使用工作流功能 <strong>一步完成整个系统发育分析操作</strong>。工作流的执行顺序是：比对 –&gt; 去除比对不佳的区域（可选）–&gt; 串联 –&gt; 分区和模型选择 –&gt; 系统发育重建（贝叶斯&#x2F;最大似然法）。您只需要为第一个程序（MAFFT）选择输入文件，因为所有下游程序将自动使用上游程序的输出作为输入文件（另请参阅<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>）：</p>
<ol>
<li>点击菜单栏中的 <code>Flowchart</code>；</li>
<li>选择所需的分析；这里我们将使用 <code>MAFFT</code>、<code>Gblocks</code>、<code>Concatenation</code>、<code>PartitionFinder</code>、<code>IQ-TREE </code>(ML) 和 <code>MrBayes</code> (BI)；</li>
<li>如上所述，您只需要在第一个程序中输入文件，因此点击红色的 <code>Input file here</code> 打开 MAFFT 程序窗口；</li>
<li>点击 MAFFT 的文件输入框查看自动准备好的输入文件（您可以选择通过 <code>No, thanks</code> 使用不同的文件）；</li>
<li>选择您在第 3 节中提取的结果 (<code>extract_results</code>)；</li>
<li><code>Extract-Seq</code> 选项选择 <code>AA</code>，氨基酸序列将自动导入；</li>
<li>只需关闭窗口即可保存导入的文件。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/flowchart_en.jpg?raw=true"></p>
<p>您可以点击每个功能窗格旁边的编辑按钮（见上图截图）来设置每个程序的参数，或者在参数汇总窗口中修改参数（推荐），如下所示（另请参阅<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>）：</p>
<ol>
<li>点击 <code>Check | Start</code> 按钮，将弹出参数汇总窗口，允许您检查和修改参数；</li>
<li>PhyloSuite 能够自动检查冲突参数。例如，因为使用了 <code>PartitionFinder</code>，所以在 <code>IQ-TREE</code> 中应勾选分区模式。点击蓝色的 ‘autocorrect’ 功能将自动完成此操作；</li>
<li>如果您想编辑任何设置（蓝色字母），只需点击它们……</li>
<li>……相应的设置窗口将弹出，并且您希望编辑的小部件将高亮显示；</li>
<li>为每个软件设置参数后，关闭窗口时将自动保存；</li>
<li>点击 <code>Ok, start</code> 启动工作流。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/auto_detect_en.jpg?raw=true"></p>
<p>您会注意到 <code>MrBayes</code> 报告了一个错误。这是由于我们数据集中一条序列中存在无法识别的字符 <code>B</code> 引起的，这是该 GenBank 记录中的注释错误。打开序列查看器或输入文件，找到有问题的字符并将其删除：</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/mrbayes_error_en.jpg?raw=true"></p>
<p>所有程序完成后，主界面将显示材料与方法及参考文献的摘要（另请参阅<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>）。</p>
<h3 id="10-系统发育树注释"><a href="#10-系统发育树注释" class="headerlink" title="10. 系统发育树注释"></a>10. 系统发育树注释</h3><p>最后，我们将介绍系统发育树的注释，因为 PhyloSuite 生成了许多对此有用的文件：</p>
<ol>
<li>首先，打开 iTOL 的主页 <a target="_blank" rel="noopener" href="https://itol.embl.de/%EF%BC%8C%E7%99%BB%E5%BD%95%E6%82%A8%E7%9A%84%E8%B4%A6%E6%88%B7%EF%BC%8C%E7%84%B6%E5%90%8E%E9%80%89%E6%8B%A9">https://itol.embl.de/，登录您的账户，然后选择</a> <code>My Trees</code>；</li>
<li>双击 <code>MrBayes_results</code> 打开 <code>MrBayes</code> 的结果，并将 *.con.tre 文件拖入图中所示区域以自动导入；</li>
<li>点击树进入编辑界面。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_input_en.jpg?raw=true"></p>
<ol start="4">
<li>支持值可以使用 <code>Advanced</code> 选项卡显示；</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/bootstrap_en.jpg?raw=true"></p>
<ol start="5">
<li><p>在 PhyloSuite 中，双击 <code>extract_results</code> 打开提取步骤的结果（第 3 节）。打开 <code>itolFiles</code> 文件夹，其中包含对 iTOL 系统发育图注释有用的文件。只需将文件拖入 iTOL 界面即可查看效果。批量替换物种名称和用颜色标记分类群在<a href="https://dongzhang0725.github.io/dongzhang0725.github.io/single-gene-phylogeny/">单基因教程</a>中已有描述，因此这里我们主要关注将基因顺序映射到系统发育树；<br>&emsp;a. 将 <code>itol_gene_order.txt</code> 文件拖入 iTOL 界面；<br>&emsp;b. 在 <code>Datasets</code> 选项卡中，您可以调整不同的设置，例如 <code>Label size factor</code> 来控制每个基因名称的字体大小。<br>&emsp;<img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/gene_order_en.jpg?raw=true"></p>
</li>
<li><p>由于有许多文件可用于树注释，您可以将它们组合成多种注释效果。这里我们演示一种：<br>&emsp;a. 拖入 <code>itol_Family_ColourRange.txt</code> 和 <code>itol_Family_ColourStrip.txt</code> 来注释科；<br>&emsp;b. 关闭 <code>color_strip</code>，因为我们只需要此功能来为分支着色（即使您取消勾选该条带，它们仍保持着色状态）；<br>&emsp;c. 点击 <code>Family text</code> 的齿轮图标更改设置；<br>&emsp;d. 通过调整 <code>Left margin</code> 值将分段的文本向左移动，而文本大小可以通过 <code>Text size factor</code> 更改；<br>&emsp;e. 在导出树之前将图例放在空白区域，这样便于后续编辑；<br>&emsp;f. 选择 <code>Export</code> 选项卡导出注释后的系统发育树（如果您打算使用 Adobe Illustrator，请使用 <code>SVG</code> 格式）。</p>
</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/itol_lcolor_en.jpg?raw=true"></p>
<p>最后，您可以使用 Adobe Illustrator（或其他合适的软件）对图像进行最终调整，例如调整科名称的位置、图例的位置以及添加渐变颜色块：</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/muti_gene_tutorial/final_tree_en.jpg?raw=true"></p>
<p>顺便提一下，<br>当您有数百个物种时，或者当您想比较基于同一数据集的多个拓扑结构时，这种注释方法的优势尤其明显。彩色注释在这种情况下非常有帮助，特别是对于识别并系群和多系群。</p>
<hr>

<h3 id="11-软件下载地址"><a href="#11-软件下载地址" class="headerlink" title="11. 软件下载地址"></a>11. 软件下载地址</h3><p><a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a> 或 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link">https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link</a> (中国)</p>
<h3 id="12-参考文献"><a href="#12-参考文献" class="headerlink" title="12. 参考文献"></a>12. 参考文献</h3><p>Zhang, D., F. Gao, I. Jakovlić, H. Zou, J. Zhang, W.X. Li, and G.T. Wang, PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Molecular Ecology Resources, 2020. 20(1): p. 348–355. DOI: 10.1111&#x2F;1755-0998.13096.</p>
</div>

<!-- JS 切换逻辑 + 按钮高亮 + 记忆用户选择 -->
<script>
document.addEventListener('DOMContentLoaded', function() {
  const buttons = document.querySelectorAll('.lang-btn');
  const langContents = document.querySelectorAll('.lang');

  // 切换语言函数
  function switchLang(lang){
    // 隐藏所有内容
    langContents.forEach(el => el.style.display = 'none');
    // 显示对应语言
    document.querySelectorAll('.lang-' + lang).forEach(el => el.style.display = 'block');
    
    // 高亮按钮
    buttons.forEach(btn => btn.classList.remove('active'));
    document.querySelector(`.lang-btn[onclick="switchLang('${lang}')"]`).classList.add('active');
    
    // 保存选择到 localStorage
    localStorage.setItem('preferredLang', lang);
  }

  // 页面加载时读取用户上次选择
  const savedLang = localStorage.getItem('preferredLang') || 'zh';
  switchLang(savedLang);

  // 给按钮绑定点击事件
  buttons.forEach(btn => {
    btn.addEventListener('click', () => {
      const lang = btn.getAttribute('onclick').match(/switchLang\('(\w+)'\)/)[1];
      switchLang(lang);
    });
  });
});
</script>

<!-- CSS 美化按钮 -->
<style>
#lang-switcher {
  display: flex;
  gap: 10px;
}

.lang-btn {
  padding: 6px 14px;
  border: 1px solid #888;
  border-radius: 6px;
  background-color: #f5f5f5;
  cursor: pointer;
  transition: 0.2s;
  font-weight: bold;
}

.lang-btn:hover {
  background-color: #e0e0e0;
}

.lang-btn.active {
  background-color: #007bff;
  color: white;
  border-color: #007bff;
}
</style>
    </div>

    
    
    

    <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/PyQT5/" rel="tag"># PyQT5</a>
              <a href="/tags/Python/" rel="tag"># Python</a>
              <a href="/tags/Bioinformatics/" rel="tag"># Bioinformatics</a>
          </div>

        

          <div class="post-nav">
            <div class="post-nav-item">
                <a href="/PhyloSuite-demo/single-gene-phylogeny/" rel="prev" title="Single gene phylogeny">
                  <i class="fa fa-angle-left"></i> Single gene phylogeny
                </a>
            </div>
            <div class="post-nav-item">
                <a href="/PhyloSuite-demo/customize_extraction/" rel="next" title="Customizing the extraction">
                  Customizing the extraction <i class="fa fa-angle-right"></i>
                </a>
            </div>
          </div>
    </footer>
  </article>
</div>






    <div class="comments gitalk-container"></div>
</div>
  </main>

  <footer class="footer">
    <div class="footer-inner">

  <div class="copyright">
    &copy; 
    <span itemprop="copyrightYear">2025</span>
    <span class="with-love">
      <i class="user"></i>
    </span>
    <span class="author" itemprop="copyrightHolder">Dong Zhang</span>
  </div>
  <div class="powered-by">Powered by <a href="https://hexo.io/" rel="noopener" target="_blank">Hexo</a> & <a href="https://theme-next.js.org/" rel="noopener" target="_blank">NexT.Gemini</a>
  </div>

    </div>
  </footer>

  
  <div class="toggle sidebar-toggle" role="button">
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
  </div>
  <div class="sidebar-dimmer"></div>
  <div class="reading-progress-bar"></div>

<noscript>
  <div class="noscript-warning">Theme NexT works best with JavaScript enabled</div>
</noscript>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/gitalk/1.8.0/gitalk.css" integrity="sha256-AJnUHL7dBv6PGaeyPQJcgQPDjt/Hn/PvYZde1iqfp8U=" crossorigin="anonymous">

<script class="next-config" data-name="gitalk" type="application/json">{"enable":true,"github_id":"dongzhang0725","repo":"PhyloSuite","client_id":"55e7b2bd2c9d294158bf","client_secret":"b0716af44800887bfab50497c5e0309525a353b8","admin_user":"dongzhang0725","distraction_free_mode":true,"proxy":"https://cors-anywhere.azm.workers.dev/https://github.com/login/oauth/access_token","language":null,"js":{"url":"https://cdnjs.cloudflare.com/ajax/libs/gitalk/1.8.0/gitalk.min.js","integrity":"sha256-MVK9MGD/XJaGyIghSVrONSnoXoGh3IFxLw0zfvzpxR4="},"path_md5":"f7fdea4908b68be2052819530c536efb"}</script>
<script src="/js/third-party/comments/gitalk.js" defer></script>

</body>
</html>
