<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width">
<meta name="theme-color" content="#222"><meta name="generator" content="Hexo 7.3.0">

  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/apple-touch-icon-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">



<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/7.0.0/css/all.min.css" integrity="sha256-VHqXKFhhMxcpubYf9xiWdCiojEbY9NexQ4jh8AxbvcM=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.1.1/animate.min.css" integrity="sha256-PR7ttpcvz8qrF57fur/yAx1qXMFJeJFiA6pSzWi0OIE=" crossorigin="anonymous">

<script class="next-config" data-name="main" type="application/json">{"hostname":"dongzhang0725.github.io","root":"/","images":"/images","scheme":"Gemini","darkmode":false,"version":"8.25.0","exturl":false,"sidebar":{"position":"left","width_expanded":320,"width_dual_column":240,"display":"post","padding":18,"offset":12},"hljswrap":true,"codeblock":{"theme":{"light":"default","dark":"stackoverflow-dark"},"prism":{"light":"prism","dark":"prism-dark"},"copy_button":{"enable":false,"style":null},"fold":{"enable":false,"height":500},"language":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"stickytabs":false,"motion":{"enable":true,"async":false,"duration":200,"transition":{"menu_item":"fadeInDown","post_block":"fadeIn","post_header":"fadeInDown","post_body":"fadeInDown","coll_header":"fadeInLeft","sidebar":"fadeInUp"}},"i18n":{"placeholder":"Searching...","empty":"We didn't find any results for the search: ${query}","hits_time":"${hits} results found in ${time} ms","hits":"${hits} results found"},"path":"/search.xml","localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false}}</script><script src="/js/config.js" defer></script>

    <meta name="description" content="English   中文      PhyloSuite: from data aquisition to phylogenetic tree annotation (single gene tree building)">
<meta property="og:type" content="article">
<meta property="og:title" content="Single gene phylogeny">
<meta property="og:url" content="https://dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/index.html">
<meta property="og:site_name" content="PhyloSuite">
<meta property="og:description" content="English   中文      PhyloSuite: from data aquisition to phylogenetic tree annotation (single gene tree building)">
<meta property="og:locale" content="en_US">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/cover.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/download_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_lineages_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_lineages_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/extract_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/rRNA_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/name_for_unification_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modify_replace_names_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/default_extract_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_load_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_prepare_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mafft_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modelfinder_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mrbayes_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/IQ-TREE_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_set_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_running_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_summary_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_input_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/drag_gb_label_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_color_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/cover.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/download_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_lineages_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_lineages_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/extract_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/rRNA_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/name_for_unification_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modify_replace_names_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/default_extract_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_load_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_prepare_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mafft_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modelfinder_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mrbayes_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/IQ-TREE_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_set_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_running_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_summary_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_input_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/drag_gb_label_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_color_en.jpg?raw=true">
<meta property="article:published_time" content="2019-03-02T03:06:19.000Z">
<meta property="article:modified_time" content="2025-11-25T05:20:09.288Z">
<meta property="article:author" content="Dong Zhang">
<meta property="article:tag" content="PyQT5">
<meta property="article:tag" content="Python">
<meta property="article:tag" content="Bioinformatics">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/cover.jpg?raw=true">


<link rel="canonical" href="https://dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">


<script class="next-config" data-name="page" type="application/json">{"sidebar":"","isHome":false,"isPost":true,"lang":"en","comments":true,"permalink":"https://dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/","path":"PhyloSuite-demo/single-gene-phylogeny/","title":"Single gene phylogeny"}</script>

<script class="next-config" data-name="calendar" type="application/json">""</script>
<title>Single gene phylogeny | PhyloSuite</title>
  








  
  <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js" integrity="sha256-XL2inqUJaslATFnHdJOi9GfQ60on8Wx1C2H8DYiN1xY=" crossorigin="anonymous" defer></script>
<script src="/js/utils.js" defer></script><script src="/js/motion.js" defer></script><script src="/js/sidebar.js" defer></script><script src="/js/next-boot.js" defer></script>

  <script src="https://cdnjs.cloudflare.com/ajax/libs/hexo-generator-searchdb/1.5.0/search.js" integrity="sha256-xFC6PJ82SL9b3WkGjFavNiA9gm5z6UBxWPiu4CYjptg=" crossorigin="anonymous" defer></script>
<script src="/js/third-party/search/local-search.js" defer></script>







  





  <noscript>
    <link rel="stylesheet" href="/css/noscript.css">
  </noscript>
</head>

<body itemscope itemtype="http://schema.org/WebPage" class="use-motion">
  <div class="headband"></div>

  <main class="main">
    <div class="column">
      <header class="header" itemscope itemtype="http://schema.org/WPHeader"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar" role="button">
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <i class="logo-line"></i>
      <p class="site-title">PhyloSuite</p>
      <i class="logo-line"></i>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger" aria-label="Search" role="button">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>



<nav class="site-nav">
  <ul class="main-menu menu"><li class="menu-item menu-item-home"><a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a></li><li class="menu-item menu-item-documentation"><a href="/documentation/" rel="section"><i class="fa fa-navicon fa-fw"></i>documentation</a></li><li class="menu-item menu-item-installation"><a href="/installation/" rel="section"><i class="fa fa-download fa-fw"></i>installation</a></li><li class="menu-item menu-item-example"><a href="/example/" rel="section"><i class="fa fa-print fa-fw"></i>example</a></li><li class="menu-item menu-item-categories"><a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>Categories</a></li><li class="menu-item menu-item-archives"><a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>Archives</a></li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>Search
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
      <div class="search-header">
        <span class="search-icon">
          <i class="fa fa-search"></i>
        </span>
        <div class="search-input-container">
          <input autocomplete="off" autocapitalize="off" maxlength="80"
                placeholder="Searching..." spellcheck="false"
                type="search" class="search-input">
        </div>
        <span class="popup-btn-close" role="button">
          <i class="fa fa-times-circle"></i>
        </span>
      </div>
      <div class="search-result-container">
        <div class="search-result-icon">
          <i class="fa fa-spinner fa-pulse fa-5x"></i>
        </div>
      </div>
    </div>
  </div>

</header>
        
  
  <aside class="sidebar">

    <div class="sidebar-inner sidebar-nav-active sidebar-toc-active">
      <ul class="sidebar-nav">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <div class="sidebar-panel-container">
        <!--noindex-->
        <div class="post-toc-wrap sidebar-panel">
            <div class="post-toc animated"><ol class="nav"><li class="nav-item nav-level-3"><a class="nav-link" href="#1-Sequence-download-and-preparation"><span class="nav-text">1. Sequence download and preparation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#2-Sequence-extraction"><span class="nav-text">2. Sequence extraction</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#3-Multiple-sequence-alignment"><span class="nav-text">3. Multiple sequence alignment</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-Nucleotide-replacement-model-selection"><span class="nav-text">4. Nucleotide replacement model selection</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-Phylogenetic-tree-reconstruction-based-on-Bayesian-Inference-BI"><span class="nav-text">5. Phylogenetic tree reconstruction based on Bayesian Inference (BI)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#6-Phylogenetic-tree-reconstruction-based-on-the-Maximum-Likelihood-method-ML"><span class="nav-text">6. Phylogenetic tree reconstruction based on the Maximum Likelihood method (ML)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#7-Flowchart"><span class="nav-text">7. Flowchart</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#8-Phylogenetic-tree-annotation"><span class="nav-text">8. Phylogenetic tree annotation</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#9-Software-download-website"><span class="nav-text">9. Software download website</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#10-Reference"><span class="nav-text">10. Reference</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#1-%E5%BA%8F%E5%88%97%E4%B8%8B%E8%BD%BD%E4%B8%8E%E5%87%86%E5%A4%87"><span class="nav-text">1. 序列下载与准备</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#2-%E5%BA%8F%E5%88%97%E6%8F%90%E5%8F%96"><span class="nav-text">2. 序列提取</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#3-%E5%A4%9A%E5%BA%8F%E5%88%97%E6%AF%94%E5%AF%B9"><span class="nav-text">3. 多序列比对</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-%E6%A0%B8%E8%8B%B7%E9%85%B8%E6%9B%BF%E4%BB%A3%E6%A8%A1%E5%9E%8B%E9%80%89%E6%8B%A9"><span class="nav-text">4. 核苷酸替代模型选择</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-%E5%9F%BA%E4%BA%8E%E8%B4%9D%E5%8F%B6%E6%96%AF%E6%8E%A8%E6%96%AD-BI-%E7%9A%84%E7%B3%BB%E7%BB%9F%E5%8F%91%E8%82%B2%E6%A0%91%E9%87%8D%E5%BB%BA"><span class="nav-text">5. 基于贝叶斯推断 (BI) 的系统发育树重建</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#6-%E5%9F%BA%E4%BA%8E%E6%9C%80%E5%A4%A7%E4%BC%BC%E7%84%B6%E6%B3%95-ML-%E7%9A%84%E7%B3%BB%E7%BB%9F%E5%8F%91%E8%82%B2%E6%A0%91%E9%87%8D%E5%BB%BA"><span class="nav-text">6. 基于最大似然法 (ML) 的系统发育树重建</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#7-%E5%B7%A5%E4%BD%9C%E6%B5%81"><span class="nav-text">7. 工作流</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#8-%E7%B3%BB%E7%BB%9F%E5%8F%91%E8%82%B2%E6%A0%91%E6%B3%A8%E9%87%8A"><span class="nav-text">8. 系统发育树注释</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#9-%E8%BD%AF%E4%BB%B6%E4%B8%8B%E8%BD%BD%E7%BD%91%E7%AB%99"><span class="nav-text">9. 软件下载网站</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#10-%E5%8F%82%E8%80%83%E6%96%87%E7%8C%AE"><span class="nav-text">10. 参考文献</span></a></li></ol></div>
        </div>
        <!--/noindex-->

        <div class="site-overview-wrap sidebar-panel">
          <div class="site-author animated" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">Dong Zhang</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap animated">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
        <a href="/archives/">
          <span class="site-state-item-count">8</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
          <a href="/categories/">
        <span class="site-state-item-count">1</span>
        <span class="site-state-item-name">categories</span></a>
      </div>
      <div class="site-state-item site-state-tags">
          <a href="/tags/">
        <span class="site-state-item-count">3</span>
        <span class="site-state-item-name">tags</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author animated">
      <span class="links-of-author-item">
        <a href="https://github.com/dongzhang0725/PhyloSuite" title="GitHub → https:&#x2F;&#x2F;github.com&#x2F;dongzhang0725&#x2F;PhyloSuite" rel="noopener me" target="_blank"><i class="fab fa-github fa-fw"></i>GitHub</a>
      </span>
      <span class="links-of-author-item">
        <a href="mailto:dongzhang0725@gmail.com" title="E-Mail → mailto:dongzhang0725@gmail.com" rel="noopener me" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
  </div>

        </div>
      </div>
        <div class="back-to-top animated" role="button" aria-label="Back to top">
          <i class="fa fa-arrow-up"></i>
          <span>0%</span>
        </div>
    </div>

    
  </aside>


    </div>

    <div class="main-inner post posts-expand">


  


<div class="post-block">
  
  

  <article itemscope itemtype="http://schema.org/Article" class="post-content" lang="en">
    <link itemprop="mainEntityOfPage" href="https://dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/avatar.gif">
      <meta itemprop="name" content="Dong Zhang">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="PhyloSuite">
      <meta itemprop="description" content="">
    </span>

    <span hidden itemprop="post" itemscope itemtype="http://schema.org/CreativeWork">
      <meta itemprop="name" content="Single gene phylogeny | PhyloSuite">
      <meta itemprop="description" content="">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          Single gene phylogeny
        </h1>

        <div class="post-meta-container">
          <div class="post-meta">
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-calendar"></i>
      </span>
      <span class="post-meta-item-text">Posted on</span>

      <time title="Created: 2019-03-02 11:06:19" itemprop="dateCreated datePublished" datetime="2019-03-02T11:06:19+08:00">2019-03-02</time>
    </span>
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-calendar-check"></i>
      </span>
      <span class="post-meta-item-text">Edited on</span>
      <time title="Modified: 2025-11-25 13:20:09" itemprop="dateModified" datetime="2025-11-25T13:20:09+08:00">2025-11-25</time>
    </span>
    <span class="post-meta-item">
      <span class="post-meta-item-icon">
        <i class="far fa-folder"></i>
      </span>
      <span class="post-meta-item-text">In</span>
        <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
          <a href="/categories/PhyloSuite-demo/" itemprop="url" rel="index"><span itemprop="name">PhyloSuite demo</span></a>
        </span>
    </span>

  
</div>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody"><!-- 切换按钮 -->
<div id="lang-switcher" style="margin-bottom: 10px;">
  <button class="lang-btn" onclick="switchLang('en')">English</button>
  <button class="lang-btn" onclick="switchLang('zh')">中文</button>
</div>

<!-- 英文内容 -->
<div class="lang lang-en">

<p>PhyloSuite: from data aquisition to phylogenetic tree annotation (single gene tree building)</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/cover.jpg?raw=true"></p>
<span id="more"></span>

<p>In order to make it easier for users to get started, here is a tutorial to briefly introduce the use of PhyloSuite, using 51 ciliate 18S sequences as an example (<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/full/10.1111/j.1550-7408.2009.00413.x ">https://onlinelibrary.wiley.com/doi/full/10.1111/j.1550-7408.2009.00413.x</a>).</p>
<h3 id="1-Sequence-download-and-preparation"><a href="#1-Sequence-download-and-preparation" class="headerlink" title="1. Sequence download and preparation"></a>1. Sequence download and preparation</h3><p>The accession numbers we will use are below (fetched from a table in the abovementioned paper):</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br></pre></td><td class="code"><pre><span class="line">AF164122 X65150 AY547546 U17356 EU286810</span><br><span class="line">AF100301 U47620 M97909 L31520 EU039903</span><br><span class="line">Z29442 U97110 EU039887 Z29438 AY007454</span><br><span class="line">AF060454 AY007445 U82204 X71134 X65149</span><br><span class="line">U57767 L26447 AF300281 AF530529 U17355</span><br><span class="line">AY331802 AY242119 AY331790 EU286812 DQ232761</span><br><span class="line">X71140 EU286811 AY331794 EU264561 EU286809</span><br><span class="line">X65151 U24248 AM292311 EU039899 DQ057347</span><br><span class="line">AF300286 EF123708 EU039896 L26446 U57765</span><br><span class="line">Z22931 AY007450 EU264560 AY378112 DQ834370</span><br><span class="line">U97112</span><br></pre></td></tr></table></figure>
<p>First, download these IDs to the PhyloSuite workspace:</p>
<ol>
<li>Select any of the work folders (here I chose ‘files’);</li>
<li>Click Open file(s)&#x2F;Input ID(s) to open the input window.</li>
<li>Copy the above IDs into the text box (spaces, line breaks, tabs, etc. are supported as separators);</li>
<li>Enter your email (tell NCBI who is downloading the sequences);</li>
<li>Click Start to download.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/download_seq_en.jpg?raw=true"></p>
<p>After the sequence is downloaded to the workspace, we find that some taxonomic groups are designated as N&#x2F;A, indicating that the taxonomic group is not recognized. Select all sequences (CTRL+A), then right-click, and select Get taxonomy (NCBI). You will get a prompt after the selection asking to confirm that you are sure that you wish to replace the current taxonomy with the taxonomic information from NCBI’s taxonomy database (this is important for subsequent phylogenetic tree annotation).</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_lineages_en.jpg?raw=true"></p>
<p>After the above steps are completed, there may still be some IDs that are not recognized, indicating their taxonomic group is not in the latest NCBI’s taxonomic classification system. You may also wish change the current names manually, in which case you can double-click the corresponding cell and enter the correct taxonomic group name.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_lineages_en.jpg?raw=true"></p>
<h3 id="2-Sequence-extraction"><a href="#2-Sequence-extraction" class="headerlink" title="2. Sequence extraction"></a>2. Sequence extraction</h3><p>Select all sequences (Ctrl+A) and open the context menu (right-click).</p>
<ol>
<li>Choose Extract</li>
<li>Since we are extracting 18S sequences, we select 18S here;</li>
<li>Choose the name-type you wish to use for each sequence: organism, ID, isolate, etc..</li>
<li>Select the taxonomic categories relevant for your study; in this case it is ‘class’.</li>
<li>Here you can select colors to designate taxa in the phylogenetic tree in iTOL. Double-click to modify the color. If the number of colors that you set is lesser than the number of studied taxonomic groups, random colors will be assigned to the remaining taxa. For example, there are 11 classes, and I only set 5 colors, so the remaining 6 classes will use random colors;</li>
<li>Click Start to run the Extraction function.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/extract_en.jpg?raw=true"></p>
<p>After the extraction, open the <code>extract_results/rRNA</code> folder (the <code>feature</code> identity of 18S is rRNA, so the folder name is rRNA). We can see that different entries use different names for 18S, so we need to unify the names first.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/rRNA_en.jpg?raw=true"></p>
<p>Open the <code>extract_results/StatFiles</code> folder, find the <code>name_for_unification.csv</code> file, change the name to 18S in the <code>New Name</code> column, and save (you must save as csv format). As I am not sure that 16S-like entries are homologous to 18S, I will leave them unchanged.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/name_for_unification_en.jpg?raw=true"></p>
<p>Then go back to the window to extract the sequence</p>
<ol>
<li>Click the Settings button next to the sequence type to enter the settings interface;</li>
<li>Drag the modified <code>name_for_unification.csv</code> file into the area indicated in the figure, and the modified names will be automatically recognized.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modify_replace_names_en.jpg?raw=true"></p>
<p>Just close the settings window, your new settings will be automatically saved. At this point, you only need to re-run the extraction anew to get the 18S sequence after replacing the name. After reopening the 18S.fas file in the rRNA folder, you will find only 42 sequences in it (remember we have a total of 51 IDs). Three more sequences, annotated as 16-like, are in the same folder. Because I wasn’t sure that these are homologues, I left them in a separate file so I can manually revise their homology. If you are certain, you can rename them to 18S as described above. The extraction of the remaining 6 missing sequences was not successful, the details of which can be found in the ‘StatFiles’ folder in files: <code>StatFiles/feature_unrecognized.csv</code> and <code>StatFiles/ID_with_no_features.csv</code>. Best way to see what went wrong is to open these sequences (main working window, find the ID, right-click, open) and compare them with successfully extracted files. Once&#x2F;if you figure out what is wrong with them, you may manually edit these files in a text editor, save them, and re-extract all sequences. </p>
<p>However, if you are confident that all the sequences are 18S, here is an easy way:</p>
<ol>
<li>Check ‘Default’ instead of ‘Custom’ in the Extracter function and set the output file name;</li>
<li>Start to extract.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/default_extract_en.jpg?raw=true"></p>
<p>Double-clicking the “18S.fas” file will open the sequence viewer, where you will see that all 51 sequences were successfully extracted. </p>
<h3 id="3-Multiple-sequence-alignment"><a href="#3-Multiple-sequence-alignment" class="headerlink" title="3. Multiple sequence alignment"></a>3. Multiple sequence alignment</h3><p>Now that we have extracted all sequences, we can start the phylogenetic analysis. For this, we have to align them first using MAFFT. There are many ways to do this, only two of which will be shown here.</p>
<ol>
<li><p>First, select the <code>extract_results</code> folder in the main PhyloSuite window (extracted in the previous step), and then open MAFFT through <code>Alignment--&gt;MAFFT</code> in the menu bar. The extracted sequences will be automatically imported into MAFFT;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_load_en.jpg?raw=true"></p>
</li>
<li><p>You can select MAFFT from the Alignment menu, and then a pop-up window will automatically offer you available input file from the entire workspace. Scroll through them to find the results that you need, or click ‘No, thanks’ and select the input file manually.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_prepare_en.jpg?raw=true"></p>
</li>
</ol>
<p>You will find that other plug-ins in PhyloSuite also use this interface to import files.</p>
<p>Next is the sequence alignment</p>
<ol>
<li>First you may choose to delete the 16S sequence files using the delete button (the button next to it can be used to view the sequence). </li>
<li>Align the sequences using Normal mode;</li>
<li>Other parameters can be set according to your own requirements; and then click Start.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mafft_en.jpg?raw=true"></p>
<p>After the alignment is completed, you can view the sequence and modify them.</p>
<ol>
<li>Select the <code>mafft_results</code> folder;</li>
<li>Double-click the alignment result to open it in the sequence viewer.</li>
<li>If necessary, manual correction of the sequence can be performed in the same way as in MEGA.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_seq_en.jpg?raw=true"></p>
<h3 id="4-Nucleotide-replacement-model-selection"><a href="#4-Nucleotide-replacement-model-selection" class="headerlink" title="4. Nucleotide replacement model selection"></a>4. Nucleotide replacement model selection</h3><p>Next is the evolution model selection. Since ModelFinder does not recognize the results of MAFFT automatically (<code>Concatenation</code> function ordinarily serves as a bridge between these two functions, but we won’t use it for a single-gene dataset), we need to manually import it. The operation is also simple:</p>
<ol>
<li>Double-click <code>mafft_results</code> to open the folder;</li>
<li>Open the ModelFinder software window via <code>Phylogeny--&gt;ModelFinder</code> in the menu bar and drag the aligned 18S sequence into the file input box of ModelFinder.</li>
<li>Select <code>Mrbayes</code> in <code>Model for</code> to select the optimal models for <code>Mrbayes</code>;</li>
<li>Other parameters can be selected according to your needs. After settings, you can click <code>Start</code> to start the program.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modelfinder_en.jpg?raw=true"></p>
<h3 id="5-Phylogenetic-tree-reconstruction-based-on-Bayesian-Inference-BI"><a href="#5-Phylogenetic-tree-reconstruction-based-on-Bayesian-Inference-BI" class="headerlink" title="5. Phylogenetic tree reconstruction based on Bayesian Inference (BI)"></a>5. Phylogenetic tree reconstruction based on Bayesian Inference (BI)</h3><ol>
<li>Double-click <code>*.iqtree</code> in the <code>ModelFinder_results</code> folder to view the selection result of the optimal model;</li>
<li>Select the <code>ModelFinder_results</code> folder in the main PhyloSuite interface;</li>
<li>Open the Bayesian software <code>Phylogeny--&gt;MrBayes</code> from the menu bar;</li>
<li>You can see that the alignment and model selection results have been automatically imported into <code>MrBayes</code>. You may choose to select one or more outgroups through the <code>Outgroup</code> drop-down menu;</li>
<li>Select the number of generations you wish to run;</li>
<li>Set other parameters if you wish to do so; and then click <code>Start</code>.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mrbayes_en.jpg?raw=true"></p>
<h3 id="6-Phylogenetic-tree-reconstruction-based-on-the-Maximum-Likelihood-method-ML"><a href="#6-Phylogenetic-tree-reconstruction-based-on-the-Maximum-Likelihood-method-ML" class="headerlink" title="6. Phylogenetic tree reconstruction based on the Maximum Likelihood method (ML)"></a>6. Phylogenetic tree reconstruction based on the Maximum Likelihood method (ML)</h3><p>IQ-TREE is able to select the best-fit model and conduct phylogenetic analysis, you can complete both in one step.</p>
<ol>
<li>Drag the aligned sequences (*.fas) directly into the IQ-TREE file input box.</li>
<li>By selecting the <code>Auto</code> model, you are allowing IQ-TREE to select the optimal evolutionary model first, and then use it to conduct the phylogenetic analysis;</li>
<li>You can choose +R to calculate the FreeRate model, which is recommended by the IQ-TREE author;</li>
<li>Since 18S is small, it is best to use the standard bootstrap method based on the author’s recommendation;</li>
<li>Other parameters can be selected according to your needs; then run it by clicking <code>Start</code>.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/IQ-TREE_en.jpg?raw=true"></p>
<h3 id="7-Flowchart"><a href="#7-Flowchart" class="headerlink" title="7. Flowchart"></a>7. Flowchart</h3><p>In addition to the above-mentioned conventional operation, PhyloSuite also provides a streamlined workflow mode that allows to conduct the analysis described above (sections 3 to 6) in a single go. You only need to input the file in the first program (MAFFT) and the remaining steps will be executed automatically in the default order: alignment–&gt;concatenation–&gt;model selection–&gt;reconstruct the phylogenetic tree; with results of the upstream program being used as the input file for the downstream program.</p>
<ol>
<li>Click <code>Flowchart</code> in the menu bar to open the program window;</li>
<li>Select the analyses you want to conduct; here it will be <code>MAFFT</code>, <code>Concatenation</code> (although it is a single gene, the Concatenation must also be selected, because the downstream programs can only recognize concatenated results as input files, in this way, the Concatenation function served as format convertion), <code>ModelFinder</code>, <code>IQ-TREE</code>, and MrBayes;</li>
<li>Click on the red <code>Input file here</code> to open the MAFFT program window;</li>
<li>Click on the MAFFT file input box select one of the automatically prepared input files (see section 3 for details);</li>
<li>Select the extraction results from before (note that I only kept 18S);</li>
<li>Just close the window and MAFFT will automatically save the input file and the parameters that you set (also applied to other programs).</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_en.jpg?raw=true"></p>
<p>After inputting the file into the MAFFT and setting the parameters, you may also wish to set the parameters for other programs.</p>
<ol>
<li>Click the <code>Check | Start</code> button, and a parameter summary window will pop up, allowing you to modify and check the parameters;</li>
<li>Click the blue words to set the parameters (the corresponding widgets will be highlighted in the pop-up window);</li>
<li>After setting the parameters for each software, just close the window to save the settings;</li>
<li>PhyloSuite also provides the ability to automatically check the conflicting parameters. For example, if the model selected by Modelfinder does not use partition model, you should also turn off partition mode for <code>MrBayes</code>. Click <code>autocorrect</code> to do this automatically.</li>
<li>After the parameters are checked (note that for the demo purpose, we chose very short running times for MrBayes and IQ-TREE), you can click <code>Ok, start</code> to start the program.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_set_en.jpg?raw=true"></p>
<p>IQ-TREE and MrBayes programs can be run simultaneously (they don’t have upstream&#x2F;downstream program relationship). After a program has finished, you can view the results by clicking the button next to the program name. </p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_running_en.jpg?raw=true"></p>
<p>After all the programs have finished, the main interface will give you a summary description of the operation, as well as references to the software used, both of which can be useful for writing the Materials &amp; Methods section of your manuscript. </p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_summary_en.jpg?raw=true"></p>
<h3 id="8-Phylogenetic-tree-annotation"><a href="#8-Phylogenetic-tree-annotation" class="headerlink" title="8. Phylogenetic tree annotation"></a>8. Phylogenetic tree annotation</h3><p>After phylogenetic tree reconstruction, the last step is to annotate the phylogenetic tree. Here we will use the results of IQ-TREE as an example.<br>1.	First open iTOL’s home page <a target="_blank" rel="noopener" href="https://itol.embl.de/">https://itol.embl.de/</a>, log in to your account, and select <code>My Trees</code>;<br>2.	Double-click <code>IQtree_results</code> to open the tree of the IQ-TREE, and drag the *.treefile file into the area shown in the figure to automatically import the phylogenetic tree;<br>   <img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_input_en.jpg?raw=true"><br>3. Click on the tree title to enter the editing interface.<br>4. Double-click <code>extract_results</code> to open the extraction files (that we generated in section 2). The files within the <code>itolFiles</code> folder are used for iTOL annotation. The operation is very simple, just drag the file into the iTOL interface. As shown in the figure, after dragging in <code>itol_gb_labels.txt</code>, you will find that the name of the species has been replaced with the accession number (in my case, there is an error log showing that some of the listed IDs were not found, because 9 sequences have not been extracted; as it will not affect the annotation, we can ignore it for the purpose of this demo). Two more files can be used to replace names (<code>itol_labels.txt</code> and <code>itol_ori_labels.txt</code>);<br>    <img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/drag_gb_label_en.jpg?raw=true"><br>5. Check file names to see which taxonomic category information they carry. For example, <code>itol_Class_ColourStrip.txt</code> and <code>itol_Class_Text.txt</code> can be used to annotate (colourize) different classes.<br>&emsp;a. Drag these two file (<code>itol_Class_ColourStrip.txt</code> and <code>itol_Class_Text.txt</code>) together into the iTOL interface;<br>&emsp;b. Each file is treated as a different dataset by the iTOL. You can set the parameters for each dataset by clicking the jagged setting button;<br>&emsp;c. Set the <code>Text size factor</code> of the <code>Class text</code> dataset to change the font size for the name of the class;<br>&emsp;d. Set the <code>Strip width</code> of the <code>color_strip</code> dataset to change the width of the vertical block.<br>&emsp;<img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_color_en.jpg?raw=true"><br>&emsp; When we extracted the sequences (section 2), there was an option to set the color for the taxonomic groups. The colors of the five classes that we set before can be seen here, while the colors of other classes were randomly chosen. Note that the location of the taxonomic group name may not be in the middle of the colour strip representing the clade (see Colpodea in the figure above), but you can export the tree into an SVG format file and then fine-tune it using Adobe Illustrator.<br>&emsp; The advantages of this annotation method are especially pronounced when you have hundreds of species, or when you want to compare multiple topologies based on the same dataset. Colourated annotation is very helpful in such cases, especially for identifying paraphyly and polyphyly</p>
<p>When annotating the BI tree, just drag the *.con.tre tree file into iTOL, and the rest of the operations are the same as for the IQ-TREE tree. </p>
<p>At this point, the single-gene phylogenetic analysis process is finished. This is the complete operation from data acquisition to phylogenetic tree annotation. Of course, you can start from any step. For example, it is possible to start directly from the MAFFT multiple sequence alignment, but in this way there is no way to annotate the phylogenetic tree.</p>
<hr>

<h3 id="9-Software-download-website"><a href="#9-Software-download-website" class="headerlink" title="9. Software download website"></a>9. Software download website</h3><p><a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a> or <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link">https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link</a> (China)</p>
<h3 id="10-Reference"><a href="#10-Reference" class="headerlink" title="10. Reference"></a>10. Reference</h3><p>Zhang, D., F. Gao, I. Jakovlić, H. Zou, J. Zhang, W.X. Li, and G.T. Wang, PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Molecular Ecology Resources, 2020. 20(1): p. 348–355. DOI: 10.1111&#x2F;1755-0998.13096. </p>
</div>

<!-- 中文内容 -->
<div class="lang lang-zh" style="display:none;">

<p><strong>PhyloSuite：从数据获取到系统发育树注释（单基因建树）</strong></p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/cover.jpg?raw=true"></p>
<!--more-->

<p>为了使用户更容易上手，这里提供一个教程，以 51 条纤毛虫 18S 序列为例 (<a target="_blank" rel="noopener" href="https://onlinelibrary.wiley.com/doi/full/10.1111/j.1550-7408.2009.00413.x ">https://onlinelibrary.wiley.com/doi/full/10.1111/j.1550-7408.2009.00413.x</a>)，简要介绍 PhyloSuite 的使用。</p>
<h3 id="1-序列下载与准备"><a href="#1-序列下载与准备" class="headerlink" title="1. 序列下载与准备"></a>1. 序列下载与准备</h3><p>我们将使用的 accession 号如下（从上述文章的表格中获取）：</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br></pre></td><td class="code"><pre><span class="line">AF164122 X65150 AY547546 U17356 EU286810</span><br><span class="line">AF100301 U47620 M97909 L31520 EU039903</span><br><span class="line">Z29442 U97110 EU039887 Z29438 AY007454</span><br><span class="line">AF060454 AY007445 U82204 X71134 X65149</span><br><span class="line">U57767 L26447 AF300281 AF530529 U17355</span><br><span class="line">AY331802 AY242119 AY331790 EU286812 DQ232761</span><br><span class="line">X71140 EU286811 AY331794 EU264561 EU286809</span><br><span class="line">X65151 U24248 AM292311 EU039899 DQ057347</span><br><span class="line">AF300286 EF123708 EU039896 L26446 U57765</span><br><span class="line">Z22931 AY007450 EU264560 AY378112 DQ834370</span><br><span class="line">U97112</span><br></pre></td></tr></table></figure>
<p>首先，将这些 ID 下载到 PhyloSuite 的工作区：</p>
<ol>
<li>选择任意工作文件夹（这里我选择了 ‘files’）；</li>
<li>点击 <code>打开文件/输入ID</code> 打开输入窗口。</li>
<li>将上面的 ID 复制到文本框中（支持空格、换行符、制表符等作为分隔符）；</li>
<li>输入您的电子邮件（告知 NCBI 是谁在下载序列）；</li>
<li>点击 <code>开始</code> 进行下载。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/download_seq_en.jpg?raw=true"></p>
<p>序列下载到工作区后，我们发现有些分类群被标记为 N&#x2F;A，表示该分类群未被识别。选择所有序列 (CTRL+A)，然后右键单击，选择 <code>获取分类信息 (NCBI)</code>。选择后会得到一个提示，要求确认您是否确定要用 NCBI 分类数据库中的分类信息替换当前的分类信息（这对后续的系统发育树注释很重要）。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_lineages_en.jpg?raw=true"></p>
<p>完成上述步骤后，可能仍有一些 ID 未被识别，表明它们的分类群不在最新的 NCBI 分类系统中。您可能也希望手动更改当前名称，在这种情况下，可以双击相应的单元格并输入正确的分类群名称。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_lineages_en.jpg?raw=true"></p>
<h3 id="2-序列提取"><a href="#2-序列提取" class="headerlink" title="2. 序列提取"></a>2. 序列提取</h3><p>选择所有序列 (Ctrl+A) 并打开上下文菜单（右键单击）。</p>
<ol>
<li>选择 <code>提取</code>；</li>
<li>由于我们提取的是 18S 序列，这里我们选择 18S；</li>
<li>选择您希望用于每条序列的名称类型：organism, ID, isolate 等；</li>
<li>选择与您研究相关的分类类别；本例中是 ‘class’；</li>
<li>在这里您可以选择颜色以在 iTOL 的系统发育树中标识分类群。双击可修改颜色。如果您设置的颜色数量少于研究的分类群数量，将为剩余的分类群分配随机颜色。例如，有 11 个纲，我只设置了 5 种颜色，那么剩余的 6 个纲将使用随机颜色；</li>
<li>点击 <code>开始</code> 运行提取功能。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/extract_en.jpg?raw=true"></p>
<p>提取后，打开 <code>extract_results/rRNA</code> 文件夹（18S 的 <code>feature</code> 标识是 rRNA，所以文件夹名是 rRNA）。我们可以看到不同的条目对 18S 使用了不同的名称，所以我们需要先统一名称。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/rRNA_en.jpg?raw=true"></p>
<p>打开 <code>extract_results/StatFiles</code> 文件夹，找到 <code>name_for_unification.csv</code> 文件，在 <code>New Name</code> 列中将名称改为 18S，然后保存（必须保存为 csv 格式）。由于我不确定 16S-like 条目是否与 18S 同源，我将保持它们不变。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/name_for_unification_en.jpg?raw=true"></p>
<p>然后返回到提取序列的窗口</p>
<ol>
<li>点击序列类型旁边的 <code>设置</code> 按钮进入设置界面；</li>
<li>将修改后的 <code>name_for_unification.csv</code> 文件拖拽到图中指示的区域，修改后的名称将被自动识别。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modify_replace_names_en.jpg?raw=true"></p>
<p>只需关闭设置窗口，您的新设置将自动保存。此时，您只需要重新运行提取即可获得替换名称后的 18S 序列。重新打开 rRNA 文件夹中的 18S.fas 文件后，您会发现其中只有 42 条序列（记得我们总共有 51 个 ID）。另外三条注释为 16S-like 的序列也在同一文件夹中。因为我不确定它们是否是同源物，所以将它们留在一个单独的文件中，以便我可以手动修正它们的同源性。如果您确定，可以如上所述将它们重命名为 18S。其余 6 条缺失序列的提取没有成功，详细信息可以在 files 中的 ‘StatFiles’ 文件夹中找到：<code>StatFiles/feature_unrecognized.csv</code> 和 <code>StatFiles/ID_with_no_features.csv</code>。查看问题的最佳方式是打开这些序列（主工作窗口，找到 ID，右键单击，打开）并与成功提取的文件进行比较。一旦&#x2F;如果您找出问题所在，可以在文本编辑器中手动编辑这些文件，保存它们，然后重新提取所有序列。</p>
<p>但是，如果您确信所有序列都是 18S，这里有一个简单的方法：</p>
<ol>
<li>在提取器功能中勾选 ‘Default’ 而不是 ‘Custom’，并设置输出文件名；</li>
<li>开始提取。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/default_extract_en.jpg?raw=true"></p>
<p>双击 “18S.fas” 文件将打开序列查看器，您将看到所有 51 条序列都已成功提取。</p>
<h3 id="3-多序列比对"><a href="#3-多序列比对" class="headerlink" title="3. 多序列比对"></a>3. 多序列比对</h3><p>现在我们已提取所有序列，可以开始系统发育分析了。为此，我们必须先使用 MAFFT 对它们进行比对。有多种方法可以做到这一点，这里只展示其中两种。</p>
<ol>
<li>首先，在主 PhyloSuite 窗口中选择 <code>extract_results</code> 文件夹（上一步提取的），然后通过菜单栏中的 <code>Alignment--&gt;MAFFT</code> 打开 MAFFT。提取的序列将自动导入 MAFFT；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_load_en.jpg?raw=true"></li>
<li>您可以从 Alignment 菜单中选择 MAFFT，然后会自动弹出一个窗口，提供整个工作区中可用的输入文件。滚动查找您需要的结果，或点击 ‘No, thanks’ 手动选择输入文件。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/auto_prepare_en.jpg?raw=true"></li>
</ol>
<p>您会发现 PhyloSuite 中的其他插件也使用此界面导入文件。</p>
<p>接下来是序列比对</p>
<ol>
<li>首先您可以选择使用删除按钮删除 16S 序列文件（旁边的按钮可用于查看序列）。</li>
<li>使用 Normal 模式比对序列；</li>
<li>其他参数可根据您自己的要求设置；然后点击 <code>开始</code>。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mafft_en.jpg?raw=true"></p>
<p>比对完成后，您可以查看序列并进行修改。</p>
<ol>
<li>选择 <code>mafft_results</code> 文件夹；</li>
<li>双击比对结果在序列查看器中打开它。</li>
<li>如有必要，可以像在 MEGA 中一样手动校正序列。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/edit_seq_en.jpg?raw=true"></p>
<h3 id="4-核苷酸替代模型选择"><a href="#4-核苷酸替代模型选择" class="headerlink" title="4. 核苷酸替代模型选择"></a>4. 核苷酸替代模型选择</h3><p>接下来是进化模型选择。由于 ModelFinder 不能自动识别 MAFFT 的结果（<code>串联</code> 功能通常作为这两个功能之间的桥梁，但我们不会对单基因数据集使用它），我们需要手动导入它。操作也很简单：</p>
<ol>
<li>双击 <code>mafft_results</code> 打开文件夹；</li>
<li>通过菜单栏中的 <code>Phylogeny--&gt;ModelFinder</code> 打开 ModelFinder 软件窗口，并将比对好的 18S 序列拖拽到 ModelFinder 的文件输入框中。</li>
<li>在 <code>Model for</code> 中选择 <code>Mrbayes</code>，以为 <code>Mrbayes</code> 选择最优模型；</li>
<li>其他参数可根据您的需要选择。设置完成后，可以点击 <code>开始</code> 启动程序。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/modelfinder_en.jpg?raw=true"></p>
<h3 id="5-基于贝叶斯推断-BI-的系统发育树重建"><a href="#5-基于贝叶斯推断-BI-的系统发育树重建" class="headerlink" title="5. 基于贝叶斯推断 (BI) 的系统发育树重建"></a>5. 基于贝叶斯推断 (BI) 的系统发育树重建</h3><ol>
<li>双击 <code>ModelFinder_results</code> 文件夹中的 <code>*.iqtree</code> 查看最优模型的选择结果；</li>
<li>在主 PhyloSuite 界面中选择 <code>ModelFinder_results</code> 文件夹；</li>
<li>从菜单栏打开贝叶斯软件 <code>Phylogeny--&gt;MrBayes</code>；</li>
<li>您可以看到比对和模型选择结果已自动导入 <code>MrBayes</code>。您可以通过 <code>Outgroup</code> 下拉菜单选择一个或多个外群；</li>
<li>选择您希望运行的代数；</li>
<li>如果您希望，可以设置其他参数；然后点击 <code>开始</code>。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/mrbayes_en.jpg?raw=true"></p>
<h3 id="6-基于最大似然法-ML-的系统发育树重建"><a href="#6-基于最大似然法-ML-的系统发育树重建" class="headerlink" title="6. 基于最大似然法 (ML) 的系统发育树重建"></a>6. 基于最大似然法 (ML) 的系统发育树重建</h3><p>IQ-TREE 能够选择最佳拟合模型并进行系统发育分析，您可以一步完成两者。</p>
<ol>
<li>直接将比对好的序列 (*.fas) 拖拽到 IQ-TREE 的文件输入框中。</li>
<li>选择 <code>Auto</code> 模型，您就允许 IQ-TREE 先选择最优进化模型，然后使用它进行系统发育分析；</li>
<li>您可以选择 +R 来计算 FreeRate 模型，这是 IQ-TREE 作者推荐的；</li>
<li>由于 18S 较小，根据作者的建议，最好使用标准的自举法；</li>
<li>其他参数可根据您的需要选择；然后点击 <code>开始</code> 运行它。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/IQ-TREE_en.jpg?raw=true"></p>
<h3 id="7-工作流"><a href="#7-工作流" class="headerlink" title="7. 工作流"></a>7. 工作流</h3><p>除了上述常规操作外，PhyloSuite 还提供了一个简化的工作流模式，允许一次性完成上述（第 3 至 6 节）描述的分析。您只需要在第一个程序 (MAFFT) 中输入文件，剩余步骤将按默认顺序自动执行：比对–&gt;串联–&gt;模型选择–&gt;重建系统发育树；上游程序的结果将作为下游程序的输入文件。</p>
<ol>
<li>点击菜单栏中的 <code>Flowchart</code> 打开程序窗口；</li>
<li>选择您想要进行的分析；这里将是 <code>MAFFT</code>、<code>Concatenation</code>（虽然是单基因，也必须选择 Concatenation，因为下游程序只能识别串联结果作为输入文件，这样 Concatenation 功能就起到了格式转换的作用）、<code>ModelFinder</code>、<code>IQ-TREE</code> 和 MrBayes；</li>
<li>点击红色的 <code>Input file here</code> 打开 MAFFT 程序窗口；</li>
<li>点击 MAFFT 文件输入框，选择其中一个自动准备好的输入文件（详见第 3 节）；</li>
<li>选择之前的提取结果（注意我只保留了 18S）；</li>
<li>只需关闭窗口，MAFFT 将自动保存输入文件以及您设置的参数（也适用于其他程序）。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_en.jpg?raw=true"></p>
<p>将文件输入 MAFFT 并设置参数后，您可能还希望为其他程序设置参数。</p>
<ol>
<li>点击 <code>Check | Start</code> 按钮，将弹出一个参数摘要窗口，允许您修改和检查参数；</li>
<li>点击蓝色文字设置参数（相应的小部件将在弹出窗口中高亮显示）；</li>
<li>为每个软件设置参数后，只需关闭窗口即可保存设置；</li>
<li>PhyloSuite 还提供了自动检查冲突参数的能力。例如，如果 Modelfinder 选择的模型不使用分区模型，您也应该为 <code>MrBayes</code> 关闭分区模式。点击 <code>autocorrect</code> 自动完成此操作。</li>
<li>参数检查完毕后（请注意，出于演示目的，我们为 MrBayes 和 IQ-TREE 选择了非常短的运行时间），您可以点击 <code>Ok, start</code> 启动程序。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_set_en.jpg?raw=true"></p>
<p>IQ-TREE 和 MrBayes 程序可以同时运行（它们没有上游&#x2F;下游程序关系）。一个程序完成后，您可以通过点击程序名称旁边的按钮查看结果。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_running_en.jpg?raw=true"></p>
<p>所有程序完成后，主界面将给出操作摘要说明，以及所用软件的参考文献，这两者都对撰写稿件的材料与方法部分很有用。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/flowchart_summary_en.jpg?raw=true"></p>
<h3 id="8-系统发育树注释"><a href="#8-系统发育树注释" class="headerlink" title="8. 系统发育树注释"></a>8. 系统发育树注释</h3><p>系统发育树重建后，最后一步是对系统发育树进行注释。这里我们将以 IQ-TREE 的结果为例。</p>
<ol>
<li>首先打开 iTOL 的主页 <a target="_blank" rel="noopener" href="https://itol.embl.de/%EF%BC%8C%E7%99%BB%E5%BD%95%E6%82%A8%E7%9A%84%E8%B4%A6%E6%88%B7%EF%BC%8C%E7%84%B6%E5%90%8E%E9%80%89%E6%8B%A9">https://itol.embl.de/，登录您的账户，然后选择</a> <code>My Trees</code>；</li>
<li>双击 <code>IQtree_results</code> 打开 IQ-TREE 的树，并将 *.treefile 文件拖入图中所示区域以自动导入系统发育树；<br>   <img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_input_en.jpg?raw=true"></li>
<li>点击树标题进入编辑界面。</li>
<li>双击 <code>extract_results</code> 打开提取文件（我们在第 2 节生成的）。<code>itolFiles</code> 文件夹内的文件用于 iTOL 注释。操作非常简单，只需将文件拖入 iTOL 界面即可。如图所示，拖入 <code>itol_gb_labels.txt</code> 后，您会发现物种名称已被替换为 accession 号（在我的例子中，有一个错误日志显示列出的一些 ID 未找到，因为有 9 条序列未被提取；由于它不会影响注释，出于本演示的目的我们可以忽略它）。还有两个文件可用于替换名称 (<code>itol_labels.txt</code> 和 <code>itol_ori_labels.txt</code>)；<br>    <img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/drag_gb_label_en.jpg?raw=true"></li>
<li>检查文件名以查看它们携带了哪个分类层级的信息。例如，<code>itol_Class_ColourStrip.txt</code> 和 <code>itol_Class_Text.txt</code> 可用于注释（着色）不同的纲。<br>&emsp;a. 将这两个文件 (<code>itol_Class_ColourStrip.txt</code> 和 <code>itol_Class_Text.txt</code>) 一起拖入 iTOL 界面；<br>&emsp;b. 每个文件都被 iTOL 视为不同的数据集。您可以通过点击锯齿状的设置按钮为每个数据集设置参数；<br>&emsp;c. 设置 <code>Class text</code> 数据集的 <code>Text size factor</code> 来更改纲名称的字体大小；<br>&emsp;d. 设置 <code>color_strip</code> 数据集的 <code>Strip width</code> 来更改垂直色块的宽度。<br>&emsp;<img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/single_gene_tutorial/itol_color_en.jpg?raw=true"><br>&emsp; 当我们提取序列时（第 2 节），有一个选项可以设置分类群的颜色。我们之前设置的五个纲的颜色可以在这里看到，而其他纲的颜色是随机选择的。请注意，分类群名称的位置可能不在代表该分支的色带中间（见上图中的 Colpodea），但您可以将树导出为 SVG 格式文件，然后使用 Adobe Illustrator 进行微调。<br>&emsp; 当您有数百个物种时，或者当您想比较基于同一数据集的多个拓扑结构时，这种注释方法的优势尤其明显。彩色注释在这种情况下非常有帮助，特别是对于识别并系群和多系群。</li>
</ol>
<p>注释 BI 树时，只需将 *.con.tre 树文件拖入 iTOL，其余操作与 IQ-TREE 树相同。</p>
<p>至此，单基因系统发育分析流程结束。这是从数据获取到系统发育树注释的完整操作。当然，您可以从任何步骤开始。例如，可以直接从 MAFFT 多序列比对开始，但这样就无法注释系统发育树。</p>
<hr>

<h3 id="9-软件下载网站"><a href="#9-软件下载网站" class="headerlink" title="9. 软件下载网站"></a>9. 软件下载网站</h3><p><a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a> 或 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link">https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link</a> (中国)</p>
<h3 id="10-参考文献"><a href="#10-参考文献" class="headerlink" title="10. 参考文献"></a>10. 参考文献</h3><p>Zhang, D., F. Gao, I. Jakovlić, H. Zou, J. Zhang, W.X. Li, and G.T. Wang, PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Molecular Ecology Resources, 2020. 20(1): p. 348–355. DOI: 10.1111&#x2F;1755-0998.13096.</p>
</div>

<!-- JS 切换逻辑 + 按钮高亮 + 记忆用户选择 -->
<script>
document.addEventListener('DOMContentLoaded', function() {
  const buttons = document.querySelectorAll('.lang-btn');
  const langContents = document.querySelectorAll('.lang');

  // 切换语言函数
  function switchLang(lang){
    // 隐藏所有内容
    langContents.forEach(el => el.style.display = 'none');
    // 显示对应语言
    document.querySelectorAll('.lang-' + lang).forEach(el => el.style.display = 'block');
    
    // 高亮按钮
    buttons.forEach(btn => btn.classList.remove('active'));
    document.querySelector(`.lang-btn[onclick="switchLang('${lang}')"]`).classList.add('active');
    
    // 保存选择到 localStorage
    localStorage.setItem('preferredLang', lang);
  }

  // 页面加载时读取用户上次选择
  const savedLang = localStorage.getItem('preferredLang') || 'zh';
  switchLang(savedLang);

  // 给按钮绑定点击事件
  buttons.forEach(btn => {
    btn.addEventListener('click', () => {
      const lang = btn.getAttribute('onclick').match(/switchLang\('(\w+)'\)/)[1];
      switchLang(lang);
    });
  });
});
</script>

<!-- CSS 美化按钮 -->
<style>
#lang-switcher {
  display: flex;
  gap: 10px;
}

.lang-btn {
  padding: 6px 14px;
  border: 1px solid #888;
  border-radius: 6px;
  background-color: #f5f5f5;
  cursor: pointer;
  transition: 0.2s;
  font-weight: bold;
}

.lang-btn:hover {
  background-color: #e0e0e0;
}

.lang-btn.active {
  background-color: #007bff;
  color: white;
  border-color: #007bff;
}
</style>
    </div>

    
    
    

    <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/PyQT5/" rel="tag"># PyQT5</a>
              <a href="/tags/Python/" rel="tag"># Python</a>
              <a href="/tags/Bioinformatics/" rel="tag"># Bioinformatics</a>
          </div>

        

          <div class="post-nav">
            <div class="post-nav-item">
                <a href="/PhyloSuite-demo/five_ways_to_import/" rel="prev" title="Five ways to import data into PhyloSuite">
                  <i class="fa fa-angle-left"></i> Five ways to import data into PhyloSuite
                </a>
            </div>
            <div class="post-nav-item">
                <a href="/PhyloSuite-demo/multi-gene-phylogeny/" rel="next" title="Muti-gene phylogeny">
                  Muti-gene phylogeny <i class="fa fa-angle-right"></i>
                </a>
            </div>
          </div>
    </footer>
  </article>
</div>






    <div class="comments gitalk-container"></div>
</div>
  </main>

  <footer class="footer">
    <div class="footer-inner">

  <div class="copyright">
    &copy; 
    <span itemprop="copyrightYear">2025</span>
    <span class="with-love">
      <i class="user"></i>
    </span>
    <span class="author" itemprop="copyrightHolder">Dong Zhang</span>
  </div>
  <div class="powered-by">Powered by <a href="https://hexo.io/" rel="noopener" target="_blank">Hexo</a> & <a href="https://theme-next.js.org/" rel="noopener" target="_blank">NexT.Gemini</a>
  </div>

    </div>
  </footer>

  
  <div class="toggle sidebar-toggle" role="button">
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
  </div>
  <div class="sidebar-dimmer"></div>
  <div class="reading-progress-bar"></div>

<noscript>
  <div class="noscript-warning">Theme NexT works best with JavaScript enabled</div>
</noscript>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/gitalk/1.8.0/gitalk.css" integrity="sha256-AJnUHL7dBv6PGaeyPQJcgQPDjt/Hn/PvYZde1iqfp8U=" crossorigin="anonymous">

<script class="next-config" data-name="gitalk" type="application/json">{"enable":true,"github_id":"dongzhang0725","repo":"PhyloSuite","client_id":"55e7b2bd2c9d294158bf","client_secret":"b0716af44800887bfab50497c5e0309525a353b8","admin_user":"dongzhang0725","distraction_free_mode":true,"proxy":"https://cors-anywhere.azm.workers.dev/https://github.com/login/oauth/access_token","language":null,"js":{"url":"https://cdnjs.cloudflare.com/ajax/libs/gitalk/1.8.0/gitalk.min.js","integrity":"sha256-MVK9MGD/XJaGyIghSVrONSnoXoGh3IFxLw0zfvzpxR4="},"path_md5":"13723fd7a4e0d71ab217ea50a7075876"}</script>
<script src="/js/third-party/comments/gitalk.js" defer></script>

</body>
</html>
