<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width">
<meta name="theme-color" content="#222"><meta name="generator" content="Hexo 7.3.0">

  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/apple-touch-icon-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">



<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/7.0.0/css/all.min.css" integrity="sha256-VHqXKFhhMxcpubYf9xiWdCiojEbY9NexQ4jh8AxbvcM=" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.1.1/animate.min.css" integrity="sha256-PR7ttpcvz8qrF57fur/yAx1qXMFJeJFiA6pSzWi0OIE=" crossorigin="anonymous">

<script class="next-config" data-name="main" type="application/json">{"hostname":"dongzhang0725.github.io","root":"/","images":"/images","scheme":"Gemini","darkmode":false,"version":"8.25.0","exturl":false,"sidebar":{"position":"left","width_expanded":320,"width_dual_column":240,"display":"post","padding":18,"offset":12},"hljswrap":true,"codeblock":{"theme":{"light":"default","dark":"stackoverflow-dark"},"prism":{"light":"prism","dark":"prism-dark"},"copy_button":{"enable":false,"style":null},"fold":{"enable":false,"height":500},"language":false},"bookmark":{"enable":false,"color":"#222","save":"auto"},"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"stickytabs":false,"motion":{"enable":true,"async":false,"duration":200,"transition":{"menu_item":"fadeInDown","post_block":"fadeIn","post_header":"fadeInDown","post_body":"fadeInDown","coll_header":"fadeInLeft","sidebar":"fadeInUp"}},"i18n":{"placeholder":"Searching...","empty":"We didn't find any results for the search: ${query}","hits_time":"${hits} results found in ${time} ms","hits":"${hits} results found"},"path":"/search.xml","localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false}}</script><script src="/js/config.js" defer></script>

    <meta name="description" content="English   中文      PhyloSuite From sequence to tree and time: Phylogenetics and molecular dating made easy with PhyloSuite  Homepage: https:&#x2F;&#x2F;dongzhang0725.github.io Examples and test run: https:&#x2F;&#x2F;">
<meta property="og:type" content="website">
<meta property="og:title" content="PhyloSuite">
<meta property="og:url" content="https://dongzhang0725.github.io/documentation/index.html">
<meta property="og:site_name" content="PhyloSuite">
<meta property="og:description" content="English   中文      PhyloSuite From sequence to tree and time: Phylogenetics and molecular dating made easy with PhyloSuite  Homepage: https:&#x2F;&#x2F;dongzhang0725.github.io Examples and test run: https:&#x2F;&#x2F;">
<meta property="og:locale" content="en_US">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/all_flowchart_english.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mainwindow.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_root.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_workfolder.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_results_folder.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/double_click.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_menubar.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/download_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/search_ncbi.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE-autoinput.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/extract_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mafft_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/MACSE_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/trimAl_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/HmmCleaner_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/gblocks_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/concatenation_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/convertFMT_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/modelfinder_demo_2.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/pf_demo_2.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit1.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit2.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit3.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit4.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit5.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/iqtree_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_cipres.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/stop_mb_run.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_continue.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_settings.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/annotations.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/parseANNT_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_notorg.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/RSCU_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/update_failed.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/open_phylosuite.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/exe_mb_terminal.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_error_win.png?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/installation/MAC_error1.png?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/all_flowchart_english.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mainwindow.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_root.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_workfolder.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_results_folder.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/double_click.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_menubar.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/download_seq_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/search_ncbi.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE-autoinput.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/extract_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mafft_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/MACSE_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/trimAl_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/HmmCleaner_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/gblocks_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/concatenation_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/convertFMT_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/modelfinder_demo_2.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/pf_demo_2.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit1.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit2.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit3.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit4.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit5.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/iqtree_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_cipres.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/stop_mb_run.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_continue.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_en.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_settings.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/annotations.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/parseANNT_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_notorg.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/RSCU_demo.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/update_failed.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/open_phylosuite.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/exe_mb_terminal.jpg?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_error_win.png?raw=true">
<meta property="og:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/installation/MAC_error1.png?raw=true">
<meta property="article:published_time" content="2019-08-11T13:24:04.000Z">
<meta property="article:modified_time" content="2025-11-26T02:22:47.266Z">
<meta property="article:author" content="Dong Zhang">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/all_flowchart_english.jpg?raw=true">


<link rel="canonical" href="https://dongzhang0725.github.io/documentation/">


<script class="next-config" data-name="page" type="application/json">{"sidebar":"","isHome":false,"isPost":false,"lang":"en","comments":true,"permalink":"https://dongzhang0725.github.io/documentation/index.html","path":"documentation/index.html","title":""}</script>

<script class="next-config" data-name="calendar" type="application/json">""</script>
<title> | PhyloSuite
</title>
  








  
  <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/3.2.1/anime.min.js" integrity="sha256-XL2inqUJaslATFnHdJOi9GfQ60on8Wx1C2H8DYiN1xY=" crossorigin="anonymous" defer></script>
<script src="/js/utils.js" defer></script><script src="/js/motion.js" defer></script><script src="/js/sidebar.js" defer></script><script src="/js/next-boot.js" defer></script>

  <script src="https://cdnjs.cloudflare.com/ajax/libs/hexo-generator-searchdb/1.5.0/search.js" integrity="sha256-xFC6PJ82SL9b3WkGjFavNiA9gm5z6UBxWPiu4CYjptg=" crossorigin="anonymous" defer></script>
<script src="/js/third-party/search/local-search.js" defer></script>







  





  <noscript>
    <link rel="stylesheet" href="/css/noscript.css">
  </noscript>
</head>

<body itemscope itemtype="http://schema.org/WebPage" class="use-motion">
  <div class="headband"></div>

  <main class="main">
    <div class="column">
      <header class="header" itemscope itemtype="http://schema.org/WPHeader"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar" role="button">
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
        <span class="toggle-line"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <i class="logo-line"></i>
      <p class="site-title">PhyloSuite</p>
      <i class="logo-line"></i>
    </a>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger" aria-label="Search" role="button">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>



<nav class="site-nav">
  <ul class="main-menu menu"><li class="menu-item menu-item-home"><a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a></li><li class="menu-item menu-item-documentation"><a href="/documentation/" rel="section"><i class="fa fa-navicon fa-fw"></i>documentation</a></li><li class="menu-item menu-item-installation"><a href="/installation/" rel="section"><i class="fa fa-download fa-fw"></i>installation</a></li><li class="menu-item menu-item-example"><a href="/example/" rel="section"><i class="fa fa-print fa-fw"></i>example</a></li><li class="menu-item menu-item-categories"><a href="/categories/" rel="section"><i class="fa fa-th fa-fw"></i>Categories</a></li><li class="menu-item menu-item-archives"><a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>Archives</a></li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>Search
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
      <div class="search-header">
        <span class="search-icon">
          <i class="fa fa-search"></i>
        </span>
        <div class="search-input-container">
          <input autocomplete="off" autocapitalize="off" maxlength="80"
                placeholder="Searching..." spellcheck="false"
                type="search" class="search-input">
        </div>
        <span class="popup-btn-close" role="button">
          <i class="fa fa-times-circle"></i>
        </span>
      </div>
      <div class="search-result-container">
        <div class="search-result-icon">
          <i class="fa fa-spinner fa-pulse fa-5x"></i>
        </div>
      </div>
    </div>
  </div>

</header>
        
  
  <aside class="sidebar">

    <div class="sidebar-inner sidebar-nav-active sidebar-toc-active">
      <ul class="sidebar-nav">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <div class="sidebar-panel-container">
        <!--noindex-->
        <div class="post-toc-wrap sidebar-panel">
            <div class="post-toc animated"><ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#1-Introduction"><span class="nav-text">1. Introduction</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#1-1-Background"><span class="nav-text">1.1. Background</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#1-2-Functions"><span class="nav-text">1.2. Functions</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#2-License-Disclaimer"><span class="nav-text">2. License &amp; Disclaimer</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#2-1-License"><span class="nav-text">2.1. License</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#2-2-Disclaimer"><span class="nav-text">2.2. Disclaimer</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#3-Operating-systems-and-installation"><span class="nav-text">3. Operating systems and installation</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#3-1-Install-compiled-PhyloSuite"><span class="nav-text">3.1. Install compiled PhyloSuite</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#3-1-1-Windows"><span class="nav-text">3.1.1. Windows</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#3-1-2-Mac-OSX-nbsp-Linux"><span class="nav-text">3.1.2. Mac OSX &amp; Linux</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#3-2-Install-using-pip"><span class="nav-text">3.2. Install using pip</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#4-Management"><span class="nav-text">4. Management</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#4-1-Interface-operation"><span class="nav-text">4.1. Interface operation</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-1-1-Brief-example"><span class="nav-text">4.1.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-2-Plugins-installation"><span class="nav-text">4.2. Plugins installation</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-2-1-Brief-example"><span class="nav-text">4.2.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-3-Import-Files"><span class="nav-text">4.3. Import Files</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-3-1-GenBank-file"><span class="nav-text">4.3.1. GenBank file</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-3-1-1-Brief-example"><span class="nav-text">4.3.1.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-3-2-Other-types-of-files"><span class="nav-text">4.3.2. Other types of files</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-3-3-Search-in-the-NCBI"><span class="nav-text">4.3.3. Search in the NCBI</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-3-3-1-Brief-example"><span class="nav-text">4.3.3.1. Brief example</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-4-GenBank-file-settings"><span class="nav-text">4.4. GenBank file settings</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-4-1-Lineage-recognition"><span class="nav-text">4.4.1. Lineage recognition</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-4-2-Information-display-and-modification"><span class="nav-text">4.4.2. Information display and modification</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-4-3-Features-extraction"><span class="nav-text">4.4.3. Features extraction</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-4-3-1-Brief-example"><span class="nav-text">4.4.3.1. Brief example</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-5-File-operation"><span class="nav-text">4.5. File operation</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-5-1-Input-files"><span class="nav-text">4.5.1. Input files</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-5-1-1-Autodetect-input-files"><span class="nav-text">4.5.1.1. Autodetect input files</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#4-5-1-2-Specify-input-files"><span class="nav-text">4.5.1.2. Specify input files</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-5-2-Output-files"><span class="nav-text">4.5.2. Output files</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-5-2-1-Brief-example"><span class="nav-text">4.5.2.1. Brief example</span></a></li></ol></li></ol></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#5-Data-analysis"><span class="nav-text">5. Data analysis</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#5-1-Extract-GenBank-file"><span class="nav-text">5.1. Extract GenBank file</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-1-1-Brief-example"><span class="nav-text">5.1.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-2-MAFFT"><span class="nav-text">5.2. MAFFT</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-2-1-Brief-example"><span class="nav-text">5.2.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-3-MACSE"><span class="nav-text">5.3. MACSE</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-3-1-Brief-example"><span class="nav-text">5.3.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-4-trimAl"><span class="nav-text">5.4. trimAl</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-4-1-Brief-example"><span class="nav-text">5.4.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-5-HmmCleaner"><span class="nav-text">5.5. HmmCleaner</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-5-1-Brief-example"><span class="nav-text">5.5.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-6-Gblocks"><span class="nav-text">5.6. Gblocks</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-6-1-Brief-example"><span class="nav-text">5.6.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-7-Concatenate-Sequences"><span class="nav-text">5.7. Concatenate Sequences</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-7-1-Brief-example"><span class="nav-text">5.7.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-8-Convert-format"><span class="nav-text">5.8. Convert format</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-8-1-Brief-example"><span class="nav-text">5.8.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-9-ModelFinder"><span class="nav-text">5.9. ModelFinder</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-9-1-Brief-example"><span class="nav-text">5.9.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-10-PartitionFinder"><span class="nav-text">5.10. PartitionFinder</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-10-1-Brief-example"><span class="nav-text">5.10.1. Brief example</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-10-2-Brief-tutorial-for-partition-editor"><span class="nav-text">5.10.2. Brief tutorial for partition editor</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-11-IQ-TREE"><span class="nav-text">5.11. IQ-TREE</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-11-1-Brief-example"><span class="nav-text">5.11.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-12-MrBayes"><span class="nav-text">5.12. MrBayes</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-12-1-Brief-example"><span class="nav-text">5.12.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-13-Flowchart"><span class="nav-text">5.13. Flowchart</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-13-1-Brief-example"><span class="nav-text">5.13.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-14-Mitogenome"><span class="nav-text">5.14. Mitogenome</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-14-1-Parse-annotations"><span class="nav-text">5.14.1. Parse annotations</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#5-14-1-1-Brief-example"><span class="nav-text">5.14.1.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-14-2-Compare-tables"><span class="nav-text">5.14.2. Compare tables</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#5-14-2-1-Brief-example"><span class="nav-text">5.14.2.1. Brief example</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-14-3-Draw-RSCU-figure"><span class="nav-text">5.14.3. Draw RSCU figure</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#5-14-3-1-Brief-example"><span class="nav-text">5.14.3.1. Brief example</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-15-Molecular-dating-analysis"><span class="nav-text">5.15. Molecular dating analysis</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#6-Citations-and-codes"><span class="nav-text">6. Citations and codes</span></a></li><li class="nav-item nav-level-1"><a class="nav-link" href="#7-Troubleshooting"><span class="nav-text">7. Troubleshooting</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#7-1-Update-failed-how-to-revert-to-previous-settings-and-plugins"><span class="nav-text">7.1. Update failed: how to revert to previous settings and plugins</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-2-PhyloSuite-run-failed"><span class="nav-text">7.2. PhyloSuite run failed</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-3-MrBayes-does-not-work"><span class="nav-text">7.3. MrBayes does not work</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-4-PhyloSuite-get-stuck"><span class="nav-text">7.4. PhyloSuite get stuck</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-5-MAFFT-error"><span class="nav-text">7.5. MAFFT error</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-6-Cannot-run-PhyloSuite-on-MAC"><span class="nav-text">7.6 Cannot run PhyloSuite on MAC</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#8-Acknowledgements"><span class="nav-text">8. Acknowledgements</span></a></li><li class="nav-item nav-level-1"><a class="nav-link" href="#1-%E7%AE%80%E4%BB%8B"><span class="nav-text">1. 简介</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#1-1-%E8%83%8C%E6%99%AF"><span class="nav-text">1.1. 背景</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#1-2-%E5%8A%9F%E8%83%BD"><span class="nav-text">1.2. 功能</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#2-%E8%AE%B8%E5%8F%AF%E4%B8%8E%E5%85%8D%E8%B4%A3%E5%A3%B0%E6%98%8E"><span class="nav-text">2. 许可与免责声明</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#2-1-%E8%AE%B8%E5%8F%AF"><span class="nav-text">2.1. 许可</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#2-2-%E5%85%8D%E8%B4%A3%E5%A3%B0%E6%98%8E"><span class="nav-text">2.2. 免责声明</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#3-%E6%93%8D%E4%BD%9C%E7%B3%BB%E7%BB%9F%E4%B8%8E%E5%AE%89%E8%A3%85"><span class="nav-text">3. 操作系统与安装</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#3-1-%E5%AE%89%E8%A3%85%E5%B7%B2%E7%BC%96%E8%AF%91%E7%9A%84-PhyloSuite"><span class="nav-text">3.1. 安装已编译的 PhyloSuite</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#3-1-1-Windows-1"><span class="nav-text">3.1.1. Windows</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#3-1-2-Mac-OSX-%E5%92%8C-Linux"><span class="nav-text">3.1.2. Mac OSX 和 Linux</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#3-2-%E4%BD%BF%E7%94%A8-pip-%E5%AE%89%E8%A3%85"><span class="nav-text">3.2. 使用 pip 安装</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#4-%E7%AE%A1%E7%90%86"><span class="nav-text">4. 管理</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#4-1-%E7%95%8C%E9%9D%A2%E6%93%8D%E4%BD%9C"><span class="nav-text">4.1. 界面操作</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-1-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">4.1.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-2-%E6%8F%92%E4%BB%B6%E5%AE%89%E8%A3%85"><span class="nav-text">4.2. 插件安装</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-2-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">4.2.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-3-%E5%AF%BC%E5%85%A5%E6%96%87%E4%BB%B6"><span class="nav-text">4.3. 导入文件</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-3-1-GenBank-%E6%96%87%E4%BB%B6"><span class="nav-text">4.3.1. GenBank 文件</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-3-1-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">4.3.1.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-3-2-%E5%85%B6%E4%BB%96%E7%B1%BB%E5%9E%8B%E7%9A%84%E6%96%87%E4%BB%B6"><span class="nav-text">4.3.2. 其他类型的文件</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-3-3-%E5%9C%A8-NCBI-%E4%B8%AD%E6%90%9C%E7%B4%A2"><span class="nav-text">4.3.3. 在 NCBI 中搜索</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-3-3-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">4.3.3.1. 简要示例</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-4-GenBank-%E6%96%87%E4%BB%B6%E8%AE%BE%E7%BD%AE"><span class="nav-text">4.4. GenBank 文件设置</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-4-1-%E8%B0%B1%E7%B3%BB%E8%AF%86%E5%88%AB"><span class="nav-text">4.4.1. 谱系识别</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-4-2-%E4%BF%A1%E6%81%AF%E6%98%BE%E7%A4%BA%E4%B8%8E%E4%BF%AE%E6%94%B9"><span class="nav-text">4.4.2. 信息显示与修改</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-4-3-%E7%89%B9%E5%BE%81%E6%8F%90%E5%8F%96"><span class="nav-text">4.4.3. 特征提取</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-4-3-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">4.4.3.1. 简要示例</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#4-5-%E6%96%87%E4%BB%B6%E6%93%8D%E4%BD%9C"><span class="nav-text">4.5. 文件操作</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#4-5-1-%E8%BE%93%E5%85%A5%E6%96%87%E4%BB%B6"><span class="nav-text">4.5.1. 输入文件</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-5-1-1-%E8%87%AA%E5%8A%A8%E6%A3%80%E6%B5%8B%E8%BE%93%E5%85%A5%E6%96%87%E4%BB%B6"><span class="nav-text">4.5.1.1. 自动检测输入文件</span></a></li><li class="nav-item nav-level-4"><a class="nav-link" href="#4-5-1-2-%E6%8C%87%E5%AE%9A%E8%BE%93%E5%85%A5%E6%96%87%E4%BB%B6"><span class="nav-text">4.5.1.2. 指定输入文件</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#4-5-2-%E8%BE%93%E5%87%BA%E6%96%87%E4%BB%B6"><span class="nav-text">4.5.2. 输出文件</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#4-5-2-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">4.5.2.1. 简要示例</span></a></li></ol></li></ol></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#5-%E6%95%B0%E6%8D%AE%E5%88%86%E6%9E%90"><span class="nav-text">5. 数据分析</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#5-1-%E6%8F%90%E5%8F%96-GenBank-%E6%96%87%E4%BB%B6"><span class="nav-text">5.1. 提取 GenBank 文件</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-1-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.1.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-2-MAFFT-1"><span class="nav-text">5.2. MAFFT</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-2-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.2.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-3-MACSE-1"><span class="nav-text">5.3. MACSE</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-3-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.3.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-4-trimAl-1"><span class="nav-text">5.4. trimAl</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-4-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.4.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-5-HmmCleaner-1"><span class="nav-text">5.5. HmmCleaner</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-5-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.5.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-6-Gblocks-1"><span class="nav-text">5.6. Gblocks</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-6-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.6.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-7-%E4%B8%B2%E8%81%94%E5%BA%8F%E5%88%97"><span class="nav-text">5.7. 串联序列</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-7-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.7.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-8-%E8%BD%AC%E6%8D%A2%E6%A0%BC%E5%BC%8F"><span class="nav-text">5.8. 转换格式</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-8-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.8.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-9-ModelFinder-1"><span class="nav-text">5.9. ModelFinder</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-9-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.9.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-10-PartitionFinder-1"><span class="nav-text">5.10. PartitionFinder</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-10-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.10.1. 简要示例</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-10-2-%E5%88%86%E5%8C%BA%E7%BC%96%E8%BE%91%E5%99%A8%E7%AE%80%E8%A6%81%E6%95%99%E7%A8%8B"><span class="nav-text">5.10.2. 分区编辑器简要教程</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-11-IQ-TREE-1"><span class="nav-text">5.11. IQ-TREE</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-11-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.11.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-12-MrBayes-1"><span class="nav-text">5.12. MrBayes</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-12-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.12.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-13-%E5%B7%A5%E4%BD%9C%E6%B5%81"><span class="nav-text">5.13. 工作流</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-13-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.13.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-14-%E7%BA%BF%E7%B2%92%E4%BD%93%E5%9F%BA%E5%9B%A0%E7%BB%84"><span class="nav-text">5.14. 线粒体基因组</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#5-14-1-%E8%A7%A3%E6%9E%90%E6%B3%A8%E9%87%8A"><span class="nav-text">5.14.1. 解析注释</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#5-14-1-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.14.1.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-14-2-%E6%AF%94%E8%BE%83%E8%A1%A8%E6%A0%BC"><span class="nav-text">5.14.2. 比较表格</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#5-14-2-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.14.2.1. 简要示例</span></a></li></ol></li><li class="nav-item nav-level-3"><a class="nav-link" href="#5-14-3-%E7%BB%98%E5%88%B6-RSCU-%E5%9B%BE"><span class="nav-text">5.14.3. 绘制 RSCU 图</span></a><ol class="nav-child"><li class="nav-item nav-level-4"><a class="nav-link" href="#5-14-3-1-%E7%AE%80%E8%A6%81%E7%A4%BA%E4%BE%8B"><span class="nav-text">5.14.3.1. 简要示例</span></a></li></ol></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#5-15-%E5%88%86%E5%AD%90%E9%92%9F%E5%88%86%E6%9E%90"><span class="nav-text">5.15. 分子钟分析</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#6-%E5%BC%95%E7%94%A8%E4%B8%8E%E4%BB%A3%E7%A0%81"><span class="nav-text">6. 引用与代码</span></a></li><li class="nav-item nav-level-1"><a class="nav-link" href="#7-%E6%95%85%E9%9A%9C%E6%8E%92%E9%99%A4"><span class="nav-text">7. 故障排除</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#7-1-%E6%9B%B4%E6%96%B0%E5%A4%B1%E8%B4%A5%EF%BC%9A%E5%A6%82%E4%BD%95%E6%81%A2%E5%A4%8D%E5%88%B0%E4%BB%A5%E5%89%8D%E7%9A%84%E8%AE%BE%E7%BD%AE%E5%92%8C%E6%8F%92%E4%BB%B6"><span class="nav-text">7.1. 更新失败：如何恢复到以前的设置和插件</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-2-PhyloSuite-%E8%BF%90%E8%A1%8C%E5%A4%B1%E8%B4%A5"><span class="nav-text">7.2. PhyloSuite 运行失败</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-3-MrBayes-%E6%97%A0%E6%B3%95%E5%B7%A5%E4%BD%9C"><span class="nav-text">7.3. MrBayes 无法工作</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-4-PhyloSuite-%E5%8D%A1%E9%A1%BF"><span class="nav-text">7.4. PhyloSuite 卡顿</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-5-MAFFT-%E9%94%99%E8%AF%AF"><span class="nav-text">7.5. MAFFT 错误</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#7-6-%E6%97%A0%E6%B3%95%E5%9C%A8MAC%E7%B3%BB%E7%BB%9F%E8%BF%90%E8%A1%8CPhyloSuite"><span class="nav-text">7.6. 无法在MAC系统运行PhyloSuite</span></a></li></ol></li><li class="nav-item nav-level-1"><a class="nav-link" href="#8-%E8%87%B4%E8%B0%A2"><span class="nav-text">8. 致谢</span></a></li></ol></div>
        </div>
        <!--/noindex-->

        <div class="site-overview-wrap sidebar-panel">
          <div class="site-author animated" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">Dong Zhang</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap animated">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
        <a href="/archives/">
          <span class="site-state-item-count">8</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
          <a href="/categories/">
        <span class="site-state-item-count">1</span>
        <span class="site-state-item-name">categories</span></a>
      </div>
      <div class="site-state-item site-state-tags">
          <a href="/tags/">
        <span class="site-state-item-count">3</span>
        <span class="site-state-item-name">tags</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author animated">
      <span class="links-of-author-item">
        <a href="https://github.com/dongzhang0725/PhyloSuite" title="GitHub → https:&#x2F;&#x2F;github.com&#x2F;dongzhang0725&#x2F;PhyloSuite" rel="noopener me" target="_blank"><i class="fab fa-github fa-fw"></i>GitHub</a>
      </span>
      <span class="links-of-author-item">
        <a href="mailto:dongzhang0725@gmail.com" title="E-Mail → mailto:dongzhang0725@gmail.com" rel="noopener me" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
  </div>

        </div>
      </div>
        <div class="back-to-top animated" role="button" aria-label="Back to top">
          <i class="fa fa-arrow-up"></i>
          <span>0%</span>
        </div>
    </div>

    
  </aside>


    </div>

    <div class="main-inner page posts-expand">


    
    
    
    <div class="post-block" lang="en"><header class="post-header">

<h1 class="post-title" itemprop="name headline">
</h1>

<div class="post-meta-container">
</div>

</header>

      
      
      
      <div class="post-body">
          <!-- 切换按钮 -->
<div id="lang-switcher" style="margin-bottom: 10px;">
  <button class="lang-btn" onclick="switchLang('en')">English</button>
  <button class="lang-btn" onclick="switchLang('zh')">中文</button>
</div>

<!-- 英文内容 -->
<div class="lang lang-en">

<center style="font-size:30px"><strong>PhyloSuite</strong></center>
<center style="font-size:17px"><strong>From sequence to tree and time: Phylogenetics and molecular dating made easy with PhyloSuite</strong></center>
<br>
<center><strong>Homepage: </strong><a href="https://dongzhang0725.github.io">https://dongzhang0725.github.io</a></center>
<center><strong>Examples and test run: </strong><a href="https://dongzhang0725.github.io/dongzhang0725.github.io/example/">https://dongzhang0725.github.io/dongzhang0725.github.io/example/</a></center>
<center><strong>Demo tutorials: </strong><a href="https://dongzhang0725.github.io/dongzhang0725.github.io/archives/">https://dongzhang0725.github.io/dongzhang0725.github.io/archives/</a></center>
<br>
<center style="font-size:17px">Dong Zhang, Fangluan Gao, Ivan Jakovlić, Hong Zou, Jin Zhang, Wen X. Li and Gui T. Wang</center>
<center style="font-size:17px">Version 2 || November 26, 2025</center>

<hr>

<h1 id="1-Introduction"><a href="#1-Introduction" class="headerlink" title="1. Introduction"></a>1. Introduction</h1><h2 id="1-1-Background"><a href="#1-1-Background" class="headerlink" title="1.1. Background"></a>1.1. <em>Background</em></h2><p>Advancements in the next-generation sequencing (NGS) technologies have resulted in a huge increase in the amount of genetic data available through public databases. While this opens a multitude of research possibilities, retrieving and managing such large amounts of data may be difficult and time-consuming for researchers who are not computer-savvy. Therefore, multifunctional, workflow-type and batch-processing enabled software packages, which can save researchers a lot of time, are becoming increasingly needed by a broad range of evolutionary biologists. PhyloSuite was designed to fill that gap: a user-friendly workflow desktop platform dedicated to streamlining molecular sequence data management and evolutionary phylogenetics studies.</p>
<h2 id="1-2-Functions"><a href="#1-2-Functions" class="headerlink" title="1.2. Functions"></a>1.2. <em>Functions</em></h2><p>PhyloSuite is a user-friendly stand-alone GUI-based software written in Python 3.6.7 and PyQT5. The functions are:</p>
<ul>
<li>retrieving, extracting, organizing and managing molecular sequence data, including GenBank entries, nucleotide and amino acid sequences, and sequences annotated in Word documents; </li>
<li>batch alignment of sequences with MAFFT, for which we added a codon alignment (translation align) mode; </li>
<li>batch alignment of protein-coding sequences or refinement of alignments with MACSE;</li>
<li>batch optimization of ambiguously aligned regions using trimAl, HmmCleaner or Gblocks; </li>
<li>batch conversion of alignment formats (FASTA, PHYLIP, PAML, AXT and NEXUS); </li>
<li>concatenation of multiple alignments into a single dataset and preparation of a partition file for downstream analyses; </li>
<li>selection of the best-fit evolutionary model and&#x2F;or partitioning scheme using ModelFinder or PartitionFinder; </li>
<li>phylogeny reconstruction using IQ-TREE (maximum likelihood) and&#x2F;or MrBayes (Bayesian inference); </li>
<li>linking the functions from (ii) to (viii) into a workflow; </li>
<li>annotating phylogenetic trees in the iTOL webtool using datasets generated by the (i) function; </li>
<li>comprehensive bioinformatic analysis of mitochondrial genomes (mitogenomes); </li>
<li>visualization and editing of sequences using a MEGA-like sequence viewer; </li>
<li>storing, organizing and visualizing data and results of each analysis in the PhyloSuite workspace.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/all_flowchart_english.jpg?raw=true"></li>
</ul>
<h1 id="2-License-Disclaimer"><a href="#2-License-Disclaimer" class="headerlink" title="2. License &amp; Disclaimer"></a>2. License &amp; Disclaimer</h1><h2 id="2-1-License"><a href="#2-1-License" class="headerlink" title="2.1. License"></a>2.1. <em>License</em></h2><p>PhyloSuite is a free software, and you are welcome to redistribute it under certain conditions. It is released under the GNU General Public License, Version 3. See <a target="_blank" rel="noopener" href="http://www.gnu.org/licenses/gpl-3.0.en.html">http://www.gnu.org/licenses/gpl-3.0.en.html</a>.</p>
<h2 id="2-2-Disclaimer"><a href="#2-2-Disclaimer" class="headerlink" title="2.2. Disclaimer"></a>2.2. <em>Disclaimer</em></h2><p>This program comes with absolutely no warranty. No guarantee of the functionality of this software, or of the accuracy of results obtained, is expressed or implied. Please inspect your results carefully.</p>
<h1 id="3-Operating-systems-and-installation"><a href="#3-Operating-systems-and-installation" class="headerlink" title="3. Operating systems and installation"></a>3. Operating systems and installation</h1><h2 id="3-1-Install-compiled-PhyloSuite"><a href="#3-1-Install-compiled-PhyloSuite" class="headerlink" title="3.1. Install compiled PhyloSuite"></a>3.1. <em>Install compiled PhyloSuite</em></h2><p>Installers for all platforms can be downloaded from <a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a>.</p>
<p><a id="download" target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases"><i class="fa fa-download"></i><span> Download Now</span> </a></p>
<p><a id="download" target="_blank" rel="noopener" href="http://phylosuite.jushengwu.com/dongzhang0725.github.io/installation/#Chinese_download_link"><i class="fa fa-download"></i><span> Chinese download links</span> </a></p>
<h3 id="3-1-1-Windows"><a href="#3-1-1-Windows" class="headerlink" title="3.1.1. Windows"></a>3.1.1. Windows</h3><p>Windows 7, 8 and 10 are supported, just double click the <code>PhyloSuite_xxx_win_setup.exe</code> to install, and run “PhyloSuite.exe” file after the installation. If the installation fails, download <code>PhyloSuite_xxx_Win.rar</code>, unzip it, and run PhyloSuite directly from this folder.</p>
<h3 id="3-1-2-Mac-OSX-nbsp-Linux"><a href="#3-1-2-Mac-OSX-nbsp-Linux" class="headerlink" title="3.1.2. Mac OSX &amp;&nbsp;Linux"></a>3.1.2. Mac OSX &amp;&nbsp;Linux</h3><p>Unzip <code>PhyloSuite_xxx_Mac.zip/PhyloSuite_xxx_Linux.tar.gz</code> to anywhere you like, and double click “PhyloSuite” (in PhyloSuite folder) to start, or use the following command: </p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">cd path/to/PhyloSuite</span><br><span class="line">./PhyloSuite</span><br></pre></td></tr></table></figure>
<p>If you encounter an error of “permission denied”, try to use the following command:</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">chmod -R 755 path/to/PhyloSuite(folder)</span><br></pre></td></tr></table></figure>

<p><span style="color:red">Note that both 64 bit and 32 bit Windows is supported (Windows 7 and above), whereas only 64 bit has been tested in Linux (Ubuntu 14.04.1 and above) and Mac OSX (macOS Sierra version 10.12.3 and above).</span></p>
<h2 id="3-2-Install-using-pip"><a href="#3-2-Install-using-pip" class="headerlink" title="3.2. Install using pip"></a>3.2. <em>Install using pip</em></h2><p>First, Python (version 3.6) should be installed and added to the environment variable in your computer. Then open the terminal and type:</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">pip install PhyloSuite</span><br></pre></td></tr></table></figure>

<p>It will take some time to install. If it installs successfully, PhyloSuite will be automatically added to the environment variables. Then open the terminal again and type:<br> <figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">PhyloSuite</span><br></pre></td></tr></table></figure></p>
<p>If the above <code>pip</code> command failed to install PhyloSuite, you can use compiled PhyloSuite (see <code>section 1</code>) or find and download the source codes here (<a target="_blank" rel="noopener" href="https://pypi.org/project/PhyloSuite/#files">https://pypi.org/project/PhyloSuite/#files</a> or <a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite">https://github.com/dongzhang0725/PhyloSuite</a>), and install it manually.</p>
<h1 id="4-Management"><a href="#4-Management" class="headerlink" title="4. Management"></a>4. Management</h1><h2 id="4-1-Interface-operation"><a href="#4-1-Interface-operation" class="headerlink" title="4.1. Interface operation"></a>4.1. <em>Interface operation</em></h2><p><span id="operation"><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mainwindow.jpg?raw=true"></span></p>
<p>PhyloSuite uses a workplace for data management (although you don’t have to use it), which is set when you first use the program. Later you can change it through the <code>WorkPlace</code> menu. There are two kinds of root folders in each workplace, <strong>GenBank_File</strong> and <strong>Other_File</strong>. The <strong>GenBank_File</strong> folder is used to manage GenBank files and deposit the results of related analyses. The <strong>Other_File</strong> folder is used to manage other types of sequence files and Word annotation files (nucleotide and amino acid sequences), as well as deposit related results. Below (one level down) from the root folders, you will find work folders, which contain the associated results folders (another level down). In each root folder, PhyloSuite will add <code>files</code> and <code>flowchart</code> work folders by default. You can create a new work folder to deposit your new work (recommended) by mousing over the root folder (GenBank_File and Other_File), either via the green ‘plus’ icon on the right or via the context (right-click) menu. You can remove a work folder via the context menu of the selected folder or by pressing the <code>Delete</code> button in your keyboard. Deleted folders are stored in, and can be recovered from, the <code>recycled</code> folder of the root folder. Once a folder is deleted from the recycle bin, it cannot be recovered. However, it is recommended to delete folders&#x2F;files from your local file system, as that will enable you to recover them using the inbuilt operating system file recovery function. <span style="color:red">Note that almost all of the selected settings, such as the window size, position, parameter settings, etc., will be remembered automatically when you close the windows (i.e., there is no need to save the settings before you close a window).</span></p>
<p>You can access a brief example demo of each function via the <code>question mark</code> button in the window of the corresponding function.</p>
<h3 id="4-1-1-Brief-example"><a href="#4-1-1-Brief-example" class="headerlink" title="4.1.1. Brief example"></a>4.1.1. Brief example</h3><ul>
<li>Clicking <code>GenBank_File</code> or <code>Other_File</code> root folders will display the home page of PhyloSuite. Hover mouse over the root folder to view <code>Add Work Folder</code> and <code>Open in Explorer</code> buttons; the former can be used to create new work folders, whereas the latter opens the folder in your local file explorer;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_root.jpg?raw=true"></li>
<li>Selecting any of your work folders (one level below the root folders) will display a list of your saved datasets. Lists of GenBank records are stored in the <code>GenBank_File</code> root folder, and sequence files are stored in the <code>Other_File</code> root folder. Hover mouse over work folders to see <code>Open in Explorer</code> and <code>GenBank File Information Display Setting</code> buttons; the later can be used to control which data (for each ID) will be displayed on the main page.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_workfolder.jpg?raw=true"></li>
<li>Selecting results folder (one level below the work folders) will display the results. Hover mouse over results folder to see <code>Open in Explorer button</code>;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_results_folder.jpg?raw=true"></li>
<li>Double-clicking any of the above folders (root, work and results) will open the folder in your local file explorer;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/double_click.jpg?raw=true"></li>
<li>Hovering mouse over menu bar to select functions to use.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_menubar.jpg?raw=true"></li>
</ul>
<h2 id="4-2-Plugins-installation"><a href="#4-2-Plugins-installation" class="headerlink" title="4.2. Plugins installation"></a>4.2. <em>Plugins installation</em></h2><p><span id="plugins">PhyloSuite</span> integrates eight plugin programs:</p>
<table>
<thead>
<tr>
<th>Programs</th>
<th>Executable File</th>
<th>Description</th>
</tr>
</thead>
<tbody><tr>
<td><a target="_blank" rel="noopener" href="https://mafft.cbrc.jp/alignment/software/" title="MAFFT">MAFFT v7.313</a></td>
<td>mafft.bat</td>
<td>Multiple alignment of amino acid or nucleotide sequences</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://www.iqtree.org/" title="IQ-TREE">IQ-TREE v. 1.6.8</a></td>
<td>iqtree.exe</td>
<td>Efficient software for phylogenomic inference</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://mrbayes.sourceforge.net/" title="MrBayes">MrBayes 3.2.6</a></td>
<td>mrbayes_x64.exe or <br>mrbayes_x86.exe</td>
<td>Bayesian inference of phylogeny</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://www.robertlanfear.com/partitionfinder/" title="PartitionFinder2">PartitionFinder2</a></td>
<td>partitionfinder folder</td>
<td>Selection of best-fit partitioning schemes and models of molecular evolution for phylogenetic analyses</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_documentation.html" title="Gblocks">Gblocks 0.91b</a></td>
<td>Gblocks.exe</td>
<td>Selection of conserved blocks from multiple alignments for use in phylogenetic analysis</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.r-project.org/" title="Rscript">Rscript 3.4.4</a></td>
<td>Rscript.exe</td>
<td>Required  for drawing RSCU figure</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.anaconda.com/download/" title="Python 2.7">Python 2.7</a></td>
<td>python.exe</td>
<td>Required by PartitionFinder2</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/" title="tbl2asn">tbl2asn</a></td>
<td>tbl2asn.exe</td>
<td>Automates the creation of sequence records for submission to GenBank (Windows only)</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.mpich.org/">MPICH2</a></td>
<td>mpirun or mpiexec</td>
<td>A high-performance and widely portable implementation of the Message Passing Interface (MPI) standard that enables a multi-thread MrBayes operation (Linux only)</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://bioweb.supagro.inra.fr/macse/index.php?menu=releases">MACSE</a></td>
<td>macse_v2.03.jar</td>
<td>Multiple Alignment of Coding Sequences Accounting for Frameshifts and Stop Codons.</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.java.com/en/download/">Java (JRE &gt; 1.5)</a></td>
<td>java.exe</td>
<td>Required by MACSE</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://trimal.cgenomics.org/downloads">trimAl</a></td>
<td>trimal.exe</td>
<td>A tool for automated alignment trimming</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://metacpan.org/pod/distribution/Bio-MUST-Apps-HmmCleaner/bin/HmmCleaner.pl">HmmCleaner</a></td>
<td>HmmCleaner.pl</td>
<td>Removing low similarity segments from your MSA</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.perl.org/get.html">Perl 5</a></td>
<td>perl.exe</td>
<td>Required by HmmCleaner</td>
</tr>
</tbody></table>
<p>These plugins can be installed in <span style="color: #C72591"><code>Settings--&gt;Plugins</code></span>.<br>This can be done in three ways:</p>
<ol>
<li>If Python 2.7, Perl 5, Java (JRE &gt; 1.5), HmmCleaner.pl and trimAl have been installed and added to the environment variable ($PATH), they will be automatically detected by PhyloSuite.</li>
<li>If you already have these programs installed on your computer, you can specify the executable file directly (as indicated in the table above). <span style="color: red">Note that for PartitionFinder2 you should specify the ‘partitionfinder-2.1.1’ folder.<span></li>
<li>If you don’t have these programs, you can use the download button to download and install them automatically. <span style="color: red">Note: Anaconda Python distribution will download for Python 2.7 (because it contains all of the dependencies required by PartitionFinder2: numpy, pandas, pytables, pyparsing, scipy and sklearn). As it is around 500M in size, so your download may take some time.<span></li>
</ol>
<p><span style="color: red">Note that the paths of these plugin programs should not contain special characters (^, {, @, etc.).<span></p>
<h3 id="4-2-1-Brief-example"><a href="#4-2-1-Brief-example" class="headerlink" title="4.2.1. Brief example"></a>4.2.1. Brief example</h3><p>See <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/how-to-configure-plugins/">How to configure plugins</a >.  </p>
<h2 id="4-3-Import-Files"><a href="#4-3-Import-Files" class="headerlink" title="4.3. Import Files"></a>4.3. <em>Import Files</em></h2><p>PhyloSuite accepts numerous file formats and extensions:</p>
<ul>
<li>GenBank file: *.gb, *.gbk, *.gbf, *.gbff</li>
<li>fasta: *.fas, *.fasta</li>
<li>phylip: *.phy, *.phylip</li>
<li>nexus: *.nex, *.nxs, *.nexus</li>
<li>Word document file: *.docx</li>
</ul>
<p>For the demo tutorial of how to import sequences to PhyloSuite, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/five_ways_to_import/">five ways to import data into PhyloSuite</a >.</p>
<h3 id="4-3-1-GenBank-file"><a href="#4-3-1-GenBank-file" class="headerlink" title="4.3.1. GenBank file"></a>4.3.1. GenBank file</h3><p><font color="red">TIP: GenBank file should be in the standard format (see <a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html" title="GenBank Flat File Format">detail</a></font>).<br></p>
<p>PhyloSuite provides three ways to import GenBank files into the work folder of the <strong>GenBank_File</strong> root folder:</p>
<ol>
<li>By using the <code>Import file(s) or ID(s)</code> function under the <strong>File</strong> menu or <code>Open file(s)</code> in the main <a href="#operation">display area</a>. This mode supports the import of complete GenBank files and lists of GenBank accession numbers (IDs), which will then be automatically retrieved from the GenBank by PhyloSuite;</li>
<li>Drag-and-drop GenBank format files into the <a href="#operation">display area</a>. </li>
<li>Copy GenBank file contents and paste them into the <a href="#operation">display area</a>.</li>
</ol>
<h4 id="4-3-1-1-Brief-example"><a href="#4-3-1-1-Brief-example" class="headerlink" title="4.3.1.1. Brief example"></a>4.3.1.1. <span id="import_gb">Brief</span> example</h4><ol>
<li>Select any of the work folders (here I chose files);</li>
<li>Click Open file(s)&#x2F;Input ID(s) to open the input window.</li>
<li>Copy the IDs into the text box (spaces, line breaks, tabs, etc. are supported as separators);</li>
<li>Enter your email (tell NCBI who is downloading the sequences);</li>
<li>Click Start to download.</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/download_seq_en.jpg?raw=true"></p>
<p>After importing GenBank files, there are options to play with, accessible via the context (right-click) menu, if not specified otherwise:</p>
<ul>
<li>Files (IDs) can be added to the dataset that you are working on either by drag&#x2F;drop or via the context menu (”Add file”).</li>
<li>The annotation of GenBank files can be standardized (this includes the gene names unification, discussed above) via the <code>File --&gt; Standardize GenBank file</code> function or via the context menu as <code>Standardization</code>. This function opens a new pop-up window, displaying eventual errors and warnings in your dataset, in which you may manually edit the files (mitogenome data only). These usually involve missing genes or non-standardized annotation. For the latter, you may click on the gear button (Settings) in the upper right corner of this window, which opens the <a href="#extract">GenBank File Extracting settings</a> window discussed above. By ticking the Set NCR threshold box, you may prompt PhyloSuite to recognize the non-coding regions as well (this allows you extract them later using the <code>extract</code> function). You can set the threshold for the size (in bp) of the NCRs you wish to be recognized.</li>
<li>For mitogenomic data, a <code>Predict tRNA (LEU and SER)</code> button is available, via which you may reannotate ambiguously annotated tRNAs with the help of ARWEN.</li>
<li>You can select the information contained within the files you wish to display via the <code>Settings --&gt; GenBank File Information Display</code>. Examples are: ID, organism, lineages, references, source, etc. (see <a href="#display">Information Display</a> section).</li>
<li>The IDs containing identical sequences (<span style="color: red">duplicates</span>) can be identified and automatically deleted using the <code>Highlight Identical Sequences</code> button (star-shaped, bottom&#x2F;right)</li>
<li>You can use the button adjacent to it, <code>Find Records by IDs</code>, to search for specific IDs.</li>
<li>Each ID can be opened with any text viewer program (Notepad for example) through the context menu, and then manually edited.</li>
<li>Selected IDs can be exported (context menu) as a GenBank (.gb) file, or a table (.csv) containing the information displayed in the GUI.</li>
<li>Selected IDs can be imported into a different work folder via drag-and-drop.</li>
</ul>
<h3 id="4-3-2-Other-types-of-files"><a href="#4-3-2-Other-types-of-files" class="headerlink" title="4.3.2. Other types of files"></a>4.3.2. Other types of files</h3><p>Similar to GenBank files, PhyloSuite provides two ways to import alignment files or Microsoft Word document files into the work folder under <strong>Other_File</strong> root folder:</p>
<ol>
<li>Using <code>Import file(s) or ID(s)</code> function under the <strong>File</strong> menu or <code>Open file(s)</code> in the <a href="#operation">display area</a>.</li>
<li>Drag-and-drop the files into the  <a href="#operation">display area</a>.</li>
</ol>
<p>After importing files, again there are many options to play with:</p>
<ul>
<li>For multiple sequence files, the number of sequences in the file and alignment status (aligned or non-aligned) will be displayed.</li>
<li>Files and sequences can be deleted, exported, or added.</li>
<li>The alignment can be directly used as input file for any (relevant) plug-in function: <code>MAFFT</code>, <code>Gblocks</code>, <code>Concatenate Sequence</code>, <code>Convert Sequence Format</code>, <code>Sequence Viewer</code>, <code>Partitionfinder2</code>, <code>ModelFinder</code>, <code>IQ-TREE</code>, and <code>MrBayes</code> functions.</li>
<li><code>Parse Annotation</code> function is available only for *.docx files.</li>
<li>All alignments could be managed in <code>Sequence Viewer</code>, they can be <em>reversed</em>, <em>complement</em>, <em>reverse complement</em> and <em>pruned</em>.</li>
</ul>
<p><span style="color: red">Note that FASTA format files can also be imported into any work folder under the GenBank_File root folder, in which case the file will be automatically converted to the GenBank file format (see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/five_ways_to_import/">five ways to import data into PhyloSuite</a > for details).</span></p>
<h3 id="4-3-3-Search-in-the-NCBI"><a href="#4-3-3-Search-in-the-NCBI" class="headerlink" title="4.3.3. Search in the NCBI"></a>4.3.3. Search in the NCBI</h3><p>You can search sequences from the NCBI’s Nucleotide and Protein databases via the <code>File --&gt; Search in NCBI</code> function.</p>
<h4 id="4-3-3-1-Brief-example"><a href="#4-3-3-1-Brief-example" class="headerlink" title="4.3.3.1. Brief example"></a>4.3.3.1. Brief example</h4><ol>
<li>   Open <code>File--&gt;Search in NCBI</code> in the menu bar;</li>
<li>   Enter keywords (<code>Monogenea[ORGN] AND (mitochondrion[TITL] OR mitochondrial[TITL]) AND 10000:50000[SLEN]</code>);</li>
<li>   Enter your email to tell NCBI who is downloading the sequences;</li>
<li>   Press Enter key or click search button to start searching;</li>
<li>   After the search is completed, select a work folder to deposit the sequences; selecting a work folder within the <code>GenBank_File</code> root folder will download sequences in the GenBank format, whereas selecting a work folder within the <code>Other_File</code> root folder will download sequences in the FASTA format;</li>
<li>   Click the <code>Download</code> button to start downloading.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/search_ncbi.jpg?raw=true"></li>
</ol>
<h2 id="4-4-GenBank-file-settings"><a href="#4-4-GenBank-file-settings" class="headerlink" title="4.4. GenBank file settings"></a>4.4. <em>GenBank file settings</em></h2><p>The format of GenBank file is show below (for detailed GenBank format please visit <a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html">https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html</a>):</p>
<pre>
FEATURES             Location/Qualifiers
    source[<span id="feature" style="color: red">Feature</span>] 1..5028
                    /organism[<span id="qualifier" style="color: red">Qualifier</span>]  ="Saccharomyces cerevisiae[<span id="value" style="color: red">Value or Name</span>]"
                    /db_xref[<span style="color: red">Qualifier</span>]   ="taxon:4932[<span style="color: red">Value or Name</span>]"
                    /chromosome[<span style="color: red">Qualifier</span>]="IX[<span style="color: red">Value or Name</span>]"
                    /map[<span style="color: red">Qualifier</span>]       ="9[<span style="color: red">Value or Name</span>]"
    CDS[<span style="color: red">Feature</span>]    1..206
                    /product[<span style="color: red">Qualifier</span>]   ="TCP1-beta[<span style="color: red">Value or Name</span>]"
</pre>

<h3 id="4-4-1-Lineage-recognition"><a href="#4-4-1-Lineage-recognition" class="headerlink" title="4.4.1. Lineage recognition"></a>4.4.1. Lineage recognition</h3><p><span id="Recognition">This</span> can be set in <code>Settings--&gt;Settings--&gt;Taxonomy Recognition</code>.<br>You can define the identifier of each <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Taxonomic_rank" title="taxonomic rank">taxonomic rank</a>. It supports the <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Wildcard_character" title="Wildcard character">wildcard character</a> <code>*</code>, for example, most of the family names end with “dae”, thus you can define the family taxonomic rank as <code>*dae</code>. However, in some taxonomic groups, rank names don’t necessarily follow the same rule, as for example in the Malacostraca, Hoplocarida and Peracarida both end with “carida”, but they are a subclass and a superorder respectively. In this case, you’d better use the full name as the identifier. Additionally, you can exclude terms when you use the wildcard character; for example, if you use <code>*oda</code> to recognize the orders in crustaceans, you will have to exclude Arthropoda by adding <code>-Arthropoda</code> in a new row of the order column, because it is a phylum.</p>
<p>&#8195;Taxonomic ranks can be added or deleted. For example, you can add a <code>Suborder</code> through the <code>Add Column</code> button, or delete any taxonomic rank by selecting it and clicking the <code>Delete Column</code> button. Please ensure that taxonomic ranks are arranged (left to right) from high to low level. You can change the order of taxonomic ranks by dragging them. Whenever you change the settings, you can update the table through the <code>Refresh table</code> option accessed via the right-click menu in the display area.</p>
<p>&#8195;Note that taxonomic lineages of each ID&#x2F;species listed in the <a href="#operation">display area</a> (GenBank_File workspace) are automatically recognized from GenBank files. If you wish, you may replace them with taxonomic data from the NCBI’s Taxonomy database (<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/taxonomy">https://www.ncbi.nlm.nih.gov/taxonomy</a>) or WoRMS database (<a target="_blank" rel="noopener" href="http://www.marinespecies.org/index.php">http://www.marinespecies.org/index.php</a>) via the context (right-click) menu of the selected ID&#x2F;species. </p>
<h3 id="4-4-2-Information-display-and-modification"><a href="#4-4-2-Information-display-and-modification" class="headerlink" title="4.4.2. Information display and modification"></a>4.4.2. Information display and modification</h3><p><span id="display">For</span> each work folder (of GenBank_File) you can define which information will be displayed. This can be set in <code>Settings--&gt;GenBank File Information Display</code> (you should select a work folder before) or click the <code>GenBank File Information Display Setting</code> button to the right of the name of each GenBank_File work folder.</p>
<p>There are numerous data in GenBank files that can be displayed, so PhyloSuite classifies them into four sections: Annotations, Lineages, Reference and Sources (<a href="#qualifier">qualifiers</a> from source feature).</p>
<table>
<thead>
<tr>
<th>Annotations</th>
<th>Lineages</th>
<th>Reference</th>
<th>Sources</th>
</tr>
</thead>
<tbody><tr>
<td>ID</td>
<td>Class</td>
<td>author(s)</td>
<td>host</td>
</tr>
<tr>
<td>Length</td>
<td>Order</td>
<td>title(s)</td>
<td>specimen_voucher</td>
</tr>
<tr>
<td>AT%</td>
<td>Superfamily</td>
<td>journal(s)</td>
<td>collection_date</td>
</tr>
<tr>
<td>Name</td>
<td>Family</td>
<td>pubmed ID(s)</td>
<td>isolation_source</td>
</tr>
<tr>
<td>Organism</td>
<td>Subfamily</td>
<td>comment(s)</td>
<td>country</td>
</tr>
<tr>
<td>Definition</td>
<td>Genus</td>
<td></td>
<td>collected_by</td>
</tr>
<tr>
<td>Date</td>
<td>…</td>
<td></td>
<td>organelle</td>
</tr>
<tr>
<td>Keywords</td>
<td></td>
<td></td>
<td>note</td>
</tr>
<tr>
<td>Molecule type</td>
<td></td>
<td></td>
<td>mol_type</td>
</tr>
<tr>
<td>Topology</td>
<td></td>
<td></td>
<td>strain</td>
</tr>
<tr>
<td>Accessions</td>
<td></td>
<td></td>
<td>db_xref</td>
</tr>
<tr>
<td>Sequence version</td>
<td></td>
<td></td>
<td>…</td>
</tr>
<tr>
<td>Source</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>Latest modified</td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody></table>
<p><span style="color: red">Note that the information for <code>Lineages</code> and <code>Sources</code> is variable. <code>Lineages</code> can be configured (see <a href="#Recognition">Lineage Recognition</a>). All of the qualifiers in the <code>source</code> <a href="#feature">feature</a> of GenBank files from all IDs in the <a href="#operation">work folder</a> will be presented as available options. In this way, the available options for <code>Source</code> are dependent on the GenBank files in this work folder.</span></p>
<p>Additionally, some of the information can be modified by double-clicking the corresponding cell. For example, there may be some errors in the lineage names, which you can correct in the <a href="#operation">display area</a>. The new name will then be used in <a href="#extracter">GenBank File Extracter</a> and other functions. Fields that cannot be modified are “ID”, “Length”, “AT%”, “State”, “Date”, “Latest modified”, “author(s)”, “title(s)”, “journal(s)”, “pubmed ID(s)”, “comment(s)”, “Keywords”, “Molecule type”, “Topology”, “Accessions”, “Sequence version”, “Source”, “State”.</p>
<h3 id="4-4-3-Features-extraction"><a href="#4-4-3-Features-extraction" class="headerlink" title="4.4.3. Features extraction"></a>4.4.3. Features extraction</h3><p><span id="extract">This</span> is a flexible function, details of which can be set in <code>Settings--&gt;GenBank File Extracting</code>.<br>There are three main steps:</p>
<ul>
<li>First, you can define which <a href="#feature">features</a> you wish to extract, such as <strong>CDS</strong>, <strong>rRNA</strong> and <strong>tRNA</strong> etc. </li>
<li>Second, you can define the value of a GenBank file <a href="#qualifier">qualifiers</a> that is to be used as the name of a feature. For example, you can select to extract only the value of the qualifier <code>product</code> for <strong>rRNA</strong>, and simultaneously select to extract the value of the qualifiers <code>gene</code> and <code>product</code> for <strong>CDS</strong>. In this case, PhyloSuite will first search the <a href="#value">value&#x2F;name</a> of <code>gene</code> for CDS, if there is no <code>gene</code> qualifier it will search the <a href="#value">value&#x2F;name</a> of <code>product</code> (<span style="color:red">note that qualifiers can be reordered by dragging</span>). If none of the specified qualifiers are found for features, it will be recorded in a table file when using the <a href="#extracter">GenBank File Extracter</a> function, or marked as an error when using the Standardization function.</li>
<li>Finally, you can uniformize the annotation of your dataset by replacing the <a href="#value">values&#x2F;names</a> searched in the previous step via the <code>Names unification</code> table. The ‘Old Name’ will be replaced with the ‘New Name’ if found in the corresponding qualifiers when using <a href="#extracter">GenBank File Extracter</a> or Standardization functions. If you wish to extract only a subset of features (genes) for which <a href="#value">value&#x2F;name</a> are available in this table, you may do so by checking the <code>Only extract these genes</code> checkbox. This table can be exported, or imported (<code>export/import settings</code> function below the table) from a comma-separated table (*.csv). There is a convenient way to uniformize names: if you extract genes without any settings for the first time, a “name_for_unification.csv” file will be generated, which can be used to set the new names and then imported into the settings (“csv” format is mandatory). <span style="color:red">Note that values&#x2F;names of all qualifiers of all features are included in a single table.</span></li>
</ul>
<p>By default, PhyloSuite provides settings for six data types (loci): Mitogenome, chloroplast genome, general, cox1, 16S and 18S. You may add more data types as desired and switch between them via the <code>Current version</code> button (bottom&#x2F;left). You are allowed to associate different settings with each data type. The three features (CDS, tRNA and rRNA) of the Mitogenome data type are fixed, so they cannot be deleted, but new features can be added. If you are not sure which data type to use, you can select <code>general</code> and then adjust the settings according to your needs.</p>
<h4 id="4-4-3-1-Brief-example"><a href="#4-4-3-1-Brief-example" class="headerlink" title="4.4.3.1. Brief example"></a>4.4.3.1. Brief example</h4><p>Please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/customize_extraction/">https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/customize_extraction&#x2F;</a>.</p>
<h2 id="4-5-File-operation"><a href="#4-5-File-operation" class="headerlink" title="4.5. File operation"></a>4.5. <em>File operation</em></h2><h3 id="4-5-1-Input-files"><a href="#4-5-1-Input-files" class="headerlink" title="4.5.1. Input files"></a>4.5.1. Input files</h3><p><span id="input">For</span> input files for the functions implemented in PhyloSuite, you can either allow the software to <strong>autodetect</strong> them from workplace or you can <strong>specify input files</strong> yourself. </p>
<h4 id="4-5-1-1-Autodetect-input-files"><a href="#4-5-1-1-Autodetect-input-files" class="headerlink" title="4.5.1.1. Autodetect input files"></a>4.5.1.1. Autodetect input files</h4><p><span id="autodetect">PhyloSuite</span> can autodetect and prepare input files for each function. For example, the <code>IQ-TREE</code> function accepts the results of <code>Concatenate Sequence (concatenate_results)</code>, <code>Partitionfinder2 (PartFind_results)</code>, <code>ModelFinder (ModelFinder_results)</code> and the <code>alignment file</code> in <code>Other_File</code>.<br>This function could be triggered in three ways:</p>
<ol>
<li>If you open <strong>IQ-TREE</strong> with the listed folders or alignment files selected, they will auto load to <strong>IQ-TREE</strong>.<br><br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE-autoinput.jpg?raw=true"></li>
<li>Every time when you open the <strong>IQ-TREE</strong>, PhyloSuite will search the entire workplace, and sort all acceptable input files for <strong>IQ-TREE</strong>.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE_en.jpg?raw=true"></li>
<li>If you are in the interface of the <strong>IQ-TREE</strong> without an input file, clicking on the input box shall open the selection from the step 2.</li>
</ol>
<p>The relationships of input files and functions are summarized below:</p>
<table>
<thead>
<tr>
<th>Function</th>
<th>Input Files</th>
</tr>
</thead>
<tbody><tr>
<td>IQ-TREE</td>
<td>concatenate_results, PartFind_results, ModelFinder_results and alignment file</td>
</tr>
<tr>
<td>MrBayes</td>
<td>PartFind_results, ModelFinder_results and alignment file</td>
</tr>
<tr>
<td>ModelFinder</td>
<td>concatenate_results and alignment file</td>
</tr>
<tr>
<td>PartitionFinder2</td>
<td>concatenate_results</td>
</tr>
<tr>
<td>MAFFT</td>
<td>extract_results and alignment file</td>
</tr>
<tr>
<td>MACSE</td>
<td>mafft_results, extract_results and alignment file</td>
</tr>
<tr>
<td>Gblocks</td>
<td>MACSE_results, mafft_results, concatenate_results and alignment file</td>
</tr>
<tr>
<td>trimAl</td>
<td>MACSE_results, mafft_results, concatenate_results and alignment file</td>
</tr>
<tr>
<td>HmmCleaner</td>
<td>MACSE_results, mafft_results, concatenate_results and alignment file</td>
</tr>
<tr>
<td>Convert Format</td>
<td>MACSE_results, mafft_results, Gblocks_results, trimAl_results, HmmCleaner_results and alignment file</td>
</tr>
<tr>
<td>Concatenate Sequence</td>
<td>MACSE_results, mafft_results, Gblocks_results, trimAl_results, HmmCleaner_results and alignment file</td>
</tr>
</tbody></table>
<p>Alignment file here refers to the alignment files listed in the <strong>Other_File</strong> root folder.<br>For results folder names refer to <a href="#Output">Output Files</a></p>
<h4 id="4-5-1-2-Specify-input-files"><a href="#4-5-1-2-Specify-input-files" class="headerlink" title="4.5.1.2. Specify input files"></a>4.5.1.2. Specify input files</h4><p>There are two ways:</p>
<ol>
<li>Drag files into the “Input” box;</li>
<li>Click the ‘open folder’ icon to the right of the input box.</li>
</ol>
<h3 id="4-5-2-Output-files"><a href="#4-5-2-Output-files" class="headerlink" title="4.5.2. Output files"></a>4.5.2. Output files</h3><p><span id="Output">The</span> results of all the functions will be automatically saved in the workplace. If you have selected a work folder, then the results will be saved to that work folder. If you haven’t selected one, the results will be saved to <code>GenBank_File/files</code> or <code>Other_File/files</code>. You may also change the results folder name, as well as select another work folder to deposit your results, via the down-arrow of the <code>Start</code> button.</p>
<p>Functions and default results folders:</p>
<table>
<thead>
<tr>
<th>Function</th>
<th>Results folder</th>
</tr>
</thead>
<tbody><tr>
<td>IQ-TREE</td>
<td>IQtree_results</td>
</tr>
<tr>
<td>MrBayes</td>
<td>MrBayes_results</td>
</tr>
<tr>
<td>ModelFinder</td>
<td>ModelFinder_results</td>
</tr>
<tr>
<td>PartitionFinder2</td>
<td>PartFind_results</td>
</tr>
<tr>
<td>MAFFT</td>
<td>mafft_results</td>
</tr>
<tr>
<td>MACSE</td>
<td>MACSE_results</td>
</tr>
<tr>
<td>Gblocks</td>
<td>Gblocks_results</td>
</tr>
<tr>
<td>trimAl</td>
<td>trimAl_results</td>
</tr>
<tr>
<td>HmmCleaner</td>
<td>HmmCleaner_results</td>
</tr>
<tr>
<td>Convert Format</td>
<td>convertFmt_results</td>
</tr>
<tr>
<td>Concatenate Sequence</td>
<td><span id="concatenate_results">concatenate_results</span></td>
</tr>
<tr>
<td>Draw RSCU figure</td>
<td>RSCUfig_results</td>
</tr>
<tr>
<td>Compare Table</td>
<td>comp_tbl_results</td>
</tr>
<tr>
<td>Flowchart</td>
<td>Flowchart_reports</td>
</tr>
</tbody></table>
<h4 id="4-5-2-1-Brief-example"><a href="#4-5-2-1-Brief-example" class="headerlink" title="4.5.2.1. Brief example"></a>4.5.2.1. Brief example</h4><p>Please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/quick_start/#7-Get-the-results">here</a>. </p>
<h1 id="5-Data-analysis"><a href="#5-Data-analysis" class="headerlink" title="5. Data analysis"></a>5. Data analysis</h1><h2 id="5-1-Extract-GenBank-file"><a href="#5-1-Extract-GenBank-file" class="headerlink" title="5.1. Extract GenBank file"></a>5.1. <em>Extract GenBank file</em></h2><p><span id="extracter">The</span> input file for this function can be loaded only by selecting IDs in the <a href="#operation">display area</a> of the work folder under the <a href="#operation">GenBank_File root folder</a>. For the results, please see the <a href="#Output">Output Files</a> section.</p>
<p>There are two modes for extraction, <code>Single loc.</code> mode will extract the entire sequence but ignore annotation and other features, which is suitable for single locus, such as 18S, cox1 and 28S etc.; <code>Custom</code> mode allows you to select or edit the type of sequence and features that you wish to extract (see <a href="#extract">GenBank File Extracting settings</a>).</p>
<p>What it can do:</p>
<ul>
<li>Extract genes defined (selected) in the <a href="#extract">GenBank File Extracting settings</a> and save them in the fasta format. For example, if you select to extract <code>CDS</code>, <code>tRNA</code> and <code>rRNA</code> <a href="#feature">features</a>, this function will extract these features from all selected GenBank files and store them in correspondingly named folders (CDS, tRNA, rRNA). Additionally, <code>CDS</code> feature will be split into two folders: ‘CDS_AA’ folder contains the amino acid sequences extracted from the <code>translation</code> <a href="#qualifier">qualifier</a>, whereas ‘CDS_NUC’ folder contains the nucleotide sequences. For the <a href="#extract">Mitogenome</a> version, there is an additional “self-translated_AA” folder that contains the amino acid sequences translated from the nucleotide sequences (CDS) by the PhyloSuite. <span style="color:red">Note that there may exist duplicated genes within one ID, in which case PhyloSuite will number the duplicated gene names in the order they occur. For example, if there are three <code>cox1</code> genes, then they will be saved as <code>cox1.fas</code>, <code>cox1_copy2.fas</code> and <code>cox1_copy3.fas</code></span>. Additionaly, PhyloSuite also provides <code>Resolve gene duplicates</code> function (available in <code>Parameters</code> tab) to automatically identify and remove duplicated genes.</li>
<li>Extract overlapping and intergenic regions.</li>
<li>Generate statistics files and files used for other analyses: <ul>
<li>Generates an extraction overview file (<code>overview.csv</code>), which records the data type settings used for extraction, all features found in the sequences, missing features or qualifiers, and genes found in each species.</li>
<li>The information about the species (IDs) included in the dataset, including organism name, lineages, A&#x2F;T&#x2F;C&#x2F;G content, and AT&#x2F;GC skewness. [<span style="color: red">StatFiles&#x2F;used_species.csv</span>]</li>
<li>A name table for editing the <code>Names unification</code> table in the <a href="#extract">GenBank File Extracting settings</a>. Using this table, you can modify the names in the ‘New Name’ column and then import it into the <code>Names unification</code> table. This table is extremely useful when extracting genes for the first time. [<span style="color: red">StatFiles&#x2F;name_for_unification.csv</span>] </li>
<li>If <code>Only extract these genes</code> is checked and none of the qualifier values conform to the name in <code>Names unification</code> table, then these values will be recorded in the <code>name_not_included.csv</code> table. [<span style="color: red">StatFiles&#x2F;name_not_included.csv</span>]</li>
<li>Overall statistics of the mitogenome, including nucleotide composition of the whole genome, protein-coding genes (PCGs), rRNA genes and tRNA genes. [<span style="color: red">StatFiles&#x2F;used_species.csv, Mitogenome version</span>]</li>
<li>Initial and stop codon, nucleotide content, skewness as well as length statistics for each PCG and rRNA genes. [<span style="color: red">StatFiles&#x2F;geneStat.csv, Mitogenome version</span>]</li>
<li>Nucleotide skewness for each codon site of PCGs. [<span style="color: red">StatFiles&#x2F;CDS&#x2F;[PCGsCodonSkew.csv | firstCodonSkew.csv | secondCodonSkew.csv | thirdCodonSkew.csv], Mitogenome version</span>]</li>
<li>Nucleotide content and skewness of individual elements and the complete mitogenome of all species (IDs) (see Fig. 2 in <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1</a> and Fig. 1 in <a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12862-018-1249-3">https://doi.org/10.1186/s12862-018-1249-3</a>) [<span style="color: red">StatFiles&#x2F;geom_line.csv, Mitogenome version</span>]</li>
<li>Nucleotide statistics of each species (IDs). [<span style="color: red">StatFiles&#x2F;speciesStat&#x2F;*IDs.csv, Mitogenome version</span>]</li>
<li>Organization table for each species (IDs). [<span style="color: red">StatFiles&#x2F;speciesStat&#x2F;*IDs_org.csv, Mitogenome version</span>]</li>
<li>Relative synonymous codon usage table. Note that the abbreviated stop codons (T–, TA-) are removed before the calculation.  [<span style="color: red">StatFiles&#x2F;RSCU&#x2F;*IDs_RSCU.csv, Mitogenome version</span>]</li>
<li>Amino acid usage table. [<span style="color: red">StatFiles&#x2F;RSCU&#x2F;*IDs_AA_usage.csv, Mitogenome version</span>]</li>
</ul>
</li>
<li>Making <a target="_blank" rel="noopener" href="https://itol.embl.de/" title="ITOL website">ITOL</a> datasets (will be activated if you check the <code>ITOL datasets</code> checkbox)<ul>
<li>These are simple *.txt files that you can directly drag-and-drop onto corresponding dendrograms in the iTOL web interface (<a target="_blank" rel="noopener" href="https://itol.embl.de">https://itol.embl.de</a>)</li>
<li>Replacing tip labels in batch. [<span style="color: red">itolFiles&#x2F;[itol_labels.txt | itol_gb_labels.txt | itol_ori_labels.txt]</span>]</li>
<li>Assigning colors to different lineages. Colors for each taxon (or lineage) can be specified in <code>Lineage color</code>. If you don’t select colors for all taxons, PhyloSuite will randomly assign colors to the remaining taxons. For adding or removing lineages, please click <code>Configure</code> button (see Fig. 1 in <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0181699">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0181699</a>). [<span style="color: red">itolFiles&#x2F;[itol_xxx_ColourStrip.txt | itol_xxx_Text.txt | itol_xxx_Colour.txt]</span>]</li>
<li>Mapping histogram to the tree (see Fig. 2 in <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2245-y">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2245-y</a>). [<span style="color: red">itolFiles&#x2F;[itolAT.txt | itolLength.txt | itolLength_stack.txt]</span>]</li>
<li>Mapping gene order to the tree. The color, length and shape of each gene icon, as well as the space between the icons (Gene Interval), for the gene order display can be modified using the <code>Gene order display</code> function. At this step you can also select NCRs to be visualized (if you have set up the PhyloSuite to recognize and extract them, including setting the size threshold, during the <code>Standardization</code> step) (see Fig. 6 in <a target="_blank" rel="noopener" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3">https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3</a>). [<span style="color: red">files&#x2F;itol_gene_order.txt, Mitogenome version</span>]</li>
</ul>
</li>
<li>Gene order file which can be used to conduct relative analysis using CREx and&#x2F;or treeREx.</li>
</ul>
<p>In the <code>Custom</code> menu, you can choose among the data types pre-set in <a href="#extract">GenBank File Extracting settings</a>. In the <code>Lineages</code> menu, you can choose which lineages to include in the results. Regarding the names of sequences, user can customize them via <code>Name Type</code> function, in which ID, organism, Family, Class, isolate, strain, etc. are available.</p>
<h3 id="5-1-1-Brief-example"><a href="#5-1-1-Brief-example" class="headerlink" title="5.1.1. Brief example"></a>5.1.1. <span id="extract_exam">Brief</span> example</h3><ol>
<li>Select IDs to extract (refer to <a href="#import_gb">this</a> to see how to import GenBank records into PhyloSuite);</li>
<li>Open <code>Extract</code> via right-click, and the sequences will be imported automatically;</li>
<li>Parameters can be set according to your own needs (if your data are mitochondrial genomes, select <code>Mitogenome</code> data-type);</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/extract_demo.jpg?raw=true"></li>
</ol>
<p>For customizing the extraction, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/customize_extraction/">Customizing the extraction</a ><br>For comprehensive demos, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">multi-gene tutorial</a > and <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">single-gene tutorial</a >. For how to use the generated iTOL datasets, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/#10-Phylogenetic-Tree-Annotation">phylogenetic tree annotation</a >.</p>
<h2 id="5-2-MAFFT"><a href="#5-2-MAFFT" class="headerlink" title="5.2. MAFFT"></a>5.2. <em>MAFFT</em></h2><p>For installation of MAFFT, please see <a href="#plugins">Plugins Installation</a> section. For input files for MAFFT, please see <a href="#input">Input Files</a> section. <span style="color: red">Note that the input file should be in the FASTA format.</span> For the results of MAFFT, please see <a href="#Output">Output Files</a> section.</p>
<p>PhyloSuite enables MAFFT to <strong>run multiple files in batches</strong> using the same set of parameters, which means that you can input multiple files into MAFFT simultaneously. PhyloSuite provides three alignment modes for MAFFT:</p></p>
<ol>
<li>Normal mode: align sequence normally.</li>
<li>Codon mode (<span style="color: green">added by PhyloSuite</span>): the nucleotide sequences of protein-coding genes are translated into AA sequences first, then the AA sequences are aligned by MAFFT, finally the AA alignments are back-translated into corresponding codons. <span style="color: red">Note that you should choose a proper code table first.</span></li>
<li>N2P mode (<span style="color: green">added by PhyloSuite</span>) is identical to the previous mode minus the last step (back-translation): PCGs are translated into AAs and aligned. The result is the AA alignment.</li>
</ol>
<p>When aligning with codon mode, if there are internal stop codons, PhyloSuite will pop up a warning window. If you are aware of this problem, feel free to ignore it and continue the alignment (select ‘Ignore’), otherwise terminate the alignment and inspect the problem (select ‘Yes’). <code>-adjustdirection</code> can auto-adjust the direction of some sequences (i.e. reverse complement). Other parameters are also available, such as <code>align strategy</code>, <code>export format</code> and <code>thread</code> etc. </p>
<p>After inputting files and setting parameters, you are ready to click the <code>Start</code> button and start the program. The run log can be viewed through <code>Show log</code> button. Once the program is finished, the parameter settings and the citation of <code>MAFFT</code> will be saved in the <code>summary.txt</code> file.</p>
<h3 id="5-2-1-Brief-example"><a href="#5-2-1-Brief-example" class="headerlink" title="5.2.1. Brief example"></a>5.2.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\MAFFT\mtDNA_36_genes\CDS_NUC’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select all 12 files;</li>
<li>Open <code>Alignment--&gt;MAFFT</code> through the menu bar;</li>
<li>Drag all 12 sequences into the file input box;</li>
<li>Parameters can be set according to your own needs (make sure to select the correct <code>Code Table</code> for protein-coding genes, here is 9);</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mafft_demo.jpg?raw=true"></li>
</ol>
<p>For comprehensive demos, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">multi-gene tutorial</a > and <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">single-gene tutorial</a >. For a comprehensive manual of MAFFT, please visit <a target="_blank" rel="noopener" href="https://mafft.cbrc.jp/alignment/software/manual/manual.html">https://mafft.cbrc.jp/alignment/software/manual/manual.html</a>.</p>
<h2 id="5-3-MACSE"><a href="#5-3-MACSE" class="headerlink" title="5.3. MACSE"></a>5.3. <em>MACSE</em></h2><p>For installation of MACSE, please see <a href="#plugins">Plugins Installation</a> section. For input files for MACSE, please see <a href="#input">Input Files</a> section. <span style="color: red">Note that the input file should be in the FASTA format.</span> For the results of MACSE, please see <a href="#Output">Output Files</a> section.</p>
<p>PhyloSuite enables MACSE to <strong>run multiple files in batches</strong> using the same set of parameters, which means that you can input multiple files into MACSE simultaneously. In addition, multi-core operation is also allowed, which allows several files (depends on threads set) to run simultaneously.</p></p>
<p>MACSE has many subprograms and it already has a GUI, so we only added <code>alignSequences</code> and <code>refineAlignment</code> subprograms to PhyloSuite. We believe these two are most suitable for PhyloSuite, in terms of complementing the shortcomings of MAFFT. The input of MACSE should be either protein-coding sequences (for <code>alignSequences</code>) or an alignment (for <code>refineAlignment</code>) generated by other programs (e.g. <code>MAFFT</code>). Regarding batch processing, if there are multiple files in both <code>Seq.</code> and <code>Seq_lr.</code> boxes, these files will be combined successively, for example, the first file of <code>Seq.</code> (-seq 1st_seq_file) will combine with the first file of <code>Seq_lr.</code> (-seq_lr 1st_seq_lr_file). Noteworthy, the <code>seq</code> and <code>seq_lr</code> options must be used together or not at all in combination with <code>Refine</code> (refineAlignment). Please note that PhyloSuite also provides the <code>View | Edit command</code> function in the dropdown arrow of the ‘Start’ button, which gives sufficient freedom for experienced users to modify and add parameters that are not included in the GUI.</p>
<p>In particular, as the generated alignment files may contain exclamation (<code>!</code>) or star (<code>*</code>) symbols (emphasize the frameshifts detected by MACSE), which may cause errors in downstream analyses. Therefore, PhyloSuite generates an additional file with <code>_removed_chars_</code> in its name, which replaces these symbols with <code>?</code>.</p>
<p>After inputting files and setting parameters, you are ready to click the <code>Start</code> button and start the program. The run log can be viewed through <code>Show log</code> button. Once the program is finished, the parameter settings and the citation of <code>MACSE</code> will be saved in the <code>summary.txt</code> file.</p>
<h3 id="5-3-1-Brief-example"><a href="#5-3-1-Brief-example" class="headerlink" title="5.3.1. Brief example"></a>5.3.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\MACSE’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select all 3 files;</li>
<li>Open <code>Alignment--&gt;MACSE (for CDS)</code> through the menu bar;</li>
<li>Drag all 3 sequences into the <code>Seq.</code> input box;</li>
<li>Parameters can be set according to your own needs (make sure to select the correct <code>Code Table</code>; in the example we selected 9);</li>
<li>parameters that are not included in the GUI can be added via the <code>View | Edit command</code> function in the dropdown arrow of the ‘Start’ button;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/MACSE_demo.jpg?raw=true"></li>
</ol>
<p>For a comprehensive manual of MACSE, please visit <a target="_blank" rel="noopener" href="https://bioweb.supagro.inra.fr/macse/index.php?menu=intro">https://bioweb.supagro.inra.fr/macse/index.php?menu=intro</a>.</p>
<h2 id="5-4-trimAl"><a href="#5-4-trimAl" class="headerlink" title="5.4. trimAl"></a>5.4. <em>trimAl</em></h2><p><span id="trimAl">For</span> the installation of trimAl, please see <a href="#plugins">Plugins Installation</a> section. For inputting files into trimAl, please see <a href="#input">Input Files</a> section. </span> For the results of trimAl, please see <a href="#Output">Output Files</a> section.</p>
<p>PhyloSuite enables trimAl to <strong>run multiple files in batches</strong> using the same set of parameters, which means you can input multiple files into trimAl simultaneously. In addition, multi-core operation is also allowed, which allows several files (depends on threads set) to run simultaneously. Please note that PhyloSuite also provides the <code>View | Edit command</code> function in the dropdown arrow of the ‘Start’ button, which gives sufficient freedom for experienced users to modify and add parameters that are not included in the GUI.</p></p>
<p>If you want to apply the results of trimAl to downstream analyses, please ensure that you select <code>fasta</code> as output format. If you select the <code>Statistics output</code>, these results will be saved to a file with the suffix “.log”. As the output file extension <code>_trimAl</code> is recognized by downstream functions, it cannot be changed.</p>
<p>After inputting files and setting parameters, clicking the <code>Start</code> button runs the program. You can view the running log through the <code>Show log</code> button. Parameter settings and citation for <code>trimAl</code> will be saved in the <code>summary.txt</code> file. </p>
<h3 id="5-4-1-Brief-example"><a href="#5-4-1-Brief-example" class="headerlink" title="5.4.1. Brief example"></a>5.4.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\trimAl_HmmCleaner’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select all 3 files;</li>
<li>Open <code>Alignment--&gt;trimAl</code> through the menu bar;</li>
<li>Drag all 3 sequences into the input box;</li>
<li>Parameters can be set according to your own needs;</li>
<li>Parameters that are not included in the GUI can be added via the <code>View | Edit command</code> function in the dropdown arrow of the ‘Start’ button;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/trimAl_demo.jpg?raw=true"></li>
</ol>
<p>For a comprehensive manual of trimAl, please visit <a target="_blank" rel="noopener" href="http://trimal.cgenomics.org">http://trimal.cgenomics.org</a>.</p>
<h2 id="5-5-HmmCleaner"><a href="#5-5-HmmCleaner" class="headerlink" title="5.5. HmmCleaner"></a>5.5. <em>HmmCleaner</em></h2><p><span id="HmmCleaner">For</span> the installation of HmmCleaner, please see <a href="#plugins">Plugins Installation</a> section. For inputting files into HmmCleaner, please see <a href="#input">Input Files</a> section. For the results of HmmCleaner, please see <a href="#Output">Output Files</a> section.</p>
<p>PhyloSuite enables HmmCleaner to <strong>run multiple files in batches</strong> using the same set of parameters, which means you can input multiple files into HmmCleaner simultaneously. In addition, multi-core operation is also allowed, which allows several files (depends on threads set) to run simultaneously. Due to HmmCleaner design contraints, this program is available only to linux and mac users. If you want to apply the results of HmmCleaner to downstream analyses, please ensure that you uncheck the <code>ali</code> output format.</p>
<p>After inputting files and setting parameters, clicking the <code>Start</code> button runs the program. You can view the running log through the <code>Show log</code> button. The parameter settings and the citation for <code>HmmCleaner</code> will be saved in the <code>summary.txt</code> file. </p>
<h3 id="5-5-1-Brief-example"><a href="#5-5-1-Brief-example" class="headerlink" title="5.5.1. Brief example"></a>5.5.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\trimAl_HmmCleaner’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select all 3 files;</li>
<li>Open <code>Alignment--&gt;HmmCleaner</code> through the menu bar;</li>
<li>Drag all 3 sequences into the input box;</li>
<li>Parameters can be set according to your own needs;</li>
<li>parameters that are not included in the GUI can be added via the <code>View | Edit command</code> function in the dropdown arrow of the ‘Start’ button;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/HmmCleaner_demo.jpg?raw=true"></li>
</ol>
<p>For a comprehensive manual of HmmCleaner, please visit <a target="_blank" rel="noopener" href="https://metacpan.org/pod/distribution/Bio-MUST-Apps-HmmCleaner/bin/HmmCleaner.pl">https://metacpan.org/pod/distribution/Bio-MUST-Apps-HmmCleaner/bin/HmmCleaner.pl</a>.</p>
<h2 id="5-6-Gblocks"><a href="#5-6-Gblocks" class="headerlink" title="5.6. Gblocks"></a>5.6. <em>Gblocks</em></h2><p><span id="gblocks">For</span> the installation of Gblocks, please see <a href="#plugins">Plugins Installation</a> section. For inputting files into Gblocks, please see <a href="#input">Input Files</a> section. <span style="color: red">Note that the input files should be in FASTA or NBRF&#x2F;PIR formats.</span> For the results of Gblocks, please see <a href="#Output">Output Files</a> section.</p>
<p>PhyloSuite enables Gblocks to <strong>run multiple files in batches</strong> using the same set of parameters, which means you can input multiple files into Gblocks simultaneously.<br>The two options, <code>Minimum Number Of Sequences For A Conserved Position</code> and <code>Minimum Number Of Sequences For A Flank Position</code> will be enabled after inputting files. As the former variable has to be &gt; half the number of sequences, whereas the latter variable has to be ≥ the value of former variable. The available values of these two variables will change according to this rule.<span style="color: red">Because of this, when running batch analyses on multiple files, the number of sequences in each file must be the same.</span> </p>
<p>As the default output file extension ‘_gb’ is recognized by downstream functions, it cannot be changed.</p>
<p>After inputting files and setting parameters, clicking the <code>Start</code> button runs the program. You can view the running log through the <code>Show log</code> button. The parameter settings and the citation for <code>Gblocks</code> will be saved in the <code>summary.txt</code> file. </p>
<h3 id="5-6-1-Brief-example"><a href="#5-6-1-Brief-example" class="headerlink" title="5.6.1. Brief example"></a>5.6.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\Gblocks\mtDNA_36_genes\CDS_NUC’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select all 12 files;</li>
<li>Open <code>Alignment--&gt;Gblocks</code> through the menu bar;</li>
<li>Drag all 12 sequences into the file input box;</li>
<li>Parameters can be set according to your own needs (ensure you choose proper data types, here it should be codons);</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/gblocks_demo.jpg?raw=true"></li>
</ol>
<p>For a comprehensive manual of Gblocks, please visit <a target="_blank" rel="noopener" href="http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_documentation.html">http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_documentation.html</a>.</p>
<h2 id="5-7-Concatenate-Sequences"><a href="#5-7-Concatenate-Sequences" class="headerlink" title="5.7. Concatenate Sequences"></a>5.7. <em>Concatenate Sequences</em></h2><p>For inputting files, please see <a href="#input">Input Files</a> section. FASTA, PHYLIP, AXT, PAML and NEXUS formats are allowed. For the results, please see <a href="#Output">Output Files</a> section.  <span style="color: red">Note that the name of the output file can be changed.</span></p>
<p>Alignments can be concatenated into a single alignment using this function. First, PhyloSuite will scan each of the alignments and collect all of the sequence names, then it will concatenate these alignments by searching the names in each alignment. If a name can’t be found in the alignment, it will be recorded in the ‘missing_genes.txt’ file.</p>
<p>A number of common formats can be chosen for the output file, such as PHYLIP, NEXUS, AXT, PAML and FASTA. Additionally, the function can record the index of each gene during the concatenation and generate a partition file, which can be used in <code>PartitionFinder</code>, <code>ModelFinder</code>, <code>IQ-TREE</code> and <code>MrBayes</code>. User can also select to remove any codon sites (such as third codon site) in this analysis.</p>
<p>You can change the order in which alignments are concatenated by dragging the files to reorder them.</p>
<h3 id="5-7-1-Brief-example"><a href="#5-7-1-Brief-example" class="headerlink" title="5.7.1. Brief example"></a>5.7.1. Brief example</h3><p><span id="concatenation">When</span> you are in the PhyloSuite root folder, go to ‘example\Concatenation\mtDNA_36_genes\36_genes_NUC’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select all 36 files;</li>
<li>Open <code>Alignment--&gt;Concatenate Sequence</code> through the menu bar;</li>
<li>Drag all the sequences into the file input box;</li>
<li>Output formats could be selected according to your own needs, you may also select <code>Linear figure</code> function if you wish to visualize the concatenated dataset;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/concatenation_demo.jpg?raw=true"></li>
</ol>
<p>For comprehensive demos, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">multi-gene tutorial</a > and <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">single-gene tutorial</a >.</p>
<h2 id="5-8-Convert-format"><a href="#5-8-Convert-format" class="headerlink" title="5.8. Convert format"></a>5.8. <em>Convert format</em></h2><p>For inputting files, please see <a href="#input">Input Files</a> section. For the results, please see <a href="#Output">Output Files</a> section.</p>
<p>PHYLIP, NEXUS, AXT, PAML and FASTA formats are supported (both for input and output files). This function also supports <strong>batch format conversion</strong>, which means that you can input multiple files simultaneously.</p>
<h3 id="5-8-1-Brief-example"><a href="#5-8-1-Brief-example" class="headerlink" title="5.8.1. Brief example"></a>5.8.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\Convert_format’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select ‘cox1_AA_mafft.fas’ and ‘cox1_NUC_mafft.fas’;</li>
<li>Open <code>Alignment--&gt;Convert Sequence Format</code> through the menu bar;</li>
<li>Drag them into the file input box;</li>
<li>Select output formats;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/convertFMT_demo.jpg?raw=true"></li>
</ol>
<h2 id="5-9-ModelFinder"><a href="#5-9-ModelFinder" class="headerlink" title="5.9. ModelFinder"></a>5.9. <em>ModelFinder</em></h2><p>For the installation of ModelFinder (IQ-TREE), please see <a href="#plugins">Plugins Installation</a> section, for input alignment files see <a href="#input">Input Files</a> section (FASTA, PHYLIP, NEXUS and CLUSTAL formats are allowed), and for the result files see <a href="#Output">Output Files</a> section.</p>
<p>You may choose to provide two more optional files: a tree file (<span style="color: red">newick format</span>) and a partition file. Please see <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/Advanced-Tutorial">http://www.iqtree.org/doc/Advanced-Tutorial</a> for the format of the partition file. The most convenient option is to directly use the results of <code>Concatenate Sequence</code> (<a href="#concatenate_results">concatenate_results</a>) as input files for ModelFinder. The concatenated alignments and the partition file will load to ModelFinder automatically (see brief example below).</p>
<p>PhyloSuite provides an additional parameter for ModelFinder settings: <code>Model for</code>. This parameter allows you to select a set of models you wish to test, suited for different phylogenetic programs (see table below). This is very useful, as different algorithms often use different model types.</p>
<table>
<thead>
<tr>
<th>Options</th>
<th>Corresponding arguments in ModelFinder</th>
</tr>
</thead>
<tbody><tr>
<td>MrBayes</td>
<td>-m TESTONLY -mset mrbayes</td>
</tr>
<tr>
<td>RaxML</td>
<td>-m TESTONLY -mset raxml</td>
</tr>
<tr>
<td>PhyML</td>
<td>-m TESTONLY -mset phyml</td>
</tr>
<tr>
<td>IQ-TREE</td>
<td>-m TESTNEWONLY</td>
</tr>
<tr>
<td>BEAST1</td>
<td>-mset JC69,TrN,TrNef,K80,K2P,F81,HKY,SYM,TIM,TVM,TVMef,GTR -mrate E,G</td>
</tr>
<tr>
<td>BEAST2</td>
<td>-mset JC69,TrN,TrNef,K80,K2P,F81,HKY,SYM,TIM,TVM,TVMef,GTR -mrate E,G</td>
</tr>
</tbody></table>
<p>After inputting files and setting the parameters, you may click <code>Start</code> button and run the program. The running log can be viewed through <code>Show log</code> button. Once the program is finished, the parameter settings and the citation for <code>IQ-TREE</code> will be saved in the <code>summary.txt</code> file.</p>
<h3 id="5-9-1-Brief-example"><a href="#5-9-1-Brief-example" class="headerlink" title="5.9.1. Brief example"></a>5.9.1. Brief example</h3><ol>
<li>Right click a result in <code>concatenate_results</code> folder (if not available, please see <a href="#concatenation">here</a> for how to make one), then select <code>Import to ModelFinder</code> in the context menu;</li>
<li>The concatenated dataset with the position index of each gene will be automatically imported;</li>
<li>Double-click the text box or click the edit button to open the partition editor window to configure <code>data blocks</code> (for how to operate partition editor, please see <a href="#partition_editor">below</a>);</li>
<li>Parameters can be set according to your own needs; parameters that are not included in the GUI can be added using the <code>View | Edit command</code> function in the dropdown arrow of the ‘Start’ button;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/modelfinder_demo_2.jpg?raw=true"></li>
</ol>
<p>For comprehensive demos, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">multi-gene tutorial</a > and <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">single-gene tutorial</a >. For a comprehensive manual of ModelFinder, please visit <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/">http://www.iqtree.org/doc/</a> and <a target="_blank" rel="noopener" href="http://iqtree.cibiv.univie.ac.at/">http://iqtree.cibiv.univie.ac.at/</a>.</p>
<h2 id="5-10-PartitionFinder"><a href="#5-10-PartitionFinder" class="headerlink" title="5.10. PartitionFinder"></a>5.10. <em>PartitionFinder</em></h2><p>For installation of PartitionFinder2, please see <a href="#plugins">Plugins Installation</a> section, for inputting the alignment file (<span style="color: red">PHYLIP format</span>) see <a href="#input">Input Files</a> section, for the results see <a href="#Output">Output Files</a> section. You may provide a tree file as well (<span style="color: red">optional, newick format</span>).</p>
<p>The most convenient way to use PartitionFinder2 is to use the results of <code>Concatenate Sequence</code> (<a href="#concatenate_results">concatenate_results</a>) as input files. The concatenated alignments and the partition file will load into the PartitionFinder2 automatically. </p>
<p>PartitionFinder2 requires a data block to run, the default format of which is (see <code>DATA BLOCKS</code> window):</p>
<blockquote>
<p>Gene1_codon1 &#x3D; 1-999\3;<br>Gene1_codon2 &#x3D; 2-999\3;<br>Gene1_codon3 &#x3D; 3-999\3;<br>Gene2 &#x3D; 1000-1665;<br>intron &#x3D; 1666-2000;</p>
</blockquote>
<p>PhyloSuite provides a partition editor function, in which you can add&#x2F;delete&#x2F;modify partitions  and convert&#x2F;cancel the selected data block to the codon format. For how to use this function, please see <a href="#partition_editor">below</a>.</p>
<p>Note that among the <code>Command line options</code>, <code>--all-states</code> and <code>--min-subset-size</code> can only be used when <code>kmeans</code> is selected in the <code>search</code> menu. The options <code>hcluster</code>, <code>rclusterf</code> and <code>rcluster</code> in <code>search</code> menu as well as <code>--rcluster-max</code> and <code>--weights</code> in the <code>Command line options</code> will be enabled only when <code>--raxml</code> is checked in the <code>Command line options</code>. The <code>unlinked</code> option in the <code>branchlengths</code> menu should be used with caution, as it may hinder convergence when using the partition results to conduct an analysis in MrBayes (because of <code>unlink brlens=(all);</code>). </p>
<p>After inputting files and setting the parameters, you can start the program (<code>Start</code> button), and view the run log through the <code>Show log</code> button. Once the program is finished, the parameter settings and the citation for <code>PartitionFinder2</code> will be saved in the <code>summary.txt</code> file.</p>
<h3 id="5-10-1-Brief-example"><a href="#5-10-1-Brief-example" class="headerlink" title="5.10.1. Brief example"></a>5.10.1. Brief example</h3><p>One design feature of PhyloSuite is a direct link between the outputs of <code>Concatenation</code> and the inputs of <code>PartitionFinder2</code>:</p>
<ol>
<li>Right click a result in <code>concatenate_results</code> folder (if not available, please see <a href="#concatenation">here</a> for how to make one), then select <code>Import to PartitionFinder2</code> in the context menu;</li>
<li>The concatenated dataset with the position index of each gene will be automatically imported;</li>
<li>Double-click the text box or click the edit button to open the partition editor window to configure <code>data blocks</code> (for how to operate partition editor, please see <a href="#partition_editor">below</a>);</li>
<li>Other parameters can be set according to your own needs (make sure you choose proper data types);</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/pf_demo_2.jpg?raw=true"></li>
</ol>
<p>For a comprehensive demo, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">multi-gene tutorial</a >. For a comprehensive manual of PartitionFinder2, please visit <a target="_blank" rel="noopener" href="http://www.robertlanfear.com/partitionfinder/assets/Manual_v2.1.x.pdf">http://www.robertlanfear.com/partitionfinder/assets/Manual_v2.1.x.pdf</a>.</p>
<h3 id="5-10-2-Brief-tutorial-for-partition-editor"><a href="#5-10-2-Brief-tutorial-for-partition-editor" class="headerlink" title="5.10.2. Brief tutorial for partition editor"></a>5.10.2. <span id="partition_editor">Brief tutorial for partition editor</span></h3><ol>
<li>The number <code>3</code> shown to the left of the data block name indicates that the length of the sequence is a multiple of 3, select one or more data blocks (make sure they are protein-coding genes) exhibiting the icon <code>3</code> and then click <code>Codon Mode</code> button, the partition(s) will be changed to codon mode, in which icons <code>1</code>, <code>2</code>, and <code>3</code> correspond to partitions comprising first, second and third codon position of the gene, respectively.;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit1.jpg?raw=true"></li>
<li>Select gene name(s) with icons <code>1</code>, <code>2</code>, and <code>3</code>, then click <code>Cancel Codon Mode</code> to change back to the normal partiton mode.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit2.jpg?raw=true"></li>
<li>The <code>Name</code>, <code>Start</code> and <code>Stop</code> columns can be modified via double-clicking the corresponding cells;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit3.jpg?raw=true"></li>
<li>Closing the window will automatically save the modified partitions;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit4.jpg?raw=true"></li>
<li>If you want to manually add partition to data blocks, you can paste the text in partition format in the text box below, and then click the <code>Recognize</code> button.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit5.jpg?raw=true"></li>
</ol>
<h2 id="5-11-IQ-TREE"><a href="#5-11-IQ-TREE" class="headerlink" title="5.11. IQ-TREE"></a>5.11. <em>IQ-TREE</em></h2><p>For the installation of IQ-TREE, please see <a href="#plugins">Plugins Installation</a> section, for the results see <a href="#Output">Output Files</a> section, for input files see <a href="#input">Input Files</a> section. FASTA, PHYLIP, NEXUS and CLUSTAL formats are allowed.</p>
<p>Optionally, you may input a partition file (check the box). Please see <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/Advanced-Tutorial">http://www.iqtree.org/doc/Advanced-Tutorial</a> for detailed format requirements for the partition file. The most convenient option is to use the results of <code>Concatenate Sequence</code> (<a href="#concatenate_results">concatenate_results</a>) as input files for IQ-TREE: the concatenated alignments and the partition file will load to IQ-TREE automatically. Similarly, when using the results of <code>PartitionFinder2</code> or <code>ModelFinder</code> as input files of IQ-TREE, the alignment file, the partition and the best-fit model selection will also load into the IQ-TREE automatically.</p>
<p>Alternatively, IQ-TREE can select the best-fit model and immediately continue with the tree reconstruction (using the inferred model) by setting <code>Models</code> to <code>Auto</code> and either check ‘FreeRate heterogeneity [+R]’ (<code>-m TESTNEW</code>) or not (<code>-m TEST</code>). We also enabled IQ-TREE to reconstruct phylogenetic trees in batches, which can be used to infer supertrees.</p>
<p>After inputting files and setting the parameters, you may start the program (<code>Start</code> button), and view the run log through the <code>Show log</code> button. Once the program is finished, the parameter settings and the citation for the <code>IQ-TREE</code> will be saved in the <code>summary.txt</code> file. </p>
<h3 id="5-11-1-Brief-example"><a href="#5-11-1-Brief-example" class="headerlink" title="5.11.1. Brief example"></a>5.11.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\IQ-TREE\mtDNA_36_genes\36_genes_NUC\normal’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >),</p>
<ol>
<li>Select ‘concatenation.phy’ file;</li>
<li>Open <code>Phylogeny--&gt;IQ-TREE</code> through the menu bar;</li>
<li>Drag it into the file input box;</li>
<li>Select best-fit evolutionary model  and associated parameters (+I, +G, etc.) (here if you choose <code>Auto</code>, IQ-TREE will select the best-fit model and immediately continue with the tree reconstruction, see above);</li>
<li>Parameters can be set according to your own needs (if you don’t have a partition file, remember to uncheck <code>Partition Mode</code>), parameters that are not included in the GUI can be added via the <code>View | Edit command</code> function in the dropdown arrow of the ‘Start’ button;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/iqtree_demo.jpg?raw=true"></li>
</ol>
<p>IQ-TREE can directly use the outputs of ModelFinder and&#x2F;or PartitionFinder2, please see  <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">multi-gene tutorial</a > and <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">single-gene tutorial</a >. For a comprehensive manual of IQ-TREE, please visit <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/">http://www.iqtree.org/doc/</a> and <a target="_blank" rel="noopener" href="http://iqtree.cibiv.univie.ac.at/">http://iqtree.cibiv.univie.ac.at/</a>.</p>
<h2 id="5-12-MrBayes"><a href="#5-12-MrBayes" class="headerlink" title="5.12. MrBayes"></a>5.12. <em>MrBayes</em></h2><p>For the installation of MrBayes, please see <a href="#plugins">Plugins Installtion</a> section, for result files see <a href="#Output">Output Files</a> section, for <a href="#input">Input Files</a> see Input Files section. <span style="color: red">Note that only the NEXUS format is allowed; if <a href="#autodetect">autodetect</a> function is used, the alignment will be converted to the NEXUS format automatically.</span> When using the results of <code>PartitionFinder2</code> or <code>ModelFinder</code> as input files for MrBayes, the alignment file and the best-fit model calculated will load into MrBayes automatically.</p>
<p>If the loaded alignment file contains a command block, you can select to run with this command block directly. The <code>Outgroup(s)</code> and <code>Models</code> parameters are enabled only after the alignment is loaded.</p>
<p>PhyloSuite provides a window to edit the partition file (activated by clicking <code>Partition Models</code>), in which you can input the name of the subset, the start and stop positions, and the best model for the subset. After editing, you can click the <code>Generate Command Block</code> button to generate the corresponding command block for the edited partition.</p>
<p>Sometimes, after finishing an analysis, you may decide that the results haven’t fully converged and that you would prefer to continue the analysis; for such circumstances, PhyloSuite provides the <code>Continue Previous Analysis</code> function, which allows you to continue any of your analyses (finished and unfinished) after setting the number of additional generations.</p>
<p>There are two ways to discard MCMC samples (not generations) when summary statistics are calculated: you can either set the specific number of samples (<code>Burnin box</code>) or the proportion (<code>Burnin Fraction box</code>) of all samples.</p>
<p>The <code>Conformat</code> parameter controls the format of the consensus tree, where <code>Simple</code> setting results in a simple consensus tree written in a format read by a variety of programs (TreeView, iTOL etc.); whereas <code>Figtree</code> setting results in a consensus tree formatted for the program FigTree, with rich summary statistics.</p>
<p>The <code>Show MrBayes Data Block</code> button allows you to add the parameters that are not included in the GUI or export the configured file and run in servers (such as CIPRES, see Brief example).</p>
<p>After inputting files and setting the parameters, you can either export the alignment and the corresponding command block to execute MrBayes separately (through <code>Show MrBayes Data Block</code>) or click <code>Start</code> button to run the program within the PhyloSuite. The run log can be viewed through the <code>Show log</code> button. Once the program is finished, the parameter settings and the citation for <code>MrBayes</code> will be saved in the <code>summary.txt</code> file.</p>
<h3 id="5-12-1-Brief-example"><a href="#5-12-1-Brief-example" class="headerlink" title="5.12.1. Brief example"></a>5.12.1. Brief example</h3><p>When you are in the PhyloSuite root folder, go to ‘example\MrBayes\mtDNA_36_genes\36_genes_NUC\normal’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >),</p>
<ol>
<li>Select <code>input.nex</code> file;</li>
<li>Open <code>Phylogeny--&gt;MrBayes</code> through the menu bar;</li>
<li>Drag it into the file input box;</li>
<li>Select best-fit evolutionary model and associated parameters (+I, +G, etc.);</li>
<li>Parameters can be set according to your own needs;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_demo.jpg?raw=true"></li>
<li>If you want to export the settings to run MrBayes on CIPRES, click <code>Show MrBayes Data Block</code>, then select ‘Save to File’, upload this file to CIPRES to run directly (remember to check <code>My Data Contains a MrBayes Data Block</code>).<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_cipres.jpg?raw=true"></li>
<li>If you want to view the tree and convergence diagnostics results when it is running, you can achieve this through the <code>Stop the run and infer the tree</code> option accessed via the dropdown arrow of the <code>Stop</code> button.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/stop_mb_run.jpg?raw=true"></li>
<li>If you wish to restart a previous run (unfinished or finished), click <code>Continue Previous Analysis</code>.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_continue.jpg?raw=true"></li>
</ol>
<p>MrBayes can directly use the outputs of ModelFinder and&#x2F;or PartitionFinder2, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">multi-gene tutorial</a > and <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">single-gene tutorial</a >. For a comprehensive manual of MrBayes, please visit <a target="_blank" rel="noopener" href="http://mrbayes.sourceforge.net/manual.php">http://mrbayes.sourceforge.net/manual.php</a>.</p>
<h2 id="5-13-Flowchart"><a href="#5-13-Flowchart" class="headerlink" title="5.13. Flowchart"></a>5.13. <em>Flowchart</em></h2><p>This function streamlines the procedure of evolutionary phylogenetics analysis, including the sequence alignment (<code>MAFFT</code> and <code>MACSE</code>), elimination of poorly aligned positions and divergent regions (<code>Gblocks</code>, <code>trimAl</code> and <code>HmmCleaner</code>), sequence concatenation (<code>Concatenation</code>), model selection (<code>ModelFinder</code> or <code>PartitionFinder</code>), and tree reconstruction (<code>MrBayes</code> and <code>IQ-TREE</code>). By default, PhyloSuite predefines seven different workflows, but you can also configure&#x2F;delete your own workflows via the add button. These allow you to repeat your analyses quickly.</p>
<p>There are several things you should keep in mind when using this function:</p>
<ul>
<li>As shown in the figure below, the execution order of these programs is [<code>MAFFT</code> and&#x2F;or <code>MACSE</code>]–&gt;[<code>Gblocks</code> or <code>trimAl</code> or <code>HmmCleaner</code>]–&gt;<code>Concatenation</code>–&gt;[<code>ModelFinder</code> or <code>PartitionFinder</code>]–&gt;[<code>IQ-TREE</code> and <code>MrBayes</code>].<br> <img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_en.jpg?raw=true"><br></li>
<li>If you simultaneously choose <code>MAFFT</code> and <code>MACSE</code>, protein-coding sequences should be used as input, and the results of <code>MAFFT</code> will be subsequently refined by <code>MACSE</code></li>
<li>Only one of the three alignment optimization programs can be selected.</li>
<li>Only one of the two model selection programs can be selected.</li>
<li>Except for the model selection programs and <code>Concatenation</code>, other programs do not have to be selected (when <code>MAFFT</code>, <code>MACSE</code>, <code>trimAl</code>, <code>HmmCleaner</code>, or <code>Gblocks</code> is selected, <code>Concatenation</code> must be retained because it serves as a bridge that connects these programs with downstream programs, even for a single gene).</li>
<li>Only the first program requires an input file(s), whereas the input file(s) of other programs will be autodetected from the results of upstream analyses. <span style="color: red">Note that the two <code>Tree Reconstruction</code> programs can use either the results of <code>ModelFinder</code> or <code>PartitionFinder</code>, and they can run in parallel.</span></li>
<li>As the <code>Minimum Number Of Sequences For A Conserved Position</code> and <code>Minimum Number Of Sequences For A Flank Position</code> options are enabled only when files are input directly into <a href="#gblocks">Gblocks</a>, these two options are set by default to the most ‘relaxed’ values (i.e. lowest values) in the Flowchart mode, unless if <code>Gblocks</code> is the first program in a Flowchart analysis, in which case you can set the two options as you would normally.</li>
<li>For the model selection and tree reconstruction, if only <code>ModelFinder</code> and <code>IQ-TREE</code> are selected, <code>IQ-TREE</code> will use the best-fit model calculated by <code>ModelFinder</code>; if only <code>ModelFinder</code> and <code>MrBayes</code> are selected, then <code>Mrbayes</code> option must be selected in the <code>Model for</code> menu of <code>ModelFinder</code>; and finally, if <code>ModelFinder</code>, <code>IQ-TREE</code> and <code>MrBayes</code> are selected, the results of <code>ModelFinder</code> will be used only for <code>MrBayes</code> (thus it will use the same settings as described in the preceding note), whereas <code>IQ-TREE</code> will first conduct the best-fit model selection inbuilt in the algorithm, and conduct the tree inference (using the <code>Auto</code> option in the <code>Models</code> menu, equivalent to <code>-m TEST</code> or <code>-m MFP</code>). </li>
<li>PhyloSuite also provides a function to check and autocorrect the parameters between selected programs, including those specified in the previous note, conflicting sequence types, conflicting partition modes, etc.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_settings.jpg?raw=true"><br> </li>
<li>When the flowchart is finished, the parameter settings and the citations of corresponding software programs will be summarized in the <a href="#operation">display area</a> of the <code>flowchart</code>.</li>
</ul>
<h3 id="5-13-1-Brief-example"><a href="#5-13-1-Brief-example" class="headerlink" title="5.13.1. Brief example"></a>5.13.1. Brief example</h3><p>Tip: if you changed the workflow settings, remember to save it using the <code>add</code> button, otherwise, it will not be remembered. For comprehensive demos, please see <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/#9-Flowchart">multi-gene tutorial</a > and <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/#7-Flowchart">single-gene tutorial</a >.</p>
<h2 id="5-14-Mitogenome"><a href="#5-14-Mitogenome" class="headerlink" title="5.14. Mitogenome"></a>5.14. <em>Mitogenome</em></h2><h3 id="5-14-1-Parse-annotations"><a href="#5-14-1-Parse-annotations" class="headerlink" title="5.14.1. Parse annotations"></a>5.14.1. Parse annotations</h3><p>This function can parse the annotations recorded in a Microsoft Word document (only *.docx extension is supported). When annotating the tRNAs, you should add the anti-codon of each tRNA gene to the end of the gene name (in brackets), for example: tRNA-Cys(GCA) (also see an example in the image below). Regarding the names of genes, PhyloSuite allows you to replace the name with other names by setting the <code>Name from Word</code> table accessed through the <code>Configure name replacing</code> button. Additionally, you can define the name of <code>product</code> <a href="#qualifier">qualifiers</a> for each protein coding gene and the abbreviation of tRNA genes used for the organization table.</p>
<p>Example of mitogenome annotation in a Word document:<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/annotations.jpg?raw=true"><br></p>
<p>The GenBank Submission Template file including the information of authors and affiliations can be generated <a target="_blank" rel="noopener" href="https://submit.ncbi.nlm.nih.gov/genbank/template/submission/" title="Submission Portal">here</a>. Several datasets from PhyloSuite can be used to generate the annotation section, including <code>Organism</code>, <code>Strain</code>, <code>Lineage</code>, etc. The <code>Release Date</code> parameter defines the release date of your sequence. <span style="color: red">Note that you should have office suite installed on your computer.</span></p>
<h4 id="5-14-1-1-Brief-example"><a href="#5-14-1-1-Brief-example" class="headerlink" title="5.14.1.1. Brief example"></a>5.14.1.1. Brief example</h4><p>When you are in the PhyloSuite root folder, go to ‘example\Parse_Word_annotations’ folder (if you don’t have the newest example folder, please download it from <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">here</a >), </p>
<ol>
<li>Select ‘Diplectanum_longipenis_mtDNA.docx’ file (you can open this file to see how to annotate the sequence);</li>
<li>Open <code>Mitogenome--&gt;Parse Annotation</code> through the menu bar;</li>
<li>Drag the file into the file input box;</li>
<li>Click the blue word (as shown in figure) to generate a template file;</li>
<li>Drag the template file into the <code>Template File</code> input box;</li>
<li>Fill in necessary information, such as <code>Organism</code>, <code>Lineage</code>, <code>Code Table</code>, etc. </li>
<li>Other Parameters can be set according to your own needs;</li>
<li>Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/parseANNT_demo.jpg?raw=true"></li>
</ol>
<h3 id="5-14-2-Compare-tables"><a href="#5-14-2-Compare-tables" class="headerlink" title="5.14.2. Compare tables"></a>5.14.2. Compare tables</h3><p>This function can compare and gather tables in the <code>speciesStat</code> subfolder under the <code>extract_results</code> folder. For organization tables, pairwise similarity calculation is allowed, in which MAFFT is invoked to make alignment and DistanceCalculator package in Biopython is used to calculate the identity of the sequence. The header of a table can be omitted from the comparison by selecting the number of rows you wish to exclude (from the top). For table examples, please see Table 1 in <a target="_blank" rel="noopener" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3">https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3</a> and Table 2 in <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-018-2910-9">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-018-2910-9</a>.</p>
<h4 id="5-14-2-1-Brief-example"><a href="#5-14-2-1-Brief-example" class="headerlink" title="5.14.2.1. Brief example"></a>5.14.2.1. Brief example</h4><p>This function can directly use the results of ‘extract’ function:</p>
<ol>
<li>Select the <code>extract_results</code> folder (if not available, please see <a href="#extract_exam">here</a> for how to make one, mitogenome datatype only);</li>
<li>Open <code>Mitogenome--&gt;Compare Table</code> through the menu bar;</li>
<li>All extracted organization tables will be automatically imported; remove the tables you are not interested in using the <code>remove</code> button;</li>
<li>Check <code>Calculate pairwise similarity</code> if you want to calculate pairwise similarity for homologous genes;</li>
<li>Start the program;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_demo.jpg?raw=true"></li>
<li>If you want to compare nucleotide composition and skewness table (identified by no ‘_org’ in its name in the results folder), you should open the <code>extract_results</code> folder first, then enter ‘extract_results\StatFiles\speciesStat’, select interested files, drag them into the <code>Tables</code> box, uncheck <code>Calculate pairwise similarity</code>, then Start the program.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_notorg.jpg?raw=true"></li>
</ol>
<h3 id="5-14-3-Draw-RSCU-figure"><a href="#5-14-3-Draw-RSCU-figure" class="headerlink" title="5.14.3. Draw RSCU figure"></a>5.14.3. Draw RSCU figure</h3><p>For the installation of Rscript, please see <a href="#plugins">Plugins Installation</a> section, and for the results please see <a href="#Output">Output Files</a> section.</p>
<p>This function can draw an RSCU figure based on the tables in the “RSCU” subfolder under the <code>extract_results/StatFiles/RSCU</code> folder. You can drag to reorder the input files and the amino acids on the x-axis. For a figure example, please see Fig. 3 in <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1</a>.</p>
<h4 id="5-14-3-1-Brief-example"><a href="#5-14-3-1-Brief-example" class="headerlink" title="5.14.3.1. Brief example"></a>5.14.3.1. Brief example</h4><p>This function can directly use the results of ‘extract’ function:</p>
<ol>
<li>Select the <code>extract_results</code> folder (if not available, please see <a href="#extract_exam">here</a> for how to make one, mitogenome datatype only);</li>
<li>Open <code>Mitogenome--&gt;Draw RSCU figure</code> through the menu bar;</li>
<li>All extracted RSCU tables will be automatically imported, remove the tables you are not interested in using the <code>remove</code> button;</li>
<li>Parameters can be set according to your own needs;</li>
<li>Start the program;<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/RSCU_demo.jpg?raw=true"></li>
</ol>
<h2 id="5-15-Molecular-dating-analysis"><a href="#5-15-Molecular-dating-analysis" class="headerlink" title="5.15. Molecular dating analysis"></a>5.15. <em>Molecular dating analysis</em></h2><p>See <a href="https://dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/">https://dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/</a> or <a target="_blank" rel="noopener" href="http://phylosuite.jushengwu.com/dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/">http://phylosuite.jushengwu.com/dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/</a></p>
<h1 id="6-Citations-and-codes"><a href="#6-Citations-and-codes" class="headerlink" title="6. Citations and codes"></a>6. Citations and codes</h1><p>If you use data generated by PhyloSuite in a scientific paper, please use the following citation:</p>
<blockquote>
<p>Zhang, D., F. Gao, I. Jakovlić, H. Zou, J. Zhang, W.X. Li, and G.T. Wang, PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Molecular Ecology Resources, 2020. 20(1): p. 348–355. DOI: 10.1111&#x2F;1755-0998.13096.</p>
</blockquote>
<p>Please also note that PhyloSuite is a plug-in program, and that you should also cite any (and every) plug-in program not designed and compiled by us that you use in your analyses. This applies to the following plug-ins:<br><br></p>
<p><strong>MAFFT</strong></p>
<blockquote>
<p>Katoh, K., and Standley, D.M. (2013). MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol 30, 772-780.</p>
</blockquote>
<br>

<p><strong>MACSE</strong></p>
<blockquote>
<p>Ranwez V, Douzery EJP, Cambon C, Chantret N, Delsuc F. 2018. MACSE v2: Toolkit for the alignment of coding sequences accounting for frameshifts and stop codons. Mol Biol Evol. 35: 2582-2584. doi: 10.1093&#x2F;molbev&#x2F;msy159.</p>
</blockquote>
<br>

<p><strong>Gblocks</strong></p>
<blockquote>
<p>Talavera, G., and Castresana, J. (2007). Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst Biol 56, 564-577.</p>
</blockquote>
<br>

<p><strong>trimAl</strong></p>
<blockquote>
<p>Capella-Gutierrez S, Silla-Martinez JM, Gabaldon T. 2009. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 25: 1972-1973. doi: 10.1093&#x2F;bioinformatics&#x2F;btp348.</p>
</blockquote>
<br>

<p><strong>HmmCleaner</strong></p>
<blockquote>
<p>Di Franco A, Poujol R, Baurain D, Philippe H. 2019. Evaluating the usefulness of alignment filtering methods to reduce the impact of errors on evolutionary inferences. BMC Evol Biol. 19: 21. doi: 10.1186&#x2F;s12862-019-1350-2.</p>
</blockquote>
<br>

<p><strong>IQ-TREE</strong></p>
<blockquote>
<p>Nguyen, L.T., Schmidt, H.A., von Haeseler, A., and Minh, B.Q. (2015). IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Mol Biol Evol 32, 268-274.</p>
</blockquote>
<br>

<p><strong>PartitionFinder2</strong></p>
<blockquote>
<p>Lanfear, R., Frandsen, P.B., Wright, A.M., Senfeld, T., and Calcott, B. (2017). PartitionFinder 2: new methods for selecting partitioned models of evolution for molecular and morphological phylogenetic analyses. Mol Biol Evol 34, 772-773.</p>
</blockquote>
<br>

<p><strong>MrBayes</strong></p>
<blockquote>
<p>Ronquist, F., Teslenko, M., van der Mark, P., Ayres, D.L., Darling, A., Höhna, S., Larget, B., Liu, L., Suchard, M.A., and Huelsenbeck, J.P. (2012). MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space. Syst Biol 61, 539-542.</p>
</blockquote>
<br>

<p>For the remaining functions, we mostly used our own Python codes, written in Python 3.6.7 and PyQT5. Biopython package was used for some functions, such as feature extraction from GenBank files, which is conducted using SeqIO module.</p>
<blockquote>
<p>Cock, P.J., Antao, T., Chang, J.T., Chapman, B.A., Cox, C.J., Dalke, A., Friedberg, I., Hamelryck, T., Kauff, F., Wilczynski, B., et al. (2009). Biopython: freely available Python tools for computational molecular biology and bioinformatics. Bioinformatics 25, 1422-1423.</p>
</blockquote>
<h1 id="7-Troubleshooting"><a href="#7-Troubleshooting" class="headerlink" title="7. Troubleshooting"></a>7. Troubleshooting</h1><h2 id="7-1-Update-failed-how-to-revert-to-previous-settings-and-plugins"><a href="#7-1-Update-failed-how-to-revert-to-previous-settings-and-plugins" class="headerlink" title="7.1. Update failed: how to revert to previous settings and plugins"></a>7.1. Update failed: how to revert to previous settings and plugins</h2><p>In some rare cases, users may encounter errors when updating PhyloSuite. As this may cause losing some of your settings and configurations, here we will demonstrate how to revert your settings and plugins to the state before update.</p>
<ol>
<li>First you should download the latest PhyloSuite package at <a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a>  or <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link">https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link</a> (China). <span style="color: red"> Note: for Windows, you should download <code>PhyloSuite_xxx_Win.rar</code>, instead of the installer file.</span></li>
</ol>
<table>
<thead>
<tr>
<th>System</th>
<th>Package file</th>
</tr>
</thead>
<tbody><tr>
<td>Windows</td>
<td>PhyloSuite_xxx_Win.rar</td>
</tr>
<tr>
<td>Linux</td>
<td>PhyloSuite_xxx_Linux.tar.gz</td>
</tr>
<tr>
<td>Mac OSX</td>
<td>PhyloSuite_xxx_Mac.zip</td>
</tr>
</tbody></table>
<ol start="2">
<li><p>Unzip the package, select and copy all files, go to the installation path of PhyloSuite, open the <code>PhyloSuite</code> folder, paste the copied files directly into this folder (if prompted so, confirm that you wish to replace files with the same name).<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/update_failed.jpg?raw=true"></p>
</li>
<li><p>Open PhyloSuite, you should find that you are running an updated version, and that your previous settings have been retained.<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/open_phylosuite.jpg?raw=true"></p>
</li>
</ol>
<h2 id="7-2-PhyloSuite-run-failed"><a href="#7-2-PhyloSuite-run-failed" class="headerlink" title="7.2. PhyloSuite run failed"></a>7.2. PhyloSuite run failed</h2><p>If the execution of PhyloSuite fails, please first try shutting down your antivirus program.</p>
<h2 id="7-3-MrBayes-does-not-work"><a href="#7-3-MrBayes-does-not-work" class="headerlink" title="7.3. MrBayes does not work"></a>7.3. MrBayes does not work</h2><p>Sometimes MrBayes will finish immediately, without reporting an error. Generally you can try to find the problem by executing MrBayes in the terminal:</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/exe_mb_terminal.jpg?raw=true"></p>
<p>If you get ‘msvcr120.dll is missing’ error in Windows, you can fix it via <a target="_blank" rel="noopener" href="https://www.drivereasy.com/knowledge/best-solutions-to-fix-msvcr120-dll-is-missing-in-windows/">this solution</a>.</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_error_win.png?raw=true"></p>
<p>For other problems, please search the <code>error code</code> in the website.</p>
<h2 id="7-4-PhyloSuite-get-stuck"><a href="#7-4-PhyloSuite-get-stuck" class="headerlink" title="7.4. PhyloSuite get stuck"></a>7.4. PhyloSuite get stuck</h2><p>If PhyloSuite become more and more stuck, this may be caused by the increasing data in a workplace. To settle this problem, you should create a new workplace. Generally, PhyloSuite encourage user to create multiple workplaces to preserve their work.</p>
<h2 id="7-5-MAFFT-error"><a href="#7-5-MAFFT-error" class="headerlink" title="7.5. MAFFT error"></a>7.5. MAFFT error</h2><p>If you encounter errors with MAFFT, such as:</p>
<ul>
<li><code>/usr/bin/awk: cannot execute binary file</code></li>
<li><code>-gt: unary operator expected</code></li>
<li><code>options: check source file</code></li>
</ul>
<p>please try the following steps:</p>
<ol>
<li>Reinstall MAFFT from the official website: <a target="_blank" rel="noopener" href="https://mafft.cbrc.jp/alignment/software/">https://mafft.cbrc.jp/alignment/software/</a>.  </li>
<li>Specify the MAFFT executable file in <strong>PhyloSuite</strong>. For detailed instructions, see <a target="_blank" rel="noopener" href="http://phylosuite.jushengwu.com/dongzhang0725.github.io/PhyloSuite-demo/how-to-configure-plugins/">How to configure plugins in PhyloSuite</a>.  </li>
<li><strong>Important:</strong> For MAFFT, make sure to specify the <code>mafft.bat</code> file as the executable in PhyloSuite.</li>
</ol>
<h2 id="7-6-Cannot-run-PhyloSuite-on-MAC"><a href="#7-6-Cannot-run-PhyloSuite-on-MAC" class="headerlink" title="7.6 Cannot run PhyloSuite on MAC"></a>7.6 Cannot run PhyloSuite on MAC</h2><p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/installation/MAC_error1.png?raw=true"></p>
<p>For <strong>macOS users</strong>, if you encounter <em>Security &amp; Privacy restrictions</em> or an alert such as <strong>“Apple cannot verify “xxx file”</strong>, you can try resolving the issue with the following command:</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">sudo xattr -rd [PhyloSuite_installation_path]/PhyloSuite</span><br></pre></td></tr></table></figure>

<h1 id="8-Acknowledgements"><a href="#8-Acknowledgements" class="headerlink" title="8. Acknowledgements"></a>8. Acknowledgements</h1><p>We would like to thank Dr. Meng Kai-Kai for helping us to set up the chloroplast genome extraction function.</p>
</div>

<!-- 中文内容 -->
<div class="lang lang-zh" style="display:none;">

<center style="font-size:30px"><strong>PhyloSuite</strong></center>
<center style="font-size:17px"><strong>用于简化分子序列数据管理和进化系统发育学研究的集成化可扩展桌面平台</strong></center>
<br>
<center><strong>主页：</strong><a href="https://dongzhang0725.github.io">https://dongzhang0725.github.io</a></center>
<center><strong>示例与测试运行：</strong><a href="https://dongzhang0725.github.io/dongzhang0725.github.io/example/">https://dongzhang0725.github.io/dongzhang0725.github.io/example/</a></center>
<center><strong>演示教程：</strong><a href="https://dongzhang0725.github.io/dongzhang0725.github.io/archives/">https://dongzhang0725.github.io/dongzhang0725.github.io/archives/</a></center>
<br>
<center style="font-size:17px">张东, 高芳銮, Ivan Jakovlić, 邹红, 张金, 李文祥, 王桂堂</center>
<center style="font-size:17px">版本 2 || 2022年11月26日</center>

<hr>

<h1 id="1-简介"><a href="#1-简介" class="headerlink" title="1. 简介"></a>1. 简介</h1><h2 id="1-1-背景"><a href="#1-1-背景" class="headerlink" title="1.1. 背景"></a>1.1. <em>背景</em></h2><p>下一代测序（NGS）技术的进步导致通过公共数据库可获取的遗传数据量急剧增加。虽然这开启了大量的研究可能性，但对于不精通计算机的研究人员来说，检索和管理如此大量的数据可能既困难又耗时。因此，能够为广泛进化生物学家节省大量时间的多功能、工作流式且支持批量处理的软件包需求日益增长。PhyloSuite 旨在填补这一空白：一个致力于简化分子序列数据管理和进化系统发育学研究的用户友好的工作流桌面平台。</p>
<h2 id="1-2-功能"><a href="#1-2-功能" class="headerlink" title="1.2. 功能"></a>1.2. <em>功能</em></h2><p>PhyloSuite 是一个用户友好的、独立的、基于 GUI 的软件，使用 Python 3.6.7 和 PyQT5 编写。其功能包括：</p>
<ul>
<li>检索、提取、组织和管理分子序列数据，包括 GenBank 条目、核苷酸和氨基酸序列，以及 Word 文档中注释的序列；</li>
<li>使用 MAFFT 进行序列批量比对，我们为此添加了密码子比对（翻译比对）模式；</li>
<li>使用 MACSE 对蛋白质编码序列进行批量比对或比对优化；</li>
<li>使用 trimAl、HmmCleaner 或 Gblocks 批量优化模糊比对区域；</li>
<li>批量转换比对格式（FASTA、PHYLIP、PAML、AXT 和 NEXUS）；</li>
<li>将多个比对串联成单个数据集，并为下游分析准备分区文件；</li>
<li>使用 ModelFinder 或 PartitionFinder 选择最佳拟合的进化模型和&#x2F;或分区方案；</li>
<li>使用 IQ-TREE（最大似然法）和&#x2F;或 MrBayes（贝叶斯推断）重建系统发育树；</li>
<li>将功能 (ii) 到 (viii) 链接成一个工作流；</li>
<li>使用功能 (i) 生成的数据集在 iTOL 网络工具中注释系统发育树；</li>
<li>对线粒体基因组进行全面的生物信息学分析；</li>
<li>使用类似 MEGA 的序列查看器可视化和编辑序列；</li>
<li>在 PhyloSuite 工作区中存储、组织和可视化每次分析的数据和结果。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/all_flowchart_english.jpg?raw=true"></li>
</ul>
<h1 id="2-许可与免责声明"><a href="#2-许可与免责声明" class="headerlink" title="2. 许可与免责声明"></a>2. 许可与免责声明</h1><h2 id="2-1-许可"><a href="#2-1-许可" class="headerlink" title="2.1. 许可"></a>2.1. <em>许可</em></h2><p>PhyloSuite 是一个自由软件，欢迎您在特定条件下重新分发它。它根据 GNU 通用公共许可证第 3 版发布。参见 <a target="_blank" rel="noopener" href="http://www.gnu.org/licenses/gpl-3.0.en.html">http://www.gnu.org/licenses/gpl-3.0.en.html</a>。</p>
<h2 id="2-2-免责声明"><a href="#2-2-免责声明" class="headerlink" title="2.2. 免责声明"></a>2.2. <em>免责声明</em></h2><p>本程序不附带任何担保。对于本软件的功能性或所获结果的准确性，不作任何明示或暗示的保证。请仔细检查您的结果。</p>
<h1 id="3-操作系统与安装"><a href="#3-操作系统与安装" class="headerlink" title="3. 操作系统与安装"></a>3. 操作系统与安装</h1><h2 id="3-1-安装已编译的-PhyloSuite"><a href="#3-1-安装已编译的-PhyloSuite" class="headerlink" title="3.1. 安装已编译的 PhyloSuite"></a>3.1. <em>安装已编译的 PhyloSuite</em></h2><p>所有平台的安装程序都可以从 <a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a> 下载。</p>
<p><a id="download" target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases"><i class="fa fa-download"></i><span> 立即下载</span> </a></p>
<p><a id="download" target="_blank" rel="noopener" href="http://phylosuite.jushengwu.com/dongzhang0725.github.io/installation/#Chinese_download_link"><i class="fa fa-download"></i><span> 中文下载链接</span> </a></p>
<h3 id="3-1-1-Windows-1"><a href="#3-1-1-Windows-1" class="headerlink" title="3.1.1. Windows"></a>3.1.1. Windows</h3><p>支持 Windows 7、8 和 10，只需双击 <code>PhyloSuite_xxx_win_setup.exe</code> 进行安装，安装后运行 “PhyloSuite.exe” 文件。如果安装失败，请下载 <code>PhyloSuite_xxx_Win.rar</code>，解压后直接从该文件夹运行 PhyloSuite。</p>
<h3 id="3-1-2-Mac-OSX-和-Linux"><a href="#3-1-2-Mac-OSX-和-Linux" class="headerlink" title="3.1.2. Mac OSX 和 Linux"></a>3.1.2. Mac OSX 和 Linux</h3><p>将 <code>PhyloSuite_xxx_Mac.zip</code>&#x2F;<code>PhyloSuite_xxx_Linux.tar.gz</code> 解压到您喜欢的任何位置，然后双击 “PhyloSuite”（在 PhyloSuite 文件夹中）启动，或使用以下命令：</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">cd path/to/PhyloSuite</span><br><span class="line">./PhyloSuite</span><br></pre></td></tr></table></figure>
<p>如果遇到 “permission denied” 错误，请尝试使用以下命令：</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">chmod -R 755 path/to/PhyloSuite(文件夹)</span><br></pre></td></tr></table></figure>

<p><span style="color:red">请注意，支持 64 位和 32 位 Windows（Windows 7 及以上版本），而在 Linux（Ubuntu 14.04.1 及以上版本）和 Mac OSX（macOS Sierra 版本 10.12.3 及以上版本）中仅测试了 64 位。</span></p>
<h2 id="3-2-使用-pip-安装"><a href="#3-2-使用-pip-安装" class="headerlink" title="3.2. 使用 pip 安装"></a>3.2. <em>使用 pip 安装</em></h2><p>首先，应在您的计算机上安装 Python（版本 3.6）并将其添加到环境变量中。然后打开终端并输入：</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">pip install PhyloSuite</span><br></pre></td></tr></table></figure>

<p>安装需要一些时间。如果安装成功，PhyloSuite 将自动添加到环境变量中。然后再次打开终端并输入：<br> <figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">PhyloSuite</span><br></pre></td></tr></table></figure></p>
<p>如果上述 <code>pip</code> 命令安装 PhyloSuite 失败，您可以使用已编译的 PhyloSuite（见 <code>第 1 节</code>）或在此处查找并下载源代码（<a target="_blank" rel="noopener" href="https://pypi.org/project/PhyloSuite/#files">https://pypi.org/project/PhyloSuite/#files</a> 或 <a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite%EF%BC%89%EF%BC%8C%E7%84%B6%E5%90%8E%E6%89%8B%E5%8A%A8%E5%AE%89%E8%A3%85%E3%80%82">https://github.com/dongzhang0725/PhyloSuite），然后手动安装。</a></p>
<h1 id="4-管理"><a href="#4-管理" class="headerlink" title="4. 管理"></a>4. 管理</h1><h2 id="4-1-界面操作"><a href="#4-1-界面操作" class="headerlink" title="4.1. 界面操作"></a>4.1. <em>界面操作</em></h2><p><span id="operation"><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mainwindow.jpg?raw=true"></span></p>
<p>PhyloSuite 使用工作区进行数据管理（尽管您不一定非要使用它），该工作区在您首次使用程序时设置。之后您可以通过 <code>工作区</code> 菜单更改它。每个工作区中有两种根文件夹：<strong>GenBank_File</strong> 和 <strong>Other_File</strong>。<strong>GenBank_File</strong> 文件夹用于管理 GenBank 文件并存放相关分析的结果。<strong>Other_File</strong> 文件夹用于管理其他类型的序列文件和 Word 注释文件（核苷酸和氨基酸序列），以及存放相关结果。在根文件夹下方（下一级），您会找到工作文件夹，其中包含关联的结果文件夹（再下一级）。在每个根文件夹中，PhyloSuite 会默认添加 <code>files</code> 和 <code>flowchart</code> 工作文件夹。您可以通过将鼠标悬停在根文件夹（GenBank_File 和 Other_File）上，然后点击右侧的绿色“加号”图标或通过上下文（右键）菜单来创建一个新的工作文件夹来存放您的新工作（推荐）。您可以通过所选文件夹的上下文菜单或按键盘上的 <code>Delete</code> 按钮来移除工作文件夹。被删除的文件夹存储在根文件夹的 <code>recycled</code> 文件夹中，并可以从中恢复。一旦文件夹从回收站中删除，将无法恢复。但是，建议从您的本地文件系统中删除文件夹&#x2F;文件，因为这样您可以使用操作系统内置的文件恢复功能来恢复它们。<span style="color:red">请注意，几乎所有选定的设置，例如窗口大小、位置、参数设置等，在您关闭窗口时都会被自动记住（即，在关闭窗口之前无需保存设置）。</span></p>
<p>您可以通过相应功能窗口中的 <code>问号</code> 按钮访问每个功能的简要示例演示。</p>
<h3 id="4-1-1-简要示例"><a href="#4-1-1-简要示例" class="headerlink" title="4.1.1. 简要示例"></a>4.1.1. 简要示例</h3><ul>
<li>点击 <code>GenBank_File</code> 或 <code>Other_File</code> 根文件夹将显示 PhyloSuite 的主页。将鼠标悬停在根文件夹上可查看 <code>添加工作文件夹</code> 和 <code>在资源管理器中打开</code> 按钮；前者可用于创建新的工作文件夹，后者则在您的本地文件资源管理器中打开该文件夹；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_root.jpg?raw=true"></li>
<li>选择您的任意工作文件夹（根文件夹下一级）将显示您保存的数据集列表。GenBank 记录列表存储在 <code>GenBank_File</code> 根文件夹中，序列文件存储在 <code>Other_File</code> 根文件夹中。将鼠标悬停在工作文件夹上可看到 <code>在资源管理器中打开</code> 和 <code>GenBank 文件信息显示设置</code> 按钮；后者可用于控制每个 ID 的哪些数据将显示在主页面。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_workfolder.jpg?raw=true"></li>
<li>选择结果文件夹（工作文件夹下一级）将显示结果。将鼠标悬停在结果文件夹上可看到 <code>在资源管理器中打开</code> 按钮；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_results_folder.jpg?raw=true"></li>
<li>双击上述任意文件夹（根文件夹、工作文件夹和结果文件夹）将在您的本地文件资源管理器中打开该文件夹；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/double_click.jpg?raw=true"></li>
<li>将鼠标悬停在菜单栏上以选择要使用的功能。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/hover_menubar.jpg?raw=true"></li>
</ul>
<h2 id="4-2-插件安装"><a href="#4-2-插件安装" class="headerlink" title="4.2. 插件安装"></a>4.2. <em>插件安装</em></h2><p><span id="plugins">PhyloSuite</span> 集成了八个插件程序：</p>
<table>
<thead>
<tr>
<th>程序</th>
<th>可执行文件</th>
<th>描述</th>
</tr>
</thead>
<tbody><tr>
<td><a target="_blank" rel="noopener" href="https://mafft.cbrc.jp/alignment/software/" title="MAFFT">MAFFT v7.313</a></td>
<td>mafft.bat</td>
<td>氨基酸或核苷酸序列的多重比对</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://www.iqtree.org/" title="IQ-TREE">IQ-TREE v. 1.6.8</a></td>
<td>iqtree.exe</td>
<td>用于系统发育组学推断的高效软件</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://mrbayes.sourceforge.net/" title="MrBayes">MrBayes 3.2.6</a></td>
<td>mrbayes_x64.exe 或 <br>mrbayes_x86.exe</td>
<td>系统发育的贝叶斯推断</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://www.robertlanfear.com/partitionfinder/" title="PartitionFinder2">PartitionFinder2</a></td>
<td>partitionfinder 文件夹</td>
<td>为系统发育分析选择最佳拟合的分区方案和分子进化模型</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_documentation.html" title="Gblocks">Gblocks 0.91b</a></td>
<td>Gblocks.exe</td>
<td>从多重比对中选择保守块用于系统发育分析</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.r-project.org/" title="Rscript">Rscript 3.4.4</a></td>
<td>Rscript.exe</td>
<td>绘制 RSCU 图所需</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.anaconda.com/download/" title="Python 2.7">Python 2.7</a></td>
<td>python.exe</td>
<td>PartitionFinder2 所需</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/" title="tbl2asn">tbl2asn</a></td>
<td>tbl2asn.exe</td>
<td>自动化创建用于提交至 GenBank 的序列记录（仅限 Windows）</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.mpich.org/">MPICH2</a></td>
<td>mpirun 或 mpiexec</td>
<td>消息传递接口（MPI）标准的高性能且广泛便携的实现，支持多线程 MrBayes 操作（仅限 Linux）</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://bioweb.supagro.inra.fr/macse/index.php?menu=releases">MACSE</a></td>
<td>macse_v2.03.jar</td>
<td>考虑移码和终止密码子的编码序列多重比对。</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.java.com/en/download/">Java (JRE &gt; 1.5)</a></td>
<td>java.exe</td>
<td>MACSE 所需</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="http://trimal.cgenomics.org/downloads">trimAl</a></td>
<td>trimal.exe</td>
<td>用于自动修剪比对的工具</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://metacpan.org/pod/distribution/Bio-MUST-Apps-HmmCleaner/bin/HmmCleaner.pl">HmmCleaner</a></td>
<td>HmmCleaner.pl</td>
<td>从您的 MSA 中移除低相似性片段</td>
</tr>
<tr>
<td><a target="_blank" rel="noopener" href="https://www.perl.org/get.html">Perl 5</a></td>
<td>perl.exe</td>
<td>HmmCleaner 所需</td>
</tr>
</tbody></table>
<p>这些插件可以在 <span style="color: #C72591"><code>设置--&gt;插件</code></span> 中安装。<br>可以通过三种方式完成：</p>
<ol>
<li>如果 Python 2.7、Perl 5、Java (JRE &gt; 1.5)、HmmCleaner.pl 和 trimAl 已安装并添加到环境变量 ($PATH) 中，它们将被 PhyloSuite 自动检测到。</li>
<li>如果您的计算机上已安装这些程序，您可以直接指定可执行文件（如上表所示）。<span style="color: red">请注意，对于 PartitionFinder2，您应指定 ‘partitionfinder-2.1.1’ 文件夹。<span></li>
<li>如果您没有这些程序，可以使用下载按钮自动下载并安装它们。<span style="color: red">注意：Python 2.7 将下载 Anaconda Python 发行版（因为它包含了 PartitionFinder2 所需的所有依赖项：numpy, pandas, pytables, pyparsing, scipy 和 sklearn）。由于其大小约为 500M，因此下载可能需要一些时间。<span></li>
</ol>
<p><span style="color: red">请注意，这些插件程序的路径不应包含特殊字符（^, {, @ 等）。<span></p>
<h3 id="4-2-1-简要示例"><a href="#4-2-1-简要示例" class="headerlink" title="4.2.1. 简要示例"></a>4.2.1. 简要示例</h3><p>请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/how-to-configure-plugins/">如何配置插件</a >。</p>
<h2 id="4-3-导入文件"><a href="#4-3-导入文件" class="headerlink" title="4.3. 导入文件"></a>4.3. <em>导入文件</em></h2><p>PhyloSuite 接受多种文件格式和扩展名：</p>
<ul>
<li>GenBank 文件：*.gb, *.gbk, *.gbf, *.gbff</li>
<li>fasta：*.fas, *.fasta</li>
<li>phylip：*.phy, *.phylip</li>
<li>nexus：*.nex, *.nxs, *.nexus</li>
<li>Word 文档文件：*.docx</li>
</ul>
<p>关于如何将序列导入 PhyloSuite 的演示教程，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/five_ways_to_import/">五种将数据导入 PhyloSuite 的方法</a >。</p>
<h3 id="4-3-1-GenBank-文件"><a href="#4-3-1-GenBank-文件" class="headerlink" title="4.3.1. GenBank 文件"></a>4.3.1. GenBank 文件</h3><p><font color="red">提示：GenBank 文件应为标准格式（参见<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html" title="GenBank Flat File Format">详情</a></font>）。<br></p>
<p>PhyloSuite 提供了三种方式将 GenBank 文件导入到 <strong>GenBank_File</strong> 根文件夹的工作文件夹中：</p>
<ol>
<li>使用 <strong>文件</strong> 菜单下的 <code>导入文件或ID</code> 功能或主<a href="#operation">显示区域</a>中的 <code>打开文件</code>。此模式支持导入完整的 GenBank 文件和 GenBank accession 号（ID）列表，PhyloSuite 将自动从 GenBank 检索这些文件；</li>
<li>将 GenBank 格式文件拖放到<a href="#operation">显示区域</a>中。</li>
<li>复制 GenBank 文件内容并将其粘贴到<a href="#operation">显示区域</a>中。</li>
</ol>
<h4 id="4-3-1-1-简要示例"><a href="#4-3-1-1-简要示例" class="headerlink" title="4.3.1.1. 简要示例"></a>4.3.1.1. <span id="import_gb">简要</span>示例</h4><ol>
<li>选择任意工作文件夹（这里我选择了 files）；</li>
<li>点击 <code>打开文件/输入ID</code> 打开输入窗口。</li>
<li>将 ID 复制到文本框中（支持空格、换行符、制表符等作为分隔符）；</li>
<li>输入您的电子邮件（告知 NCBI 是谁在下载序列）；</li>
<li>点击 <code>开始</code> 进行下载。</li>
</ol>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/download_seq_en.jpg?raw=true"></p>
<p>导入 GenBank 文件后，有许多选项可以使用，可通过上下文（右键）菜单访问（除非另有说明）：</p>
<ul>
<li>可以通过拖放或上下文菜单（”添加文件”）将文件（ID）添加到您正在处理的数据集中。</li>
<li>可以通过 <code>文件 --&gt; 标准化 GenBank 文件</code> 功能或上下文菜单中的 <code>标准化</code> 来标准化 GenBank 文件的注释（这包括上面讨论的基因名称统一）。此功能会打开一个新的弹出窗口，显示数据集中可能存在的错误和警告，您可以在其中手动编辑文件（仅限线粒体基因组数据）。这些通常涉及缺失的基因或非标准化的注释。对于后者，您可以点击此窗口右上角的齿轮按钮（设置），打开上面讨论的 <a href="#extract">GenBank 文件提取设置</a> 窗口。通过勾选 <code>设置 NCR 阈值</code> 框，您可以提示 PhyloSuite 也识别非编码区域（这允许您稍后使用 <code>提取</code> 功能提取它们）。您可以设置希望识别的 NCR 大小（以 bp 为单位）的阈值。</li>
<li>对于线粒体基因组数据，可以使用 <code>预测 tRNA (LEU 和 SER)</code> 按钮，通过该按钮您可以在 ARWEN 的帮助下重新注释模糊注释的 tRNA。</li>
<li>您可以通过 <code>设置 --&gt; GenBank 文件信息显示</code> 选择您希望显示的文件中包含的信息。例如：ID、organism、lineages、references、source 等（参见<a href="#display">信息显示</a>部分）。</li>
<li>包含相同序列（<span style="color: red">重复项</span>）的 ID 可以使用 <code>高亮相同序列</code> 按钮（星形，底部&#x2F;右侧）识别并自动删除。</li>
<li>您可以使用其旁边的 <code>按 ID 查找记录</code> 按钮搜索特定的 ID。</li>
<li>每个 ID 可以通过上下文菜单用任何文本查看器程序（例如记事本）打开，然后手动编辑。</li>
<li>选定的 ID 可以（通过上下文菜单）导出为 GenBank (.gb) 文件，或包含 GUI 中显示信息的表格 (.csv)。</li>
<li>选定的 ID 可以通过拖放导入到不同的工作文件夹中。</li>
</ul>
<h3 id="4-3-2-其他类型的文件"><a href="#4-3-2-其他类型的文件" class="headerlink" title="4.3.2. 其他类型的文件"></a>4.3.2. 其他类型的文件</h3><p>与 GenBank 文件类似，PhyloSuite 提供了两种方式将比对文件或 Microsoft Word 文档文件导入到 <strong>Other_File</strong> 根文件夹下的工作文件夹中：</p>
<ol>
<li>使用 <strong>文件</strong> 菜单下的 <code>导入文件或ID</code> 功能或<a href="#operation">显示区域</a>中的 <code>打开文件</code>。</li>
<li>将文件拖放到<a href="#operation">显示区域</a>中。</li>
</ol>
<p>导入文件后，同样有许多选项可以使用：</p>
<ul>
<li>对于多序列文件，将显示文件中的序列数量和比对状态（已比对或未比对）。</li>
<li>可以删除、导出或添加文件和序列。</li>
<li>比对可以直接用作任何（相关）插件功能的输入文件：<code>MAFFT</code>、<code>Gblocks</code>、<code>串联序列</code>、<code>转换序列格式</code>、<code>序列查看器</code>、<code>Partitionfinder2</code>、<code>ModelFinder</code>、<code>IQ-TREE</code> 和 <code>MrBayes</code> 功能。</li>
<li><code>解析注释</code> 功能仅适用于 *.docx 文件。</li>
<li>所有比对都可以在 <code>序列查看器</code> 中管理，它们可以被_反向_、<em>互补</em>、<em>反向互补_和_修剪</em>。</li>
</ul>
<p><span style="color: red">请注意，FASTA 格式文件也可以导入到 GenBank_File 根文件夹下的任何工作文件夹中，在这种情况下，文件将自动转换为 GenBank 文件格式（详见 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/five_ways_to_import/">五种将数据导入 PhyloSuite 的方法</a >）。</span></p>
<h3 id="4-3-3-在-NCBI-中搜索"><a href="#4-3-3-在-NCBI-中搜索" class="headerlink" title="4.3.3. 在 NCBI 中搜索"></a>4.3.3. 在 NCBI 中搜索</h3><p>您可以通过 <code>文件 --&gt; 在 NCBI 中搜索</code> 功能从 NCBI 的核苷酸和蛋白质数据库中搜索序列。</p>
<h4 id="4-3-3-1-简要示例"><a href="#4-3-3-1-简要示例" class="headerlink" title="4.3.3.1. 简要示例"></a>4.3.3.1. 简要示例</h4><ol>
<li>   打开菜单栏中的 <code>文件--&gt;在 NCBI 中搜索</code>；</li>
<li>   输入关键词（<code>Monogenea[ORGN] AND (mitochondrion[TITL] OR mitochondrial[TITL]) AND 10000:50000[SLEN]</code>）；</li>
<li>   输入您的电子邮件以告知 NCBI 是谁在下载序列；</li>
<li>   按 Enter 键或点击搜索按钮开始搜索；</li>
<li>   搜索完成后，选择一个工作文件夹来存放序列；选择 <code>GenBank_File</code> 根文件夹下的工作文件夹将以 GenBank 格式下载序列，而选择 <code>Other_File</code> 根文件夹下的工作文件夹将以 FASTA 格式下载序列；</li>
<li>   点击 <code>下载</code> 按钮开始下载。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/search_ncbi.jpg?raw=true"></li>
</ol>
<h2 id="4-4-GenBank-文件设置"><a href="#4-4-GenBank-文件设置" class="headerlink" title="4.4. GenBank 文件设置"></a>4.4. <em>GenBank 文件设置</em></h2><p>GenBank 文件的格式如下所示（有关详细的 GenBank 格式，请访问 <a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html">https://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html</a>）：</p>
<pre>
FEATURES             Location/Qualifiers
    source[<span id="feature" style="color: red">Feature</span>] 1..5028
                    /organism[<span id="qualifier" style="color: red">Qualifier</span>]  ="Saccharomyces cerevisiae[<span id="value" style="color: red">Value or Name</span>]"
                    /db_xref[<span style="color: red">Qualifier</span>]   ="taxon:4932[<span style="color: red">Value or Name</span>]"
                    /chromosome[<span style="color: red">Qualifier</span>]="IX[<span style="color: red">Value or Name</span>]"
                    /map[<span style="color: red">Qualifier</span>]       ="9[<span style="color: red">Value or Name</span>]"
    CDS[<span style="color: red">Feature</span>]    1..206
                    /product[<span style="color: red">Qualifier</span>]   ="TCP1-beta[<span style="color: red">Value or Name</span>]"
</pre>

<h3 id="4-4-1-谱系识别"><a href="#4-4-1-谱系识别" class="headerlink" title="4.4.1. 谱系识别"></a>4.4.1. 谱系识别</h3><p><span id="Recognition">这</span>可以在 <code>设置--&gt;设置--&gt;分类识别</code> 中设置。<br>您可以定义每个<a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Taxonomic_rank" title="taxonomic rank">分类层级</a>的标识符。它支持<a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Wildcard_character" title="Wildcard character">通配符</a> <code>*</code>，例如，大多数科名以 “dae” 结尾，因此您可以将科分类层级定义为 <code>*dae</code>。然而，在某些分类群中，层级名称不一定遵循相同的规则，例如在 Malacostraca 中，Hoplocarida 和 Peracarida 都以 “carida” 结尾，但它们分别是亚纲和总目。在这种情况下，最好使用全名作为标识符。此外，当您使用通配符时，可以排除某些术语；例如，如果您使用 <code>*oda</code> 来识别甲壳动物的目，则必须通过在新的 order 列中添加 <code>-Arthropoda</code> 来排除 Arthropoda，因为它是一个门。</p>
<p>&#8195;可以添加或删除分类层级。例如，您可以通过 <code>添加列</code> 按钮添加一个 <code>亚目</code>，或者通过选择任意分类层级并点击 <code>删除列</code> 按钮来删除它。请确保分类层级从左到右按从高到低的级别排列。您可以通过拖拽来更改分类层级的顺序。每当您更改设置时，可以通过在显示区域的右键菜单中选择 <code>刷新表格</code> 来更新表格。</p>
<p>&#8195;请注意，<a href="#operation">显示区域</a>（GenBank_File 工作区）中列出的每个 ID&#x2F;物种 的分类谱系是从 GenBank 文件中自动识别的。如果您愿意，可以通过所选 ID&#x2F;物种 的上下文（右键）菜单将其替换为来自 NCBI 分类数据库（<a target="_blank" rel="noopener" href="https://www.ncbi.nlm.nih.gov/taxonomy">https://www.ncbi.nlm.nih.gov/taxonomy</a>）或 WoRMS 数据库（<a target="_blank" rel="noopener" href="http://www.marinespecies.org/index.php">http://www.marinespecies.org/index.php</a>）的分类数据。</p>
<h3 id="4-4-2-信息显示与修改"><a href="#4-4-2-信息显示与修改" class="headerlink" title="4.4.2. 信息显示与修改"></a>4.4.2. 信息显示与修改</h3><p><span id="display">对于</span>每个工作文件夹（GenBank_File 下的），您可以定义将显示哪些信息。这可以在 <code>设置--&gt;GenBank 文件信息显示</code> 中设置（您应事先选择一个工作文件夹），或者点击每个 GenBank_File 工作文件夹名称右侧的 <code>GenBank 文件信息显示设置</code> 按钮。</p>
<p>GenBank 文件中有大量数据可以显示，因此 PhyloSuite 将它们分为四个部分：注释、谱系、参考文献和来源（来自 source 特征的<a href="#qualifier">限定符</a>）。</p>
<table>
<thead>
<tr>
<th>注释</th>
<th>谱系</th>
<th>参考文献</th>
<th>来源</th>
</tr>
</thead>
<tbody><tr>
<td>ID</td>
<td>纲</td>
<td>作者</td>
<td>host</td>
</tr>
<tr>
<td>长度</td>
<td>目</td>
<td>标题</td>
<td>specimen_voucher</td>
</tr>
<tr>
<td>AT%</td>
<td>总科</td>
<td>期刊</td>
<td>collection_date</td>
</tr>
<tr>
<td>名称</td>
<td>科</td>
<td>pubmed ID</td>
<td>isolation_source</td>
</tr>
<tr>
<td>物种</td>
<td>亚科</td>
<td>评论</td>
<td>country</td>
</tr>
<tr>
<td>定义</td>
<td>属</td>
<td></td>
<td>collected_by</td>
</tr>
<tr>
<td>日期</td>
<td>…</td>
<td></td>
<td>organelle</td>
</tr>
<tr>
<td>关键词</td>
<td></td>
<td></td>
<td>note</td>
</tr>
<tr>
<td>分子类型</td>
<td></td>
<td></td>
<td>mol_type</td>
</tr>
<tr>
<td>拓扑结构</td>
<td></td>
<td></td>
<td>strain</td>
</tr>
<tr>
<td>Accessions</td>
<td></td>
<td></td>
<td>db_xref</td>
</tr>
<tr>
<td>序列版本</td>
<td></td>
<td></td>
<td>…</td>
</tr>
<tr>
<td>来源</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>最后修改日期</td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody></table>
<p><span style="color: red">请注意，<code>谱系</code> 和 <code>来源</code> 的信息是可变的。<code>谱系</code> 可以配置（参见<a href="#Recognition">谱系识别</a>）。所有 ID 的 GenBank 文件的 <code>source</code> <a href="#feature">特征</a>中的所有限定符将作为可用选项呈现。这样，<code>来源</code> 的可用选项取决于此工作文件夹中的 GenBank 文件。</span></p>
<p>此外，某些信息可以通过双击相应的单元格进行修改。例如，谱系名称可能存在一些错误，您可以在<a href="#operation">显示区域</a>中更正。新名称随后将用于 <a href="#extracter">GenBank 文件提取器</a> 和其他功能。无法修改的字段是 “ID”、”长度”、”AT%”、”状态”、”日期”、”最后修改日期”、”作者”、”标题”、”期刊”、”pubmed ID”、”评论”、”关键词”、”分子类型”、”拓扑结构”、”Accessions”、”序列版本”、”来源”、”状态”。</p>
<h3 id="4-4-3-特征提取"><a href="#4-4-3-特征提取" class="headerlink" title="4.4.3. 特征提取"></a>4.4.3. 特征提取</h3><p><span id="extract">这是</span>一个灵活的功能，详细信息可以在 <code>设置--&gt;GenBank 文件提取</code> 中设置。<br>主要有三个步骤：</p>
<ul>
<li>首先，您可以定义希望提取哪些<a href="#feature">特征</a>，例如 <strong>CDS</strong>、<strong>rRNA</strong> 和 <strong>tRNA</strong> 等。</li>
<li>其次，您可以定义 GenBank 文件<a href="#qualifier">限定符</a>的值，该值将用作特征的名称。例如，您可以选择仅提取 <strong>rRNA</strong> 的限定符 <code>product</code> 的值，同时选择提取 <strong>CDS</strong> 的限定符 <code>gene</code> 和 <code>product</code> 的值。在这种情况下，PhyloSuite 将首先搜索 CDS 的 <code>gene</code> 的<a href="#value">值&#x2F;名称</a>，如果没有 <code>gene</code> 限定符，它将搜索 <code>product</code> 的<a href="#value">值&#x2F;名称</a>（<span style="color:red">注意限定符可以通过拖拽重新排序</span>）。如果在特征中找不到任何指定的限定符，在使用 <a href="#extracter">GenBank 文件提取器</a> 功能时，它将被记录在一个表格文件中，或者在使用标准化功能时被标记为错误。</li>
<li>最后，您可以通过 <code>名称统一</code> 表替换上一步中搜索到的<a href="#value">值&#x2F;名称</a>来统一数据集的注释。当使用 <a href="#extracter">GenBank 文件提取器</a> 或标准化功能时，如果在相应的限定符中找到 ‘旧名称’，它将被替换为 ‘新名称’。如果您希望仅提取此表中<a href="#value">值&#x2F;名称</a>可用的特征（基因）子集，您可以通过勾选 <code>仅提取这些基因</code> 复选框来实现。此表可以导出，或从逗号分隔表 (*.csv) 导入（表下方的 <code>导出/导入设置</code> 功能）。有一个统一名称的便捷方法：如果您第一次没有任何设置就提取基因，将生成一个 “name_for_unification.csv” 文件，可用于设置新名称，然后导入到设置中（必须使用 “csv” 格式）。<span style="color:red">请注意，所有特征的所有限定符的值&#x2F;名称都包含在单个表中。</span></li>
</ul>
<p>默认情况下，PhyloSuite 为六种数据类型（基因座）提供了设置：线粒体基因组、叶绿体基因组、通用、cox1、16S 和 18S。您可以根据需要添加更多数据类型，并通过 <code>当前版本</code> 按钮（左下角）在它们之间切换。允许您为每种数据类型关联不同的设置。线粒体基因组数据类型的三个特征（CDS、tRNA 和 rRNA）是固定的，因此无法删除，但可以添加新特征。如果您不确定使用哪种数据类型，可以选择 <code>通用</code>，然后根据您的需要调整设置。</p>
<h4 id="4-4-3-1-简要示例"><a href="#4-4-3-1-简要示例" class="headerlink" title="4.4.3.1. 简要示例"></a>4.4.3.1. 简要示例</h4><p>请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/customize_extraction/">https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/customize_extraction&#x2F;</a>。</p>
<h2 id="4-5-文件操作"><a href="#4-5-文件操作" class="headerlink" title="4.5. 文件操作"></a>4.5. <em>文件操作</em></h2><h3 id="4-5-1-输入文件"><a href="#4-5-1-输入文件" class="headerlink" title="4.5.1. 输入文件"></a>4.5.1. 输入文件</h3><p><span id="input">对于</span> PhyloSuite 中实现的功能的输入文件，您可以让软件从工作区__自动检测__它们，或者您可以自己__指定输入文件__。</p>
<h4 id="4-5-1-1-自动检测输入文件"><a href="#4-5-1-1-自动检测输入文件" class="headerlink" title="4.5.1.1. 自动检测输入文件"></a>4.5.1.1. 自动检测输入文件</h4><p><span id="autodetect">PhyloSuite</span> 可以自动检测并为每个功能准备输入文件。例如，<code>IQ-TREE</code> 功能接受 <code>串联序列 (concatenate_results)</code>、<code>Partitionfinder2 (PartFind_results)</code>、<code>ModelFinder (ModelFinder_results)</code> 和 <code>Other_File</code> 中的 <code>比对文件</code> 的结果。<br>此功能可以通过三种方式触发：</p>
<ol>
<li>如果您在选择了列出的文件夹或比对文件的情况下打开 <strong>IQ-TREE</strong>，它们将自动加载到 <strong>IQ-TREE</strong> 中。<br><br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE-autoinput.jpg?raw=true"></li>
<li>每次您打开 <strong>IQ-TREE</strong> 时，PhyloSuite 都会搜索整个工作区，并为 <strong>IQ-TREE</strong> 排序所有可接受的输入文件。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/IQ-TREE_en.jpg?raw=true"></li>
<li>如果您在 <strong>IQ-TREE</strong> 界面中没有输入文件，点击输入框将打开步骤 2 中的选择。</li>
</ol>
<p>输入文件和功能之间的关系总结如下：</p>
<table>
<thead>
<tr>
<th>功能</th>
<th>输入文件</th>
</tr>
</thead>
<tbody><tr>
<td>IQ-TREE</td>
<td>concatenate_results, PartFind_results, ModelFinder_results 和比对文件</td>
</tr>
<tr>
<td>MrBayes</td>
<td>PartFind_results, ModelFinder_results 和比对文件</td>
</tr>
<tr>
<td>ModelFinder</td>
<td>concatenate_results 和比对文件</td>
</tr>
<tr>
<td>PartitionFinder2</td>
<td>concatenate_results</td>
</tr>
<tr>
<td>MAFFT</td>
<td>extract_results 和比对文件</td>
</tr>
<tr>
<td>MACSE</td>
<td>mafft_results, extract_results 和比对文件</td>
</tr>
<tr>
<td>Gblocks</td>
<td>MACSE_results, mafft_results, concatenate_results 和比对文件</td>
</tr>
<tr>
<td>trimAl</td>
<td>MACSE_results, mafft_results, concatenate_results 和比对文件</td>
</tr>
<tr>
<td>HmmCleaner</td>
<td>MACSE_results, mafft_results, concatenate_results 和比对文件</td>
</tr>
<tr>
<td>转换格式</td>
<td>MACSE_results, mafft_results, Gblocks_results, trimAl_results, HmmCleaner_results 和比对文件</td>
</tr>
<tr>
<td>串联序列</td>
<td>MACSE_results, mafft_results, Gblocks_results, trimAl_results, HmmCleaner_results 和比对文件</td>
</tr>
</tbody></table>
<p>这里的比对文件指的是列在 <strong>Other_File</strong> 根文件夹中的比对文件。<br>结果文件夹名称请参考<a href="#Output">输出文件</a></p>
<h4 id="4-5-1-2-指定输入文件"><a href="#4-5-1-2-指定输入文件" class="headerlink" title="4.5.1.2. 指定输入文件"></a>4.5.1.2. 指定输入文件</h4><p>有两种方式：</p>
<ol>
<li>将文件拖入 “输入” 框；</li>
<li>点击输入框右侧的 ‘打开文件夹’ 图标。</li>
</ol>
<h3 id="4-5-2-输出文件"><a href="#4-5-2-输出文件" class="headerlink" title="4.5.2. 输出文件"></a>4.5.2. 输出文件</h3><p><span id="Output">所有</span>功能的结果将自动保存在工作区中。如果您选择了一个工作文件夹，则结果将保存到该工作文件夹中。如果您没有选择，结果将保存到 <code>GenBank_File/files</code> 或 <code>Other_File/files</code>。您也可以通过 <code>开始</code> 按钮的下拉箭头更改结果文件夹名称，以及选择另一个工作文件夹来存放您的结果。</p>
<p>功能和默认结果文件夹：</p>
<table>
<thead>
<tr>
<th>功能</th>
<th>结果文件夹</th>
</tr>
</thead>
<tbody><tr>
<td>IQ-TREE</td>
<td>IQtree_results</td>
</tr>
<tr>
<td>MrBayes</td>
<td>MrBayes_results</td>
</tr>
<tr>
<td>ModelFinder</td>
<td>ModelFinder_results</td>
</tr>
<tr>
<td>PartitionFinder2</td>
<td>PartFind_results</td>
</tr>
<tr>
<td>MAFFT</td>
<td>mafft_results</td>
</tr>
<tr>
<td>MACSE</td>
<td>MACSE_results</td>
</tr>
<tr>
<td>Gblocks</td>
<td>Gblocks_results</td>
</tr>
<tr>
<td>trimAl</td>
<td>trimAl_results</td>
</tr>
<tr>
<td>HmmCleaner</td>
<td>HmmCleaner_results</td>
</tr>
<tr>
<td>转换格式</td>
<td>convertFmt_results</td>
</tr>
<tr>
<td>串联序列</td>
<td><span id="concatenate_results">concatenate_results</span></td>
</tr>
<tr>
<td>绘制 RSCU 图</td>
<td>RSCUfig_results</td>
</tr>
<tr>
<td>比较表格</td>
<td>comp_tbl_results</td>
</tr>
<tr>
<td>工作流</td>
<td>Flowchart_reports</td>
</tr>
</tbody></table>
<h4 id="4-5-2-1-简要示例"><a href="#4-5-2-1-简要示例" class="headerlink" title="4.5.2.1. 简要示例"></a>4.5.2.1. 简要示例</h4><p>请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/quick_start/#7-获取结果">此处</a>。</p>
<h1 id="5-数据分析"><a href="#5-数据分析" class="headerlink" title="5. 数据分析"></a>5. 数据分析</h1><h2 id="5-1-提取-GenBank-文件"><a href="#5-1-提取-GenBank-文件" class="headerlink" title="5.1. 提取 GenBank 文件"></a>5.1. <em>提取 GenBank 文件</em></h2><p><span id="extracter">此</span>功能的输入文件只能通过选择 <a href="#operation">GenBank_File 根文件夹</a>下的工作文件夹的<a href="#operation">显示区域</a>中的 ID 来加载。关于结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>有两种提取模式，<code>单基因座</code> 模式将提取整个序列但忽略注释和其他特征，适用于单基因座，如 18S、cox1 和 28S 等；<code>自定义</code> 模式允许您选择或编辑希望提取的序列类型和特征（参见 <a href="#extract">GenBank 文件提取设置</a>）。</p>
<p>它能做什么：</p>
<ul>
<li>提取在 <a href="#extract">GenBank 文件提取设置</a> 中定义（选择）的基因，并以 fasta 格式保存它们。例如，如果您选择提取 <code>CDS</code>、<code>tRNA</code> 和 <code>rRNA</code> <a href="#feature">特征</a>，此功能将从所有选定的 GenBank 文件中提取这些特征，并将它们存储在相应命名的文件夹（CDS, tRNA, rRNA）中。此外，<code>CDS</code> 特征将被分成两个文件夹：’CDS_AA’ 文件夹包含从 <code>translation</code> <a href="#qualifier">限定符</a> 提取的氨基酸序列，而 ‘CDS_NUC’ 文件夹包含核苷酸序列。对于<a href="#extract">线粒体基因组</a>版本，还有一个额外的 “self-translated_AA” 文件夹，其中包含由 PhyloSuite 从核苷酸序列（CDS）翻译而来的氨基酸序列。<span style="color:red">请注意，一个 ID 内可能存在重复的基因，在这种情况下，PhyloSuite 将按出现顺序对重复的基因名称进行编号。例如，如果有三个 <code>cox1</code> 基因，那么它们将被保存为 <code>cox1.fas</code>、<code>cox1_copy2.fas</code> 和 <code>cox1_copy3.fas</code></span>。此外，PhyloSuite 还提供了 <code>解析基因重复</code> 功能（在 <code>参数</code> 选项卡中可用）来自动识别和移除重复的基因。</li>
<li>提取重叠区和基因间区。</li>
<li>生成统计文件和其他分析使用的文件：<ul>
<li>生成提取概览文件 (<code>overview.csv</code>)，其中记录了用于提取的数据类型设置、序列中发现的所有特征、缺失的特征或限定符，以及每个物种中发现的基因。</li>
<li>数据集中包含的物种（ID）信息，包括物种名称、谱系、A&#x2F;T&#x2F;C&#x2F;G 含量以及 AT&#x2F;GC 偏度。[<span style="color: red">StatFiles&#x2F;used_species.csv</span>]</li>
<li>用于编辑 <a href="#extract">GenBank 文件提取设置</a> 中 <code>名称统一</code> 表的名称表。使用此表，您可以修改 ‘新名称’ 列中的名称，然后将其导入到 <code>名称统一</code> 表中。在首次提取基因时，此表非常有用。[<span style="color: red">StatFiles&#x2F;name_for_unification.csv</span>]</li>
<li>如果勾选了 <code>仅提取这些基因</code> 且没有任何限定符值符合 <code>名称统一</code> 表中的名称，则这些值将被记录在 <code>name_not_included.csv</code> 表中。[<span style="color: red">StatFiles&#x2F;name_not_included.csv</span>]</li>
<li>线粒体基因组的总体统计，包括全基因组的核苷酸组成、蛋白质编码基因（PCGs）、rRNA 基因和 tRNA 基因。[<span style="color: red">StatFiles&#x2F;used_species.csv，线粒体基因组版本</span>]</li>
<li>每个 PCG 和 rRNA 基因的起始和终止密码子、核苷酸含量、偏度以及长度统计。[<span style="color: red">StatFiles&#x2F;geneStat.csv，线粒体基因组版本</span>]</li>
<li>PCGs 每个密码子位点的核苷酸偏度。[<span style="color: red">StatFiles&#x2F;CDS&#x2F;[PCGsCodonSkew.csv | firstCodonSkew.csv | secondCodonSkew.csv | thirdCodonSkew.csv]，线粒体基因组版本</span>]</li>
<li>所有物种（ID）的单个元件和完整线粒体基因组的核苷酸含量和偏度（参见 <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1</a> 中的图 2 和 <a target="_blank" rel="noopener" href="https://doi.org/10.1186/s12862-018-1249-3">https://doi.org/10.1186/s12862-018-1249-3</a> 中的图 1）[<span style="color: red">StatFiles&#x2F;geom_line.csv，线粒体基因组版本</span>]</li>
<li>每个物种（ID）的核苷酸统计。[<span style="color: red">StatFiles&#x2F;speciesStat&#x2F;*IDs.csv，线粒体基因组版本</span>]</li>
<li>每个物种（ID）的组织结构表。[<span style="color: red">StatFiles&#x2F;speciesStat&#x2F;*IDs_org.csv，线粒体基因组版本</span>]</li>
<li>相对同义密码子使用表。请注意，在计算之前会移除简写的终止密码子（T–, TA-）。[<span style="color: red">StatFiles&#x2F;RSCU&#x2F;*IDs_RSCU.csv，线粒体基因组版本</span>]</li>
<li>氨基酸使用表。[<span style="color: red">StatFiles&#x2F;RSCU&#x2F;*IDs_AA_usage.csv，线粒体基因组版本</span>]</li>
</ul>
</li>
<li>制作 <a target="_blank" rel="noopener" href="https://itol.embl.de/" title="ITOL 网站">ITOL</a> 数据集（如果勾选 <code>ITOL 数据集</code> 复选框将被激活）<ul>
<li>这些是简单的 *.txt 文件，您可以直接将它们拖放到 iTOL 网络界面（<a target="_blank" rel="noopener" href="https://itol.embl.de">https://itol.embl.de</a>）中相应的系统树上。</li>
<li>批量替换末端标签。[<span style="color: red">itolFiles&#x2F;[itol_labels.txt | itol_gb_labels.txt | itol_ori_labels.txt]</span>]</li>
<li>为不同的谱系分配颜色。每个分类单元（或谱系）的颜色可以在 <code>谱系颜色</code> 中指定。如果您没有为所有分类单元选择颜色，PhyloSuite 将为剩余的分类单元随机分配颜色。要添加或移除谱系，请点击 <code>配置</code> 按钮（参见 <a target="_blank" rel="noopener" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0181699">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0181699</a> 中的图 1）。[<span style="color: red">itolFiles&#x2F;[itol_xxx_ColourStrip.txt | itol_xxx_Text.txt | itol_xxx_Colour.txt]</span>]</li>
<li>将柱状图映射到系统树上（参见 <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2245-y">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2245-y</a> 中的图 2）。[<span style="color: red">itolFiles&#x2F;[itolAT.txt | itolLength.txt | itolLength_stack.txt]</span>]</li>
<li>将基因顺序映射到系统树上。基因顺序显示中每个基因图标的颜色、长度和形状，以及图标之间的间距（基因间隔），可以使用 <code>基因顺序显示</code> 功能进行修改。在此步骤中，您还可以选择要可视化的 NCRs（如果您已在 <code>标准化</code> 步骤中设置了 PhyloSuite 来识别和提取它们，包括设置大小阈值）（参见 <a target="_blank" rel="noopener" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3">https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3</a> 中的图 6）。[<span style="color: red">files&#x2F;itol_gene_order.txt，线粒体基因组版本</span>]</li>
</ul>
</li>
<li>可用于使用 CREx 和&#x2F;或 treeREx 进行相关分析的基因顺序文件。</li>
</ul>
<p>在 <code>自定义</code> 菜单中，您可以选择在 <a href="#extract">GenBank 文件提取设置</a> 中预设的数据类型。在 <code>谱系</code> 菜单中，您可以选择在结果中包含哪些谱系。关于序列的名称，用户可以通过 <code>名称类型</code> 功能自定义，其中 ID、organism、科、纲、isolate、strain 等都是可用的。</p>
<h3 id="5-1-1-简要示例"><a href="#5-1-1-简要示例" class="headerlink" title="5.1.1. 简要示例"></a>5.1.1. <span id="extract_exam">简要</span>示例</h3><ol>
<li>选择要提取的 ID（参考<a href="#import_gb">此处</a>了解如何将 GenBank 记录导入 PhyloSuite）；</li>
<li>通过右键单击打开 <code>提取</code>，序列将自动导入；</li>
<li>参数可根据您自己的需要进行设置（如果您的数据是线粒体基因组，请选择 <code>线粒体基因组</code> 数据类型）；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/extract_demo.jpg?raw=true"></li>
</ol>
<p>关于自定义提取，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/customize_extraction/">自定义提取</a ><br>关于综合演示，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">多基因教程</a > 和 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">单基因教程</a >。关于如何使用生成的 iTOL 数据集，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/#10-系统发育树注释">系统发育树注释</a >。</p>
<h2 id="5-2-MAFFT-1"><a href="#5-2-MAFFT-1" class="headerlink" title="5.2. MAFFT"></a>5.2. <em>MAFFT</em></h2><p>关于 MAFFT 的安装，请参阅<a href="#plugins">插件安装</a>部分。关于 MAFFT 的输入文件，请参阅<a href="#input">输入文件</a>部分。<span style="color: red">请注意，输入文件应为 FASTA 格式。</span> 关于 MAFFT 的结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>PhyloSuite 使 MAFFT 能够使用同一组参数__批量运行多个文件__，这意味着您可以同时将多个文件输入到 MAFFT 中。PhyloSuite 为 MAFFT 提供了三种比对模式：</p></p>
<ol>
<li>常规模式：正常比对序列。</li>
<li>密码子模式（<span style="color: green">由 PhyloSuite 添加</span>）：首先将蛋白质编码基因的核苷酸序列翻译成 AA 序列，然后通过 MAFFT 比对 AA 序列，最后将 AA 比对结果反向翻译回相应的密码子。<span style="color: red">请注意，您应首先选择合适的密码子表。</span></li>
<li>N2P 模式（<span style="color: green">由 PhyloSuite 添加</span>）与前一模式相同，但省略最后一步（反向翻译）：将 PCGs 翻译成 AAs 并进行比对。结果是 AA 比对。</li>
</ol>
<p>当使用密码子模式进行比对时，如果存在内部终止密码子，PhyloSuite 将弹出一个警告窗口。如果您了解此问题，可以忽略它并继续比对（选择‘忽略’），否则终止比对并检查问题（选择‘是’）。<code>-adjustdirection</code> 可以自动调整某些序列的方向（即反向互补）。其他参数也可用，例如 <code>比对策略</code>、<code>导出格式</code> 和 <code>线程</code> 等。</p>
<p>输入文件并设置参数后，您可以点击 <code>开始</code> 按钮启动程序。可以通过 <code>显示日志</code> 按钮查看运行日志。程序完成后，<code>MAFFT</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-2-1-简要示例"><a href="#5-2-1-简要示例" class="headerlink" title="5.2.1. 简要示例"></a>5.2.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\MAFFT\mtDNA_36_genes\CDS_NUC’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择全部 12 个文件；</li>
<li>通过菜单栏打开 <code>比对--&gt;MAFFT</code>；</li>
<li>将所有 12 个序列拖放到文件输入框中；</li>
<li>参数可根据您自己的需要进行设置（确保为蛋白质编码基因选择正确的 <code>密码子表</code>，此处为 9）；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mafft_demo.jpg?raw=true"></li>
</ol>
<p>关于综合演示，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">多基因教程</a > 和 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">单基因教程</a >。关于 MAFFT 的完整手册，请访问 <a target="_blank" rel="noopener" href="https://mafft.cbrc.jp/alignment/software/manual/manual.html">https://mafft.cbrc.jp/alignment/software/manual/manual.html</a>。</p>
<h2 id="5-3-MACSE-1"><a href="#5-3-MACSE-1" class="headerlink" title="5.3. MACSE"></a>5.3. <em>MACSE</em></h2><p>关于 MACSE 的安装，请参阅<a href="#plugins">插件安装</a>部分。关于 MACSE 的输入文件，请参阅<a href="#input">输入文件</a>部分。<span style="color: red">请注意，输入文件应为 FASTA 格式。</span> 关于 MACSE 的结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>PhyloSuite 使 MACSE 能够使用同一组参数__批量运行多个文件__，这意味着您可以同时将多个文件输入到 MACSE 中。此外，还允许多核操作，允许多个文件（取决于设置的线程数）同时运行。</p></p>
<p>MACSE 有许多子程序，并且它本身已有 GUI，因此我们只将 <code>alignSequences</code> 和 <code>refineAlignment</code> 两个子程序添加到 PhyloSuite 中。我们认为这两个在弥补 MAFFT 的不足方面最适合 PhyloSuite。MACSE 的输入应该是蛋白质编码序列（用于 <code>alignSequences</code>）或其他程序（例如 <code>MAFFT</code>）生成的比对（用于 <code>refineAlignment</code>）。关于批量处理，如果 <code>Seq.</code> 和 <code>Seq_lr.</code> 框中都有多个文件，这些文件将依次组合，例如，<code>Seq.</code> 的第一个文件（-seq 1st_seq_file）将与 <code>Seq_lr.</code> 的第一个文件（-seq_lr 1st_seq_lr_file）组合。值得注意的是，<code>seq</code> 和 <code>seq_lr</code> 选项必须一起使用，或者在与 <code>Refine</code>（refineAlignment）结合使用时完全不使用。请注意，PhyloSuite 还在 ‘开始’ 按钮的下拉箭头中提供了 <code>查看 | 编辑命令</code> 功能，这为有经验的用户提供了足够的自由度来修改和添加 GUI 中未包含的参数。</p>
<p>特别是，由于生成的比对文件可能包含感叹号（<code>!</code>）或星号（<code>*</code>）符号（强调 MACSE 检测到的移码），这可能会导致下游分析出错。因此，PhyloSuite 会生成一个名称中包含 <code>_removed_chars_</code> 的附加文件，该文件将这些符号替换为 <code>?</code>。</p>
<p>输入文件并设置参数后，您可以点击 <code>开始</code> 按钮启动程序。可以通过 <code>显示日志</code> 按钮查看运行日志。程序完成后，<code>MACSE</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-3-1-简要示例"><a href="#5-3-1-简要示例" class="headerlink" title="5.3.1. 简要示例"></a>5.3.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\MACSE’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择全部 3 个文件；</li>
<li>通过菜单栏打开 <code>比对--&gt;MACSE (用于 CDS)</code>；</li>
<li>将所有 3 个序列拖放到 <code>Seq.</code> 输入框中；</li>
<li>参数可根据您自己的需要进行设置（确保选择正确的 <code>密码子表</code>；在示例中我们选择了 9）；</li>
<li>可以通过 ‘开始’ 按钮下拉箭头中的 <code>查看 | 编辑命令</code> 功能添加 GUI 中未包含的参数；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/MACSE_demo.jpg?raw=true"></li>
</ol>
<p>关于 MACSE 的完整手册，请访问 <a target="_blank" rel="noopener" href="https://bioweb.supagro.inra.fr/macse/index.php?menu=intro">https://bioweb.supagro.inra.fr/macse/index.php?menu=intro</a>。</p>
<h2 id="5-4-trimAl-1"><a href="#5-4-trimAl-1" class="headerlink" title="5.4. trimAl"></a>5.4. <em>trimAl</em></h2><p><span id="trimAl">关于</span> trimAl 的安装，请参阅<a href="#plugins">插件安装</a>部分。关于将文件输入 trimAl，请参阅<a href="#input">输入文件</a>部分。关于 trimAl 的结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>PhyloSuite 使 trimAl 能够使用同一组参数__批量运行多个文件__，这意味着您可以同时将多个文件输入到 trimAl 中。此外，还允许多核操作，允许多个文件（取决于设置的线程数）同时运行。请注意，PhyloSuite 还在 ‘开始’ 按钮的下拉箭头中提供了 <code>查看 | 编辑命令</code> 功能，这为有经验的用户提供了足够的自由度来修改和添加 GUI 中未包含的参数。</p></p>
<p>如果您想将 trimAl 的结果应用于下游分析，请确保选择 <code>fasta</code> 作为输出格式。如果您选择 <code>统计输出</code>，这些结果将保存到后缀为 “.log” 的文件中。由于输出文件扩展名 <code>_trimAl</code> 被下游功能识别，因此无法更改。</p>
<p>输入文件并设置参数后，点击 <code>开始</code> 按钮运行程序。您可以通过 <code>显示日志</code> 按钮查看运行日志。<code>trimAl</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-4-1-简要示例"><a href="#5-4-1-简要示例" class="headerlink" title="5.4.1. 简要示例"></a>5.4.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\trimAl_HmmCleaner’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择全部 3 个文件；</li>
<li>通过菜单栏打开 <code>比对--&gt;trimAl</code>；</li>
<li>将所有 3 个序列拖放到输入框中；</li>
<li>参数可根据您自己的需要进行设置；</li>
<li>可以通过 ‘开始’ 按钮下拉箭头中的 <code>查看 | 编辑命令</code> 功能添加 GUI 中未包含的参数；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/trimAl_demo.jpg?raw=true"></li>
</ol>
<p>关于 trimAl 的完整手册，请访问 <a target="_blank" rel="noopener" href="http://trimal.cgenomics.org">http://trimal.cgenomics.org</a>。</p>
<h2 id="5-5-HmmCleaner-1"><a href="#5-5-HmmCleaner-1" class="headerlink" title="5.5. HmmCleaner"></a>5.5. <em>HmmCleaner</em></h2><p><span id="HmmCleaner">关于</span> HmmCleaner 的安装，请参阅<a href="#plugins">插件安装</a>部分。关于将文件输入 HmmCleaner，请参阅<a href="#input">输入文件</a>部分。关于 HmmCleaner 的结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>PhyloSuite 使 HmmCleaner 能够使用同一组参数__批量运行多个文件__，这意味着您可以同时将多个文件输入到 HmmCleaner 中。此外，还允许多核操作，允许多个文件（取决于设置的线程数）同时运行。由于 HmmCleaner 的设计限制，此程序仅适用于 Linux 和 Mac 用户。如果您想将 HmmCleaner 的结果应用于下游分析，请确保取消勾选 <code>ali</code> 输出格式。</p>
<p>输入文件并设置参数后，点击 <code>开始</code> 按钮运行程序。您可以通过 <code>显示日志</code> 按钮查看运行日志。<code>HmmCleaner</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-5-1-简要示例"><a href="#5-5-1-简要示例" class="headerlink" title="5.5.1. 简要示例"></a>5.5.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\trimAl_HmmCleaner’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择全部 3 个文件；</li>
<li>通过菜单栏打开 <code>比对--&gt;HmmCleaner</code>；</li>
<li>将所有 3 个序列拖放到输入框中；</li>
<li>参数可根据您自己的需要进行设置；</li>
<li>可以通过 ‘开始’ 按钮下拉箭头中的 <code>查看 | 编辑命令</code> 功能添加 GUI 中未包含的参数；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/HmmCleaner_demo.jpg?raw=true"></li>
</ol>
<p>关于 HmmCleaner 的完整手册，请访问 <a target="_blank" rel="noopener" href="https://metacpan.org/pod/distribution/Bio-MUST-Apps-HmmCleaner/bin/HmmCleaner.pl">https://metacpan.org/pod/distribution/Bio-MUST-Apps-HmmCleaner/bin/HmmCleaner.pl</a>。</p>
<h2 id="5-6-Gblocks-1"><a href="#5-6-Gblocks-1" class="headerlink" title="5.6. Gblocks"></a>5.6. <em>Gblocks</em></h2><p><span id="gblocks">关于</span> Gblocks 的安装，请参阅<a href="#plugins">插件安装</a>部分。关于将文件输入 Gblocks，请参阅<a href="#input">输入文件</a>部分。<span style="color: red">请注意，输入文件应为 FASTA 或 NBRF&#x2F;PIR 格式。</span> 关于 Gblocks 的结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>PhyloSuite 使 Gblocks 能够使用同一组参数__批量运行多个文件__，这意味着您可以同时将多个文件输入到 Gblocks 中。<br><code>保守位置的最小序列数</code> 和 <code>侧翼位置的最小序列数</code> 这两个选项将在输入文件后启用。因为前者变量必须 &gt; 序列数的一半，而后者变量必须 ≥ 前者变量的值。这两个变量的可用值将根据此规则变化。<span style="color: red">因此，在对多个文件进行批量分析时，每个文件中的序列数必须相同。</span></p>
<p>由于默认的输出文件扩展名 ‘_gb’ 被下游功能识别，因此无法更改。</p>
<p>输入文件并设置参数后，点击 <code>开始</code> 按钮运行程序。您可以通过 <code>显示日志</code> 按钮查看运行日志。<code>Gblocks</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-6-1-简要示例"><a href="#5-6-1-简要示例" class="headerlink" title="5.6.1. 简要示例"></a>5.6.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\Gblocks\mtDNA_36_genes\CDS_NUC’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择全部 12 个文件；</li>
<li>通过菜单栏打开 <code>比对--&gt;Gblocks</code>；</li>
<li>将所有 12 个序列拖放到文件输入框中；</li>
<li>参数可根据您自己的需要进行设置（确保选择合适的数据类型，此处应为密码子）；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/gblocks_demo.jpg?raw=true"></li>
</ol>
<p>关于 Gblocks 的完整手册，请访问 <a target="_blank" rel="noopener" href="http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_documentation.html">http://molevol.cmima.csic.es/castresana/Gblocks/Gblocks_documentation.html</a>。</p>
<h2 id="5-7-串联序列"><a href="#5-7-串联序列" class="headerlink" title="5.7. 串联序列"></a>5.7. <em>串联序列</em></h2><p>关于输入文件，请参阅<a href="#input">输入文件</a>部分。允许 FASTA、PHYLIP、AXT、PAML 和 NEXUS 格式。关于结果，请参阅<a href="#Output">输出文件</a>部分。<span style="color: red">请注意，输出文件的名称可以更改。</span></p>
<p>可以使用此功能将多个比对串联成一个单一的比对。首先，PhyloSuite 将扫描每个比对并收集所有序列名称，然后通过在每个比对中搜索名称来串联这些比对。如果在比对中找不到某个名称，它将被记录在 ‘missing_genes.txt’ 文件中。</p>
<p>可以为输出文件选择多种常见格式，例如 PHYLIP、NEXUS、AXT、PAML 和 FASTA。此外，该功能可以记录串联过程中每个基因的索引，并生成一个分区文件，该文件可用于 <code>PartitionFinder</code>、<code>ModelFinder</code>、<code>IQ-TREE</code> 和 <code>MrBayes</code>。用户还可以选择在此分析中移除任何密码子位点（例如第三密码子位点）。</p>
<p>您可以通过拖拽文件重新排序来更改比对串联的顺序。</p>
<h3 id="5-7-1-简要示例"><a href="#5-7-1-简要示例" class="headerlink" title="5.7.1. 简要示例"></a>5.7.1. 简要示例</h3><p><span id="concatenation">当</span>您在 PhyloSuite 根目录下时，进入 ‘example\Concatenation\mtDNA_36_genes\36_genes_NUC’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择全部 36 个文件；</li>
<li>通过菜单栏打开 <code>比对--&gt;串联序列</code>；</li>
<li>将所有序列拖放到文件输入框中；</li>
<li>输出格式可根据您自己的需要选择，如果您希望可视化串联数据集，也可以选择 <code>线性图</code> 功能；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/concatenation_demo.jpg?raw=true"></li>
</ol>
<p>关于综合演示，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">多基因教程</a > 和 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">单基因教程</a >。</p>
<h2 id="5-8-转换格式"><a href="#5-8-转换格式" class="headerlink" title="5.8. 转换格式"></a>5.8. <em>转换格式</em></h2><p>关于输入文件，请参阅<a href="#input">输入文件</a>部分。关于结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>支持 PHYLIP、NEXUS、AXT、PAML 和 FASTA 格式（输入和输出文件均支持）。此功能还支持__批量格式转换__，这意味着您可以同时输入多个文件。</p>
<h3 id="5-8-1-简要示例"><a href="#5-8-1-简要示例" class="headerlink" title="5.8.1. 简要示例"></a>5.8.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\Convert_format’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择 ‘cox1_AA_mafft.fas’ 和 ‘cox1_NUC_mafft.fas’；</li>
<li>通过菜单栏打开 <code>比对--&gt;转换序列格式</code>；</li>
<li>将它们拖放到文件输入框中；</li>
<li>选择输出格式；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/convertFMT_demo.jpg?raw=true"></li>
</ol>
<h2 id="5-9-ModelFinder-1"><a href="#5-9-ModelFinder-1" class="headerlink" title="5.9. ModelFinder"></a>5.9. <em>ModelFinder</em></h2><p>关于 ModelFinder (IQ-TREE) 的安装，请参阅<a href="#plugins">插件安装</a>部分，关于输入比对文件请参阅<a href="#input">输入文件</a>部分（允许 FASTA、PHYLIP、NEXUS 和 CLUSTAL 格式），关于结果文件请参阅<a href="#Output">输出文件</a>部分。</p>
<p>您可以选择提供两个额外的可选文件：一个树文件（<span style="color: red">newick 格式</span>）和一个分区文件。关于分区文件的格式，请参阅 <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/Advanced-Tutorial">http://www.iqtree.org/doc/Advanced-Tutorial</a>。最方便的选项是直接使用 <code>串联序列</code> (<a href="#concatenate_results">concatenate_results</a>) 的结果作为 ModelFinder 的输入文件。串联的比对和分区文件将自动加载到 ModelFinder 中（参见下面的简要示例）。</p>
<p>PhyloSuite 为 ModelFinder 设置提供了一个附加参数：<code>模型用于</code>。此参数允许您选择一组希望测试的模型，适用于不同的系统发育程序（参见下表）。这非常有用，因为不同的算法通常使用不同的模型类型。</p>
<table>
<thead>
<tr>
<th>选项</th>
<th>ModelFinder 中的对应参数</th>
</tr>
</thead>
<tbody><tr>
<td>MrBayes</td>
<td>-m TESTONLY -mset mrbayes</td>
</tr>
<tr>
<td>RaxML</td>
<td>-m TESTONLY -mset raxml</td>
</tr>
<tr>
<td>PhyML</td>
<td>-m TESTONLY -mset phyml</td>
</tr>
<tr>
<td>IQ-TREE</td>
<td>-m TESTNEWONLY</td>
</tr>
<tr>
<td>BEAST1</td>
<td>-mset JC69,TrN,TrNef,K80,K2P,F81,HKY,SYM,TIM,TVM,TVMef,GTR -mrate E,G</td>
</tr>
<tr>
<td>BEAST2</td>
<td>-mset JC69,TrN,TrNef,K80,K2P,F81,HKY,SYM,TIM,TVM,TVMef,GTR -mrate E,G</td>
</tr>
</tbody></table>
<p>输入文件并设置参数后，您可以点击 <code>开始</code> 按钮运行程序。可以通过 <code>显示日志</code> 按钮查看运行日志。程序完成后，<code>IQ-TREE</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-9-1-简要示例"><a href="#5-9-1-简要示例" class="headerlink" title="5.9.1. 简要示例"></a>5.9.1. 简要示例</h3><ol>
<li>右键单击 <code>concatenate_results</code> 文件夹中的一个结果（如果不可用，请参阅<a href="#concatenation">此处</a>了解如何创建），然后在上下文菜单中选择 <code>导入到 ModelFinder</code>；</li>
<li>带有每个基因位置索引的串联数据集将自动导入；</li>
<li>双击文本框或点击编辑按钮打开分区编辑器窗口以配置 <code>数据块</code>（关于如何操作分区编辑器，请参阅<a href="#partition_editor">下文</a>）；</li>
<li>参数可根据您自己的需要进行设置；可以通过 ‘开始’ 按钮下拉箭头中的 <code>查看 | 编辑命令</code> 功能添加 GUI 中未包含的参数；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/modelfinder_demo_2.jpg?raw=true"></li>
</ol>
<p>关于综合演示，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">多基因教程</a > 和 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">单基因教程</a >。关于 ModelFinder 的完整手册，请访问 <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/">http://www.iqtree.org/doc/</a> 和 <a target="_blank" rel="noopener" href="http://iqtree.cibiv.univie.ac.at/">http://iqtree.cibiv.univie.ac.at/</a>。</p>
<h2 id="5-10-PartitionFinder-1"><a href="#5-10-PartitionFinder-1" class="headerlink" title="5.10. PartitionFinder"></a>5.10. <em>PartitionFinder</em></h2><p>关于 PartitionFinder2 的安装，请参阅<a href="#plugins">插件安装</a>部分，关于输入比对文件（<span style="color: red">PHYLIP 格式</span>）请参阅<a href="#input">输入文件</a>部分，关于结果请参阅<a href="#Output">输出文件</a>部分。您也可以提供一个树文件（<span style="color: red">可选，newick 格式</span>）。</p>
<p>使用 PartitionFinder2 最方便的方法是使用 <code>串联序列</code> (<a href="#concatenate_results">concatenate_results</a>) 的结果作为输入文件。串联的比对和分区文件将自动加载到 PartitionFinder2 中。</p>
<p>PartitionFinder2 需要一个数据块才能运行，其默认格式如下（参见 <code>数据块</code> 窗口）：</p>
<blockquote>
<p>Gene1_codon1 &#x3D; 1-999\3;<br>Gene1_codon2 &#x3D; 2-999\3;<br>Gene1_codon3 &#x3D; 3-999\3;<br>Gene2 &#x3D; 1000-1665;<br>intron &#x3D; 1666-2000;</p>
</blockquote>
<p>PhyloSuite 提供了一个分区编辑器功能，您可以在其中添加&#x2F;删除&#x2F;修改分区，并将选定的数据块转换&#x2F;取消转换为密码子格式。关于如何使用此功能，请参阅<a href="#partition_editor">下文</a>。</p>
<p>请注意，在 <code>命令行选项</code> 中，<code>--all-states</code> 和 <code>--min-subset-size</code> 仅当在 <code>搜索</code> 菜单中选择 <code>kmeans</code> 时才能使用。<code>搜索</code> 菜单中的 <code>hcluster</code>、<code>rclusterf</code> 和 <code>rcluster</code> 选项以及 <code>命令行选项</code> 中的 <code>--rcluster-max</code> 和 <code>--weights</code> 仅当在 <code>命令行选项</code> 中勾选了 <code>--raxml</code> 时才会启用。应谨慎使用 <code>分支长度</code> 菜单中的 <code>unlinked</code> 选项，因为在使用分区结果在 MrBayes 中进行分析时，它可能会阻碍收敛（因为 <code>unlink brlens=(all);</code>）。</p>
<p>输入文件并设置参数后，您可以启动程序（<code>开始</code> 按钮），并通过 <code>显示日志</code> 按钮查看运行日志。程序完成后，<code>PartitionFinder2</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-10-1-简要示例"><a href="#5-10-1-简要示例" class="headerlink" title="5.10.1. 简要示例"></a>5.10.1. 简要示例</h3><p>PhyloSuite 的一个设计特点是 <code>串联</code> 的输出与 <code>PartitionFinder2</code> 的输入之间有直接链接：</p>
<ol>
<li>右键单击 <code>concatenate_results</code> 文件夹中的一个结果（如果不可用，请参阅<a href="#concatenation">此处</a>了解如何创建），然后在上下文菜单中选择 <code>导入到 PartitionFinder2</code>；</li>
<li>带有每个基因位置索引的串联数据集将自动导入；</li>
<li>双击文本框或点击编辑按钮打开分区编辑器窗口以配置 <code>数据块</code>（关于如何操作分区编辑器，请参阅<a href="#partition_editor">下文</a>）；</li>
<li>其他参数可根据您自己的需要进行设置（确保选择正确的数据类型）；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/pf_demo_2.jpg?raw=true"></li>
</ol>
<p>关于综合演示，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">多基因教程</a >。关于 PartitionFinder2 的完整手册，请访问 <a target="_blank" rel="noopener" href="http://www.robertlanfear.com/partitionfinder/assets/Manual_v2.1.x.pdf">http://www.robertlanfear.com/partitionfinder/assets/Manual_v2.1.x.pdf</a>。</p>
<h3 id="5-10-2-分区编辑器简要教程"><a href="#5-10-2-分区编辑器简要教程" class="headerlink" title="5.10.2. 分区编辑器简要教程"></a>5.10.2. <span id="partition_editor">分区编辑器简要教程</span></h3><ol>
<li>数据显示块名称左侧显示的数字 <code>3</code> 表示序列长度是 3 的倍数，选择一个或多个显示 <code>3</code> 图标的数据块（确保它们是蛋白质编码基因），然后点击 <code>密码子模式</code> 按钮，分区将更改为密码子模式，其中图标 <code>1</code>、<code>2</code> 和 <code>3</code> 分别对应包含该基因第一、第二和第三密码子位置的分区；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit1.jpg?raw=true"></li>
<li>选择带有图标 <code>1</code>、<code>2</code> 和 <code>3</code> 的基因名称，然后点击 <code>取消密码子模式</code> 以切换回正常的分区模式。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit2.jpg?raw=true"></li>
<li><code>名称</code>、<code>起始</code> 和 <code>终止</code> 列可以通过双击相应的单元格进行修改；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit3.jpg?raw=true"></li>
<li>关闭窗口将自动保存修改后的分区；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit4.jpg?raw=true"></li>
<li>如果您想手动向数据块添加分区，可以在下面的文本框中粘贴分区格式的文本，然后点击 <code>识别</code> 按钮。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/part_edit5.jpg?raw=true"></li>
</ol>
<h2 id="5-11-IQ-TREE-1"><a href="#5-11-IQ-TREE-1" class="headerlink" title="5.11. IQ-TREE"></a>5.11. <em>IQ-TREE</em></h2><p>关于 IQ-TREE 的安装，请参阅<a href="#plugins">插件安装</a>部分；关于结果，请参阅<a href="#Output">输出文件</a>部分；关于输入文件，请参阅<a href="#input">输入文件</a>部分。允许 FASTA、PHYLIP、NEXUS 和 CLUSTAL 格式。</p>
<p>可选地，您可以输入一个分区文件（勾选复选框）。关于分区文件的详细格式要求，请参阅 <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/Advanced-Tutorial">http://www.iqtree.org/doc/Advanced-Tutorial</a>。最方便的选项是使用 <code>串联序列</code> (<a href="#concatenate_results">concatenate_results</a>) 的结果作为 IQ-TREE 的输入文件：串联的比对和分区文件将自动加载到 IQ-TREE 中。类似地，当使用 <code>PartitionFinder2</code> 或 <code>ModelFinder</code> 的结果作为 IQ-TREE 的输入文件时，比对文件、分区和计算出的最佳拟合模型也将自动加载到 IQ-TREE 中。</p>
<p>或者，IQ-TREE 可以通过将 <code>模型</code> 设置为 <code>自动</code> 并勾选（<code>-m TESTNEW</code>）或不勾选（<code>-m TEST</code>）’FreeRate 异质性 [+R]’ 来选择最佳拟合模型并立即继续进行树重建（使用推断出的模型）。我们还使 IQ-TREE 能够批量重建系统发育树，这可用于推断超树。</p>
<p>输入文件并设置参数后，您可以启动程序（<code>开始</code> 按钮），并通过 <code>显示日志</code> 按钮查看运行日志。程序完成后，<code>IQ-TREE</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-11-1-简要示例"><a href="#5-11-1-简要示例" class="headerlink" title="5.11.1. 简要示例"></a>5.11.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\IQ-TREE\mtDNA_36_genes\36_genes_NUC\normal’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择 ‘concatenation.phy’ 文件；</li>
<li>通过菜单栏打开 <code>系统发育--&gt;IQ-TREE</code>；</li>
<li>将其拖放到文件输入框中；</li>
<li>选择最佳拟合进化模型及相关参数（+I, +G 等）（这里如果您选择 <code>自动</code>，IQ-TREE 将选择最佳拟合模型并立即继续进行树重建，见上文）；</li>
<li>参数可根据您自己的需要进行设置（如果您没有分区文件，请记住取消勾选 <code>分区模式</code>），可以通过 ‘开始’ 按钮下拉箭头中的 <code>查看 | 编辑命令</code> 功能添加 GUI 中未包含的参数；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/iqtree_demo.jpg?raw=true"></li>
</ol>
<p>IQ-TREE 可以直接使用 ModelFinder 和&#x2F;或 PartitionFinder2 的输出，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">多基因教程</a > 和 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">单基因教程</a >。关于 IQ-TREE 的完整手册，请访问 <a target="_blank" rel="noopener" href="http://www.iqtree.org/doc/">http://www.iqtree.org/doc/</a> 和 <a target="_blank" rel="noopener" href="http://iqtree.cibiv.univie.ac.at/">http://iqtree.cibiv.univie.ac.at/</a>。</p>
<h2 id="5-12-MrBayes-1"><a href="#5-12-MrBayes-1" class="headerlink" title="5.12. MrBayes"></a>5.12. <em>MrBayes</em></h2><p>关于 MrBayes 的安装，请参阅<a href="#plugins">插件安装</a>部分；关于结果文件，请参阅<a href="#Output">输出文件</a>部分；关于<a href="#input">输入文件</a>，请参阅输入文件部分。<span style="color: red">请注意，仅允许 NEXUS 格式；如果使用<a href="#autodetect">自动检测</a>功能，比对将自动转换为 NEXUS 格式。</span> 当使用 <code>PartitionFinder2</code> 或 <code>ModelFinder</code> 的结果作为 MrBayes 的输入文件时，比对文件和计算出的最佳拟合模型将自动加载到 MrBayes 中。</p>
<p>如果加载的比对文件包含命令块，您可以选择直接使用此命令块运行。<code>外群</code> 和 <code>模型</code> 参数仅在比对加载后启用。</p>
<p>PhyloSuite 提供了一个编辑分区文件的窗口（通过点击 <code>分区模型</code> 激活），您可以在其中输入子集的名称、起始和终止位置以及该子集的最佳模型。编辑后，您可以点击 <code>生成命令块</code> 按钮为编辑后的分区生成相应的命令块。</p>
<p>有时，在完成分析后，您可能会认为结果尚未完全收敛，并希望继续分析；对于这种情况，PhyloSuite 提供了 <code>继续先前分析</code> 功能，允许您在设置额外的代数后继续任何分析（已完成或未完成）。</p>
<p>在计算汇总统计量时，有两种丢弃 MCMC 样本（非代数）的方法：您可以设置特定的样本数量（<code>Burnin 框</code>）或所有样本的比例（<code>Burnin 分数框</code>）。</p>
<p><code>共识树格式</code> 参数控制共识树的格式，其中 <code>简单</code> 设置生成一个简单的共识树，其格式可被各种程序（TreeView、iTOL 等）读取；而 <code>Figtree</code> 设置生成一个为 FigTree 程序格式化的共识树，带有丰富的汇总统计信息。</p>
<p><code>显示 MrBayes 数据块</code> 按钮允许您添加 GUI 中未包含的参数，或导出配置的文件并在服务器（如 CIPRES，参见简要示例）上运行。</p>
<p>输入文件并设置参数后，您可以导出比对和相应的命令块以单独执行 MrBayes（通过 <code>显示 MrBayes 数据块</code>），或者点击 <code>开始</code> 按钮在 PhyloSuite 内运行程序。可以通过 <code>显示日志</code> 按钮查看运行日志。程序完成后，<code>MrBayes</code> 的参数设置和引用将保存在 <code>summary.txt</code> 文件中。</p>
<h3 id="5-12-1-简要示例"><a href="#5-12-1-简要示例" class="headerlink" title="5.12.1. 简要示例"></a>5.12.1. 简要示例</h3><p>当您在 PhyloSuite 根目录下时，进入 ‘example\MrBayes\mtDNA_36_genes\36_genes_NUC\normal’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择 <code>input.nex</code> 文件；</li>
<li>通过菜单栏打开 <code>系统发育--&gt;MrBayes</code>；</li>
<li>将其拖放到文件输入框中；</li>
<li>选择最佳拟合进化模型及相关参数（+I, +G 等）；</li>
<li>参数可根据您自己的需要进行设置；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_demo.jpg?raw=true"></li>
<li>如果您想导出设置以在 CIPRES 上运行 MrBayes，请点击 <code>显示 MrBayes 数据块</code>，然后选择 ‘保存到文件’，将此文件上传到 CIPRES 直接运行（记住勾选 <code>我的数据包含 MrBayes 数据块</code>）。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_cipres.jpg?raw=true"></li>
<li>如果您想在运行时查看树和收敛诊断结果，可以通过 <code>停止</code> 按钮下拉箭头中的 <code>停止运行并推断树</code> 选项来实现。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/stop_mb_run.jpg?raw=true"></li>
<li>如果您希望重新启动先前的运行（未完成或已完成），请点击 <code>继续先前分析</code>。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mb_continue.jpg?raw=true"></li>
</ol>
<p>MrBayes 可以直接使用 ModelFinder 和&#x2F;或 PartitionFinder2 的输出，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/">多基因教程</a > 和 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/">单基因教程</a >。关于 MrBayes 的完整手册，请访问 <a target="_blank" rel="noopener" href="http://mrbayes.sourceforge.net/manual.php">http://mrbayes.sourceforge.net/manual.php</a>。</p>
<h2 id="5-13-工作流"><a href="#5-13-工作流" class="headerlink" title="5.13. 工作流"></a>5.13. <em>工作流</em></h2><p>此功能简化了进化系统发育学分析的过程，包括序列比对（<code>MAFFT</code> 和 <code>MACSE</code>）、消除比对不佳的位置和差异区域（<code>Gblocks</code>、<code>trimAl</code> 和 <code>HmmCleaner</code>）、序列串联（<code>串联</code>）、模型选择（<code>ModelFinder</code> 或 <code>PartitionFinder</code>）以及树重建（<code>MrBayes</code> 和 <code>IQ-TREE</code>）。默认情况下，PhyloSuite 预定义了七种不同的工作流，但您也可以通过添加按钮配置&#x2F;删除自己的工作流。这些允许您快速重复分析。</p>
<p>使用此功能时，有几件事需要牢记：</p>
<ul>
<li>如下图所示，这些程序的执行顺序是 [<code>MAFFT</code> 和&#x2F;或 <code>MACSE</code>]–&gt;[<code>Gblocks</code> 或 <code>trimAl</code> 或 <code>HmmCleaner</code>]–&gt;<code>串联</code>–&gt;[<code>ModelFinder</code> 或 <code>PartitionFinder</code>]–&gt;[<code>IQ-TREE</code> 和 <code>MrBayes</code>]。<br> <img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_en.jpg?raw=true"><br></li>
<li>如果您同时选择 <code>MAFFT</code> 和 <code>MACSE</code>，应使用蛋白质编码序列作为输入，并且 <code>MAFFT</code> 的结果随后将由 <code>MACSE</code> 进行优化。</li>
<li>只能选择三种比对优化程序中的一种。</li>
<li>只能选择两种模型选择程序中的一种。</li>
<li>除了模型选择程序和 <code>串联</code> 外，其他程序不必选择（当选择 <code>MAFFT</code>、<code>MACSE</code>、<code>trimAl</code>、<code>HmmCleaner</code> 或 <code>Gblocks</code> 时，必须保留 <code>串联</code>，因为它充当连接这些程序与下游程序的桥梁，即使是单基因也是如此）。</li>
<li>只有第一个程序需要输入文件，而其他程序的输入文件将从上游分析的结果中自动检测。<span style="color: red">请注意，两个 <code>树重建</code> 程序可以使用 <code>ModelFinder</code> 或 <code>PartitionFinder</code> 的结果，并且它们可以并行运行。</span></li>
<li>由于 <code>保守位置的最小序列数</code> 和 <code>侧翼位置的最小序列数</code> 选项仅在文件直接输入到 <a href="#gblocks">Gblocks</a> 时才启用，因此在工作流模式下，这两个选项默认设置为最“宽松”的值（即最低值），除非 <code>Gblocks</code> 是工作流分析中的第一个程序，在这种情况下，您可以像通常那样设置这两个选项。</li>
<li>对于模型选择和树重建，如果仅选择 <code>ModelFinder</code> 和 <code>IQ-TREE</code>，则 <code>IQ-TREE</code> 将使用 <code>ModelFinder</code> 计算出的最佳拟合模型；如果仅选择 <code>ModelFinder</code> 和 <code>MrBayes</code>，则必须在 <code>ModelFinder</code> 的 <code>模型用于</code> 菜单中选择 <code>Mrbayes</code> 选项；最后，如果选择了 <code>ModelFinder</code>、<code>IQ-TREE</code> 和 <code>MrBayes</code>，则 <code>ModelFinder</code> 的结果将仅用于 <code>MrBayes</code>（因此它将使用前述说明中描述的相同设置），而 <code>IQ-TREE</code> 将首先执行算法内置的最佳拟合模型选择，然后进行树推断（使用 <code>模型</code> 菜单中的 <code>自动</code> 选项，相当于 <code>-m TEST</code> 或 <code>-m MFP</code>）。</li>
<li>PhyloSuite 还提供了一个功能，用于检查和自动校正所选程序之间的参数，包括前述说明中指定的参数、冲突的序列类型、冲突的分区模式等。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/flowchart_settings.jpg?raw=true"><br></li>
<li>当工作流完成时，相应软件程序的参数设置和引用将汇总在 <code>工作流</code> 的<a href="#operation">显示区域</a>中。</li>
</ul>
<h3 id="5-13-1-简要示例"><a href="#5-13-1-简要示例" class="headerlink" title="5.13.1. 简要示例"></a>5.13.1. 简要示例</h3><p>提示：如果您更改了工作流设置，请记住使用 <code>添加</code> 按钮保存它，否则它将不会被记住。关于综合演示，请参阅 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/multi-gene-phylogeny/#9-工作流">多基因教程</a > 和 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/PhyloSuite-demo/single-gene-phylogeny/#7-工作流">单基因教程</a >。</p>
<h2 id="5-14-线粒体基因组"><a href="#5-14-线粒体基因组" class="headerlink" title="5.14. 线粒体基因组"></a>5.14. <em>线粒体基因组</em></h2><h3 id="5-14-1-解析注释"><a href="#5-14-1-解析注释" class="headerlink" title="5.14.1. 解析注释"></a>5.14.1. 解析注释</h3><p>此功能可以解析记录在 Microsoft Word 文档中的注释（仅支持 *.docx 扩展名）。注释 tRNA 时，应将每个 tRNA 基因的反密码子添加到基因名称的末尾（在括号中），例如：tRNA-Cys(GCA)（另请参见下图中的示例）。关于基因的名称，PhyloSuite 允许您通过 <code>配置名称替换</code> 按钮访问的 <code>来自 Word 的名称</code> 表来将名称替换为其他名称。此外，您可以为每个蛋白质编码基因定义 <code>product</code> <a href="#qualifier">限定符</a> 的名称，以及用于组织结构表的 tRNA 基因的缩写。</p>
<p>Word 文档中线粒体基因组注释示例：<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/annotations.jpg?raw=true"><br></p>
<p>包含作者和单位信息的 GenBank 提交模板文件可以<a target="_blank" rel="noopener" href="https://submit.ncbi.nlm.nih.gov/genbank/template/submission/" title="提交门户">在此处</a>生成。来自 PhyloSuite 的几个数据集可用于生成注释部分，包括 <code>物种</code>、<code>品系</code>、<code>谱系</code> 等。<code>发布日期</code> 参数定义您序列的发布日期。<span style="color: red">请注意，您的计算机上应安装有 Office 套件。</span></p>
<h4 id="5-14-1-1-简要示例"><a href="#5-14-1-1-简要示例" class="headerlink" title="5.14.1.1. 简要示例"></a>5.14.1.1. 简要示例</h4><p>当您在 PhyloSuite 根目录下时，进入 ‘example\Parse_Word_annotations’ 文件夹（如果您没有最新的示例文件夹，请从 <a target="_blank" rel="noopener" href="https://raw.githubusercontent.com/dongzhang0725/PhyloSuite/master/example.zip">此处</a > 下载），</p>
<ol>
<li>选择 ‘Diplectanum_longipenis_mtDNA.docx’ 文件（您可以打开此文件查看如何注释序列）；</li>
<li>通过菜单栏打开 <code>线粒体基因组--&gt;解析注释</code>；</li>
<li>将文件拖放到文件输入框中；</li>
<li>点击蓝色文字（如图所示）生成模板文件；</li>
<li>将模板文件拖放到 <code>模板文件</code> 输入框中；</li>
<li>填写必要信息，例如 <code>物种</code>、<code>谱系</code>、<code>密码子表</code> 等。</li>
<li>其他参数可根据您自己的需要进行设置；</li>
<li>启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/parseANNT_demo.jpg?raw=true"></li>
</ol>
<h3 id="5-14-2-比较表格"><a href="#5-14-2-比较表格" class="headerlink" title="5.14.2. 比较表格"></a>5.14.2. 比较表格</h3><p>此功能可以比较和汇总 <code>extract_results</code> 文件夹下 <code>speciesStat</code> 子文件夹中的表格。对于组织结构表，允许进行成对相似性计算，其中调用 MAFFT 进行比对，并使用 Biopython 中的 DistanceCalculator 包计算序列的同一性。可以通过选择希望排除的行数（从顶部开始）来将表格的标题从比较中省略。关于表格示例，请参阅 <a target="_blank" rel="noopener" href="https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3">https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-018-1249-3</a> 中的表 1 和 <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-018-2910-9">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-018-2910-9</a> 中的表 2。</p>
<h4 id="5-14-2-1-简要示例"><a href="#5-14-2-1-简要示例" class="headerlink" title="5.14.2.1. 简要示例"></a>5.14.2.1. 简要示例</h4><p>此功能可以直接使用 ‘extract’ 功能的结果：</p>
<ol>
<li>选择 <code>extract_results</code> 文件夹（如果不可用，请参阅<a href="#extract_exam">此处</a>了解如何创建，仅限线粒体基因组数据类型）；</li>
<li>通过菜单栏打开 <code>线粒体基因组--&gt;比较表格</code>；</li>
<li>所有提取的组织结构表将自动导入；使用 <code>移除</code> 按钮移除您不感兴趣的表格；</li>
<li>如果要计算同源基因的成对相似性，请勾选 <code>计算成对相似性</code>；</li>
<li>启动程序；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_demo.jpg?raw=true"></li>
<li>如果要比较核苷酸组成和偏度表（通过结果文件夹中名称不包含 ‘_org’ 来识别），您应首先打开 <code>extract_results</code> 文件夹，然后进入 ‘extract_results\StatFiles\speciesStat’，选择感兴趣的文件，将它们拖放到 <code>表格</code> 框中，取消勾选 <code>计算成对相似性</code>，然后启动程序。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/compare_table_notorg.jpg?raw=true"></li>
</ol>
<h3 id="5-14-3-绘制-RSCU-图"><a href="#5-14-3-绘制-RSCU-图" class="headerlink" title="5.14.3. 绘制 RSCU 图"></a>5.14.3. 绘制 RSCU 图</h3><p>关于 Rscript 的安装，请参阅<a href="#plugins">插件安装</a>部分，关于结果，请参阅<a href="#Output">输出文件</a>部分。</p>
<p>此功能可以根据 <code>extract_results/StatFiles/RSCU</code> 文件夹下 “RSCU” 子文件夹中的表格绘制 RSCU 图。您可以通过拖拽重新排序输入文件和 x 轴上的氨基酸。关于图形示例，请参阅 <a target="_blank" rel="noopener" href="https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1">https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2404-1</a> 中的图 3。</p>
<h4 id="5-14-3-1-简要示例"><a href="#5-14-3-1-简要示例" class="headerlink" title="5.14.3.1. 简要示例"></a>5.14.3.1. 简要示例</h4><p>此功能可以直接使用 ‘extract’ 功能的结果：</p>
<ol>
<li>选择 <code>extract_results</code> 文件夹（如果不可用，请参阅<a href="#extract_exam">此处</a>了解如何创建，仅限线粒体基因组数据类型）；</li>
<li>通过菜单栏打开 <code>线粒体基因组--&gt;绘制 RSCU 图</code>；</li>
<li>所有提取的 RSCU 表将自动导入，使用 <code>移除</code> 按钮移除您不感兴趣的表格；</li>
<li>参数可根据您自己的需要进行设置；</li>
<li>启动程序；<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/RSCU_demo.jpg?raw=true"></li>
</ol>
<h2 id="5-15-分子钟分析"><a href="#5-15-分子钟分析" class="headerlink" title="5.15. 分子钟分析"></a>5.15. <em>分子钟分析</em></h2><p>请参阅 <a href="https://dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/">https://dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/</a> 或 <a target="_blank" rel="noopener" href="http://phylosuite.jushengwu.com/dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/">http://phylosuite.jushengwu.com/dongzhang0725.github.io/PhyloSuite-demo/Molecular-dating-analysis/</a></p>
<h1 id="6-引用与代码"><a href="#6-引用与代码" class="headerlink" title="6. 引用与代码"></a>6. 引用与代码</h1><p>如果您在科学论文中使用 PhyloSuite 生成的数据，请使用以下引用：</p>
<blockquote>
<p>Zhang, D., F. Gao, I. Jakovlić, H. Zou, J. Zhang, W.X. Li, and G.T. Wang, PhyloSuite: An integrated and scalable desktop platform for streamlined molecular sequence data management and evolutionary phylogenetics studies. Molecular Ecology Resources, 2020. 20(1): p. 348–355. DOI: 10.1111&#x2F;1755-0998.13096.</p>
</blockquote>
<p>还请注意的是，PhyloSuite 是一个插件程序，您还应引用您在分析中使用的、非我们设计和编译的任何（及每一个）插件程序。这适用于以下插件：<br><br></p>
<p><strong>MAFFT</strong></p>
<blockquote>
<p>Katoh, K., and Standley, D.M. (2013). MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol 30, 772-780.</p>
</blockquote>
<br>

<p><strong>MACSE</strong></p>
<blockquote>
<p>Ranwez V, Douzery EJP, Cambon C, Chantret N, Delsuc F. 2018. MACSE v2: Toolkit for the alignment of coding sequences accounting for frameshifts and stop codons. Mol Biol Evol. 35: 2582-2584. doi: 10.1093&#x2F;molbev&#x2F;msy159.</p>
</blockquote>
<br>

<p><strong>Gblocks</strong></p>
<blockquote>
<p>Talavera, G., and Castresana, J. (2007). Improvement of phylogenies after removing divergent and ambiguously aligned blocks from protein sequence alignments. Syst Biol 56, 564-577.</p>
</blockquote>
<br>

<p><strong>trimAl</strong></p>
<blockquote>
<p>Capella-Gutierrez S, Silla-Martinez JM, Gabaldon T. 2009. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 25: 1972-1973. doi: 10.1093&#x2F;bioinformatics&#x2F;btp348.</p>
</blockquote>
<br>

<p><strong>HmmCleaner</strong></p>
<blockquote>
<p>Di Franco A, Poujol R, Baurain D, Philippe H. 2019. Evaluating the usefulness of alignment filtering methods to reduce the impact of errors on evolutionary inferences. BMC Evol Biol. 19: 21. doi: 10.1186&#x2F;s12862-019-1350-2.</p>
</blockquote>
<br>

<p><strong>IQ-TREE</strong></p>
<blockquote>
<p>Nguyen, L.T., Schmidt, H.A., von Haeseler, A., and Minh, B.Q. (2015). IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies. Mol Biol Evol 32, 268-274.</p>
</blockquote>
<br>

<p><strong>PartitionFinder2</strong></p>
<blockquote>
<p>Lanfear, R., Frandsen, P.B., Wright, A.M., Senfeld, T., and Calcott, B. (2017). PartitionFinder 2: new methods for selecting partitioned models of evolution for molecular and morphological phylogenetic analyses. Mol Biol Evol 34, 772-773.</p>
</blockquote>
<br>

<p><strong>MrBayes</strong></p>
<blockquote>
<p>Ronquist, F., Teslenko, M., van der Mark, P., Ayres, D.L., Darling, A., Höhna, S., Larget, B., Liu, L., Suchard, M.A., and Huelsenbeck, J.P. (2012). MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space. Syst Biol 61, 539-542.</p>
</blockquote>
<br>

<p>对于其余功能，我们主要使用自己的 Python 代码，使用 Python 3.6.7 和 PyQT5 编写。某些功能使用了 Biopython 包，例如从 GenBank 文件中提取特征，这是使用 SeqIO 模块进行的。</p>
<blockquote>
<p>Cock, P.J., Antao, T., Chang, J.T., Chapman, B.A., Cox, C.J., Dalke, A., Friedberg, I., Hamelryck, T., Kauff, F., Wilczynski, B., et al. (2009). Biopython: freely available Python tools for computational molecular biology and bioinformatics. Bioinformatics 25, 1422-1423.</p>
</blockquote>
<h1 id="7-故障排除"><a href="#7-故障排除" class="headerlink" title="7. 故障排除"></a>7. 故障排除</h1><h2 id="7-1-更新失败：如何恢复到以前的设置和插件"><a href="#7-1-更新失败：如何恢复到以前的设置和插件" class="headerlink" title="7.1. 更新失败：如何恢复到以前的设置和插件"></a>7.1. 更新失败：如何恢复到以前的设置和插件</h2><p>在极少数情况下，用户在更新 PhyloSuite 时可能会遇到错误。因为这可能导致您丢失一些设置和配置，这里我们将演示如何将您的设置和插件恢复到更新前的状态。</p>
<ol>
<li>首先，您应该下载最新的 PhyloSuite 包，地址为 <a target="_blank" rel="noopener" href="https://github.com/dongzhang0725/PhyloSuite/releases">https://github.com/dongzhang0725/PhyloSuite/releases</a> 或 <a href="https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link">https://dongzhang0725.github.io/dongzhang0725.github.io/installation/#Chinese_download_link</a>（中国）。<span style="color: red"> 注意：对于 Windows，您应该下载 <code>PhyloSuite_xxx_Win.rar</code>，而不是安装程序文件。</span></li>
</ol>
<table>
<thead>
<tr>
<th>系统</th>
<th>包文件</th>
</tr>
</thead>
<tbody><tr>
<td>Windows</td>
<td>PhyloSuite_xxx_Win.rar</td>
</tr>
<tr>
<td>Linux</td>
<td>PhyloSuite_xxx_Linux.tar.gz</td>
</tr>
<tr>
<td>Mac OSX</td>
<td>PhyloSuite_xxx_Mac.zip</td>
</tr>
</tbody></table>
<ol start="2">
<li><p>解压包，选择并复制所有文件，转到 PhyloSuite 的安装路径，打开 <code>PhyloSuite</code> 文件夹，将复制的文件直接粘贴到此文件夹中（如果提示，请确认您希望替换同名的文件）。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/update_failed.jpg?raw=true"></p>
</li>
<li><p>打开 PhyloSuite，您应该会发现您正在运行更新后的版本，并且您以前的设置已被保留。<br><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/open_phylosuite.jpg?raw=true"></p>
</li>
</ol>
<h2 id="7-2-PhyloSuite-运行失败"><a href="#7-2-PhyloSuite-运行失败" class="headerlink" title="7.2. PhyloSuite 运行失败"></a>7.2. PhyloSuite 运行失败</h2><p>如果 PhyloSuite 执行失败，请首先尝试关闭您的杀毒软件。</p>
<h2 id="7-3-MrBayes-无法工作"><a href="#7-3-MrBayes-无法工作" class="headerlink" title="7.3. MrBayes 无法工作"></a>7.3. MrBayes 无法工作</h2><p>有时 MrBayes 会立即完成，但不报告错误。通常您可以尝试在终端中执行 MrBayes 来查找问题：</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/exe_mb_terminal.jpg?raw=true"></p>
<p>如果您在 Windows 中遇到 ‘msvcr120.dll 丢失’ 错误，您可以通过<a target="_blank" rel="noopener" href="https://www.drivereasy.com/knowledge/best-solutions-to-fix-msvcr120-dll-is-missing-in-windows/">此解决方案</a>进行修复。</p>
<p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/manual/mrbayes_error_win.png?raw=true"></p>
<p>对于其他问题，请在网站上搜索 <code>错误代码</code>。</p>
<h2 id="7-4-PhyloSuite-卡顿"><a href="#7-4-PhyloSuite-卡顿" class="headerlink" title="7.4. PhyloSuite 卡顿"></a>7.4. PhyloSuite 卡顿</h2><p>如果 PhyloSuite 变得越来越卡顿，这可能是由于工作区中的数据不断增加造成的。要解决此问题，您应该创建一个新的工作区。通常，PhyloSuite 鼓励用户创建多个工作区来保存他们的工作。</p>
<h2 id="7-5-MAFFT-错误"><a href="#7-5-MAFFT-错误" class="headerlink" title="7.5. MAFFT 错误"></a>7.5. MAFFT 错误</h2><p>如果您遇到 MAFFT 错误，例如：</p>
<ul>
<li><code>/usr/bin/awk: 无法执行二进制文件</code></li>
<li><code>-gt: 需要一元运算符</code></li>
<li><code>选项：检查源文件</code></li>
</ul>
<p>请尝试以下步骤：</p>
<ol>
<li>从官方网站重新安装 MAFFT：<a target="_blank" rel="noopener" href="https://mafft.cbrc.jp/alignment/software/">https://mafft.cbrc.jp/alignment/software/</a>。</li>
<li>在 <strong>PhyloSuite</strong> 中指定 MAFFT 可执行文件。详细说明请参阅 <a target="_blank" rel="noopener" href="http://phylosuite.jushengwu.com/dongzhang0725.github.io/PhyloSuite-demo/how-to-configure-plugins/">如何在 PhyloSuite 中配置插件</a>。</li>
<li><strong>重要提示：</strong> 对于 MAFFT，请确保在 PhyloSuite 中将 <code>mafft.bat</code> 文件指定为可执行文件。</li>
</ol>
<h2 id="7-6-无法在MAC系统运行PhyloSuite"><a href="#7-6-无法在MAC系统运行PhyloSuite" class="headerlink" title="7.6. 无法在MAC系统运行PhyloSuite"></a>7.6. 无法在MAC系统运行PhyloSuite</h2><p><img src="https://github.com/dongzhang0725/PhyloSuite_tutorial/blob/master/images/installation/MAC_error1.png?raw=true"></p>
<p>对于MAC用户，如果遇到<code>安全性/隐私问题</code>或<code>Apple无法验证“某某文件”</code>问题，可尝试使用以下命令解决：</p>
<figure class="highlight plaintext"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">sudo xattr -rd [PhyloSuite_installation_path]/PhyloSuite</span><br></pre></td></tr></table></figure>

<h1 id="8-致谢"><a href="#8-致谢" class="headerlink" title="8. 致谢"></a>8. 致谢</h1><p>我们要感谢孟开开博士帮助我们建立了叶绿体基因组提取功能。</p>
</div>

<!-- JS 切换逻辑 + 按钮高亮 + 记忆用户选择 -->
<script>
document.addEventListener('DOMContentLoaded', function() {
  const buttons = document.querySelectorAll('.lang-btn');
  const langContents = document.querySelectorAll('.lang');

  // 切换语言函数
  function switchLang(lang){
    // 隐藏所有内容
    langContents.forEach(el => el.style.display = 'none');
    // 显示对应语言
    document.querySelectorAll('.lang-' + lang).forEach(el => el.style.display = 'block');
    
    // 高亮按钮
    buttons.forEach(btn => btn.classList.remove('active'));
    document.querySelector(`.lang-btn[onclick="switchLang('${lang}')"]`).classList.add('active');
    
    // 保存选择到 localStorage
    localStorage.setItem('preferredLang', lang);
  }

  // 页面加载时读取用户上次选择
  const savedLang = localStorage.getItem('preferredLang') || 'zh';
  switchLang(savedLang);

  // 给按钮绑定点击事件
  buttons.forEach(btn => {
    btn.addEventListener('click', () => {
      const lang = btn.getAttribute('onclick').match(/switchLang\('(\w+)'\)/)[1];
      switchLang(lang);
    });
  });
});
</script>

<!-- CSS 美化按钮 -->
<style>
#lang-switcher {
  display: flex;
  gap: 10px;
}

.lang-btn {
  padding: 6px 14px;
  border: 1px solid #888;
  border-radius: 6px;
  background-color: #f5f5f5;
  cursor: pointer;
  transition: 0.2s;
  font-weight: bold;
}

.lang-btn:hover {
  background-color: #e0e0e0;
}

.lang-btn.active {
  background-color: #007bff;
  color: white;
  border-color: #007bff;
}
</style>
      </div>
      
      
      
    </div>

    
    


    <div class="comments gitalk-container"></div>
</div>
  </main>

  <footer class="footer">
    <div class="footer-inner">

  <div class="copyright">
    &copy; 
    <span itemprop="copyrightYear">2025</span>
    <span class="with-love">
      <i class="user"></i>
    </span>
    <span class="author" itemprop="copyrightHolder">Dong Zhang</span>
  </div>
  <div class="powered-by">Powered by <a href="https://hexo.io/" rel="noopener" target="_blank">Hexo</a> & <a href="https://theme-next.js.org/" rel="noopener" target="_blank">NexT.Gemini</a>
  </div>

    </div>
  </footer>

  
  <div class="toggle sidebar-toggle" role="button">
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
    <span class="toggle-line"></span>
  </div>
  <div class="sidebar-dimmer"></div>
  <div class="reading-progress-bar"></div>

<noscript>
  <div class="noscript-warning">Theme NexT works best with JavaScript enabled</div>
</noscript>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/gitalk/1.8.0/gitalk.css" integrity="sha256-AJnUHL7dBv6PGaeyPQJcgQPDjt/Hn/PvYZde1iqfp8U=" crossorigin="anonymous">

<script class="next-config" data-name="gitalk" type="application/json">{"enable":true,"github_id":"dongzhang0725","repo":"PhyloSuite","client_id":"55e7b2bd2c9d294158bf","client_secret":"b0716af44800887bfab50497c5e0309525a353b8","admin_user":"dongzhang0725","distraction_free_mode":true,"proxy":"https://cors-anywhere.azm.workers.dev/https://github.com/login/oauth/access_token","language":null,"js":{"url":"https://cdnjs.cloudflare.com/ajax/libs/gitalk/1.8.0/gitalk.min.js","integrity":"sha256-MVK9MGD/XJaGyIghSVrONSnoXoGh3IFxLw0zfvzpxR4="},"path_md5":"41996dd9e4e7fa35a307f5f9e74853e7"}</script>
<script src="/js/third-party/comments/gitalk.js" defer></script>

</body>
</html>
